{% include header.html %}

# environmental DNA (eDNA)

## Project Title
eDNA checklist

## Project Lead(s)
Miwa Takahashi, miwa.takahashi@csiro.au, Commonwealth Scientific and Industrial Research Organisation (CSIRO)

## Team members
 - Luke Thompson, lrt175@ngi.msstate.edu, Mississippi State University
 - Katherine Silliman, ks567@msstate.edu, Mississippi State University
 - Chris Hunter, only1chunts@gmail.com, GigaScience Press
 - Peter Woollard, woollard@ebi.ac.uk, European Molecular Biology Laboratory 
 - Joana Pauperio, joanap@ebi.ac.uk, European Molecular Biology Laboratory
 - Steve Formel, steve@formeldataservices.com, Formel Data Services
 - Camila Babo, camila.babo@cibio.up.pt, CIBIO - Research Centre in Biodiversity and Genetic Resources
 - Tobias Frøslev, tfroeslev@gbif.org, GBIF
 - Lynn M Schriml, lynn.schriml@gmail.com, University of Maryland
 - Saara Suominen, s.suominen@unesco.org, Ocean Biodiversity Information System
 - Kristy Deiner, alpinedna@gmail.com


## Elevator pitch (20 – 50 words)
Environmental DNA (eDNA) is a powerful biodiversity monitoring tool, but inconsistent/incomplete metadata limits its interoperability and reuse. This project seeks to solve that challenge by incorporating existing and new terms and controlled vocabularies which accommodate unique eDNA requirements into the GSC suite.

## Project Summary 
Biodiversity assessment using environmental DNA (eDNA) has emerged as a powerful tool for monitoring species and ecosystems. Scientific studies generate vast amounts of eDNA sequences and species detections. However, these datasets are often scattered across repositories, with inconsistent and inadequate metadata, limiting their interoperation, reuse, and overall impact. Adopting FAIR (Findable, Accessible, Interoperable, Reusable) data practices with eDNA data can transform monitoring of biodiversity and individual species, and support data-driven conservation at broad scales. 
Despite growing recognition of FAIR principles, their implementation in the eDNA community remains limited, partly due to gaps in adapting existing vocabularies, such as Darwin Core (DwC) and Minimum Information about any (x) Sequence (MIxS), to eDNA-specific needs and workflows. For example, the approaches and parameters used for quality filtering, species detection, monitoring contamination and excluding non-target taxa vary greatly between studies, depending on the study scopes. Ensuring such metadata is FAIR is critical for future reuse, particularly when species detection requires high confidence in taxonomic assignments. Increasing efforts have been made to establish minimum reporting requirements to validate eDNA study methods and data (e.g., Klymus et al., 2020, Thalinger et al., 2021, Klymus et al., 2024, Takahashi et al., in press). These requirements need further development to fit into the GSC suite. 
Leveraging the network of collaborators (e.g., GSC, TDWG, ENA, GBIF, OBIS, ALA), we aim to develop a MIXS - eDNA checklist, bridge gaps in metadata standards for eDNA-specific workflows, and facilitate its adoption within the INSDC system, ensuring broad and long-term usability and accessibility across data infrastructures and eDNA practitioners.

## Project initiation date 
The project will start in May 2025.

## What will this project aim to contribute to the GSC?
This project will aim to extend the MIxS checklist by incorporating terms from other checklists (e.g., DNA-derived data extension, DwC terms, FAIRe metadata checklist). This contribution will help standardise eDNA data reporting and improve interoperability across global omics/biodiversity data infrastructures. 

## Have you spoken about the project already within GSC?
The project has been discussed with the GSC members (i.e., Chris Hunter, Lynn Schriml, Peter Woollard). 

## Which existing projects, if any, does this one replace/complement/subsume/expand? Explain briefly why an extra project is needed/justified (what gap does it fill?)
The eDNA approach has unique workflows and data attributes that distinguish it from other types of biodiversity and omics data, which are not fully addressed by existing MIxS checklists. An extra project is needed to fill that gap by incorporating eDNA-specific expertise and perspectives, ensuring that metadata standards align with the distinct needs of the eDNA community. 

## How does this project fit into GSC’s mission statement?
Our project aims to make eDNA data FAIR by establishing dedicated eDNA data standards, designed for adoption by major eDNA data platforms, including GBIF and INSDC. As eDNA data encompasses both biodiversity and genomics attributes, this initiative directly aligns with GSC’s mission to enable genomic data integration, discovery, and comparison through international community-driven standards.’

## Will you start a GSC working group?
Yes. While existing initiatives focused on eDNA data standards have been engaging a broad community across disciplines, there is a need for a more technically focused GSC working group dedicated to developing the GSC-eDNA checklist. I plan to establish this group, incorporating select members already involved in eDNA data standardisation efforts, along with interested GSC members who can support this technical effort. 

## How do you wish to further engage the GSC?
I wish to expand the team by engaging GSC members to leverage their expertise and insights. Regular discussions with GSC members will be key to making progress and refining our approach. Linking to other GSC projects, particularly MIOP (https://www.gensc.org/pages/projects/miop-project.html) and MInAs (http://www.gensc.org/pages/projects/minas-proposal.html), would allow us to facilitate knowledge exchange, alignment of objectives, and sharing of protocols, checklist terms, and outputs, and ultimately enhance interoperability and impact across initiatives. 
To support consistent collaboration, we hold working group meetings on the 2nd and 4th Tuesdays of each month, immediately before the GSC CIG meetings. This schedule provides a clear pathway for raising relevant topics and obtaining timely input from the wider GSC community.

## Do you already have a website or do you wish to create a homepage for the project in the GSC website?
 - eDNA project under GSC github: https://github.com/orgs/GenomicsStandardsConsortium/projects/7
 - I wish to use the GSC github as a discussion/communication platform for this project. 

## What other resources might you like from what the GSC can offer (mailing lists, etc)?
I would like to engage with GSC members to benefit from their expertise in data standards, receiving guidance and feedback through GitHub discussions, the GSC mailing list, and regular meetings. Additionally, access to relevant GSC resources, such as documentation and connections to related initiatives, would further support the development and adoption of eDNA data standards.

## What kind of timeline are you working to for building consensus, releasing a first version etc
Our goal is to develop and integrate the first version of an eDNA package or checklist within the MIxS suite by the end of 2025. 

## How is this work currently funded (list grants, funders, in kind contributions, etc)?
This is a community-driven project, with the members offering their time and efforts to achieve the project goals. The resources (i.e., time) of each member were provided by each of the roll’s funding bodies. Significant time will be supplied from the core members, and their times/roles are funded by their organisations/grants/funders.

## What resources will be required for completion?
Expertise insights/time from the team members and GSC members to guide the project and develop an eDNA checklist within the MIxS suite.

## What are your current plans for publishing/promoting the project?
We plan to publish an article to introduce the MIxS - eDNA package once established (aiming at early 2026). We plan to actively promote the project through conferences and workshops throughout 2025 and beyond.  

## References or relevant websites (for further reading)
 - TDWG eDNA Task Group: https://www.tdwg.org/community/gbwg/enviro/
 - FAIRe website: https://fair-edna.github.io/index.html 
 - Manuscript, accepted, currently in production: Takahashi, M., Frøslev, T.G., Paupério, J., Thalinger, B., Klymus, K., Helbing, C.C., Villacorta-Rath, C., Silliman, K., Thompson, L.R., Jungbluth, S.P., Yong, S.Y., Formel, S., Jenkins, G., Laporte, M., Deagle, B., Rajbhandari, S., Jeppesen, T.S., Bissett, A., Jerde, G., Hahn, E., Schriml, L.M., Hunter, C., Newman, P., Woollard, P., Harper, L.R., Dunn, N., West, K., Haderlé, R., Wilkinson, S., Acharya-Patel, N., Lopez, M.L.D., Cochrane, G., Berry, O. (in press) A metadata checklist and data formatting guidelines to make eDNA FAIR (Findable, Accessible, Interoperable and Reusable). Environmental DNA. 
 - UN Ocean Decade: https://oceandecade.org/actions/making-environmental-dna-fair/

