{% include header.html %}

# MIxS Extensions (Packages)

The MIxS standard is designed as an overarching framework, in order to create a single entry point to all minimum information checklists and extensions from the GSC. MIxS includes the technology-specific checklists from the previous MIGS, MIMS and MIMARKS standards, provides a way of introducing additional checklists such as MISAG, MIMAG and MIUViG, and also allows annotation of sample data using extensions.Initially introduced as Environmental extensions to standardize sets of measurements and observations describing particular habitats that are applicable across all GSC checklists, we now refer to them as just extensions to allow for the future expansion into non-environmental extensions such as the ancient-DNA specific extension current under review.
With this modular and extensible framework, both horizontal (i.e new checklists) and vertical (i.e. new extensions) expansion is possible and encouraged.

![mixs_ext_graphic](/images/mixs_ext_graphic-1024x731.png)

For the complete documentation on each checklist, extension and individual term please visit the [MIxS repository content pages](https://genomicsstandardsconsortium.github.io/mixs/).

## Current Extensions (packages)
These are the current set of environmental packages:

 * [MIxS-agriculture](https://genomicsstandardsconsortium.github.io/mixs/Agriculture/) (J. P. Dundore-Arias, et.al 2020) 

 * [MIxS-air](https://genomicsstandardsconsortium.github.io/mixs/air) (Yilmaz et al. 2011) 
  
 * [MIxS-built environment](https://genomicsstandardsconsortium.github.io/mixs/BuiltEnvironment) (Glass et al. 2014)
 
 * [MIxS-food-animal and animal feed](https://genomicsstandardsconsortium.github.io/mixs/FoodAnimalAndAnimalFeed) 

 * [MIxS-food-farm environment](https://genomicsstandardsconsortium.github.io/mixs/FoodFarmEnvironment/) 

 * [MIxS-food-food production facility](https://genomicsstandardsconsortium.github.io/mixs/FoodFoodProductionFacility)

 * [MIxS-food-human foods](https://genomicsstandardsconsortium.github.io/mixs/FoodHumanFoods)
 
 * [MIxS-host-associated](https://genomicsstandardsconsortium.github.io/mixs/HostAssociated) (Yilmaz et al. 2011)

 * [MIxS-human-associated](https://genomicsstandardsconsortium.github.io/mixs/HumanAssociated) (Yilmaz et al. 2011) 

 * [MIxS-human-gut](https://genomicsstandardsconsortium.github.io/mixs/HumanGut) (Yilmaz et al. 2011) 

 * [MIxS-human-oral](https://genomicsstandardsconsortium.github.io/mixs/HumanOral) (Yilmaz et al. 2011) 

 * [MIxS-human-skin](https://genomicsstandardsconsortium.github.io/mixs/HumanSkin) (Yilmaz et al. 2011)

 * [MIxS-human-vaginal](https://genomicsstandardsconsortium.github.io/mixs/HumanVaginal) (Yilmaz et al. 2011) 

 * [MIxS-hydrocarbon resources-cores](https://genomicsstandardsconsortium.github.io/mixs/HydrocarbonResourcesCores) (Tsesmetzis et al. 2016)

 * [MIxS-hydrocarbon resources-fluids/swabs](https://genomicsstandardsconsortium.github.io/mixs/HydrocarbonResourcesFluidsSwabs) (Tsesmetzis et al. 2016) .

 * [MIxS-microbial mat/biofilm](https://genomicsstandardsconsortium.github.io/mixs/MicrobialMatBiofilm) (Yilmaz et al. 2011) 

 * [MIxS-miscellaneous natural or artificial environment](https://genomicsstandardsconsortium.github.io/mixs/MiscellaneousNaturalOrArtificialEnvironment) (Yilmaz et al. 2011) 

 * [MIxS-plant-associated](https://genomicsstandardsconsortium.github.io/mixs/PlantAssociated) (Yilmaz et.al. 2011)

 * [MIxS-sediment](https://genomicsstandardsconsortium.github.io/mixs/Sediment) (Yilmaz et.al. 2011)

 * [MIxS-soil](https://genomicsstandardsconsortium.github.io/mixs/Soil) (Yilmaz et.al. 2011) 

 * [MIxS-symbiont-associated](https://genomicsstandardsconsortium.github.io/mixs/SymbiontAssociated) (Jorge et al., 2022) 
 
 * [MIxS-wastewater/sludge](https://genomicsstandardsconsortium.github.io/mixs/WastewaterSludge) (Yilmaz et.al. 2011) 
 

 * [MIxS-water](https://genomicsstandardsconsortium.github.io/mixs/Water) (Yilmaz et.al. 2011) 


[Dundore-Arias, JP. et al., "Community-Driven Metadata Standards for Agricultural Microbiome Research" <i>Phytobiomes Journal</i> 2020 4:2, 115-121 DOI:10.1094/PBIOMES-09-19-0051-P](https://doi.org/10.1094/PBIOMES-09-19-0051-P).

[Glass, EM. et al,. "MIxS-BE: a MIxS extension defining a minimum information standard for sequence data from the built environment." <i>The ISME journal</i> vol. 8,1 (2014): 1-3. doi:10.1038/ismej.2013.176](https://doi.org/10.1038/ismej.2013.176)

[Jorge, F., et al. "MIxS-SA: a MIxS extension defining the minimum information standard for sequence data from symbiont-associated micro-organisms." <i>ISME COMMUN.</i> 2, 9 (2022). DOI:10.1038/s43705-022-00092-w](https://doi.org/10.1038/s43705-022-00092-w)

[Tsesmetzis N. et al., "MIxS-HCR: a MIxS extension defining a minimal information standard for sequence data from environments pertaining to hydrocarbon resources" <i>Stand Genomic Sci.</i> 2016 Oct 12;11:78. doi: 10.1186/s40793-016-0203-5. eCollection 2016.](https://doi.org/10.1186/s40793-016-0203-5)

[Yilmaz, P. et al., "Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications." <i>Nature biotechnology</i> vol. 29,5 (2011): 415-20. doi:10.1038/nbt.1823](https://doi.org/10.1038/nbt.1823).
 

<!--

### Agriculture

### Air

### Built environment

The Sloan Foundation has established the Microbiology of the Built Environment (BE) to uncover the complexity of microbial ecosystems of inside spaces. Bringing together researchers, architects and engineers, the Microbiome of the Built Environment Data Analysis Core (MoBeDAC, Folker Meyer PI) is developing and coordinating a cohesive representation of the microbial community in built environments. MoBeDAC ([http://mobedac.org/](http://mobedac.org/)) has established a working group, led by Elizabeth Glass and Lynn Schriml, to expand the GSC MIxS standard for microbial sequences collected from built environments.

The initial [BE package](http://gensc.org/projects/mixs-indoor-gsc-project/ "MIxS Indoor GSC Project") represents a minimal metadata description of the built environment to be collected and reported for each sequenced sample. The BE package includes core terms for surface material, humidity, temperature, moisture and occupancy type along with specific metadata terms describing the indoor air, building and sample properties. Samples collected, sequenced and annotated with MIxS-BE metadata from waste-water, air filters, air and surfaces of indoor spaces provides a rigorous and structured tool for analysis of microbial sequences and ecosystems of the indoor and outdoor environments.

The BE package is currently being expanded to comprise a full representation of all metadata describing the building, building materials, samples, building environment, and occupants.

<<<<<<< Updated upstream
=======
You can download the BE package separately **[here]() NB this link still needs to be added**.

>>>>>>> Stashed changes
**Contact**

*   Lynn Schriml (lschriml@som.umaryland.edu)
*   Elizabeth M. Glass (marland@mcs.anl.gov)

### Food-animal and animal feed

### Food-farm environment

### Food-food production facility

### Food-human foods

### Host-associated

### Human-associated

### Human-gut

### Human-oral

### Human-skin

### Human-vaginal

### Hydrocarbon resources-cores
*   Hydrocarbon resources package, Nicolas Tsesmetzis (nicolas.tsesmetzis@shell.com)

### Hydrocarbon resources-fluids/swabs

### Microbial mat/biofilm

### Miscellaneous natural or artificial environment

### Plant-associated

### Sediment

### Soil

### Symbiont-associated

### Wastewater/sludge

### Water
-->

## Planned Extensions

Four further extensions to MIxS checklists are currently under development:

*   Plant genomes, Ramona Walls (rlwalls2008@gmail.com)
*   Pathogen package, Lynn Schriml (lschriml@som.umaryland.edu)
*   Single cell genomes and genome assemblies from metagenomes, Tanja Woyke (twoyke@lbl.gov) and Susannah Tringe (SGTringe@lbl.gov)
*   [Urobiome](https://github.com/GenomicsStandardsConsortium/mixs/projects/4), Lisa Karstens.

## Developing Extensions

Proposing and developing new extensions is simple. Development of new extensions should be an open and iterative process engaging the GSC community, the GSCâ€™s MIxS developers, and finally stakeholders across your community.

We can recommend the following steps to anyone interested in developing new extensions:

1.  identify reported metadata from literature and/or sequence databases
2.  solicit community input
3.  formulate an initial checklist or package
4.  present initial list and rationale behind the extension at a GSC meeting. Alternatively, solicit feedback via our general mailing list or compliance and interoperability mailing list (gensc-cig\[at\]googlegroups.com)
5.  coordinate efforts with the MIxS working group to finalize metadata item descriptions, requirements, and other format issues
6.  procure final approval from GSC MIxS group leader

Once the above steps are completed, the new extension is integrated into our common MIxS database and maintained there. Furthermore, with a new release of MIxS checklists, the extension will also be included.




<!--
### Minimal Information about a Biosynthetic Gene cluster (MIBiG)

[MIBiG](http://gensc.org/projects/mibig/ "MIBiG") utilizes the environmental and ecological parameters from the MIxS standards, but extends them towards covering information on biosynthetic gene clusters. Information on, e.g., enzyme function, substrate specificities, functional subclusters, regulatory and transport systems, operon structure, chemical moieties of the end compound and its intermediates, biosynthetic precursor compounds, compound bioactivity and molecular targets and compound toxicity have been added to allow cross-linking the information to biochemistry, pharmaceutical properties, genomic structure and ecology. Using the already developed computational pipeline for analysis of biosynthetic gene clusters antiSMASH (http://antismash.secondarymetabolites.org/), which has quickly become a standard in the field, information on characterized biosynthetic gene clusters will be linked to the untapped wealth of thousands of unknown gene clusters that have recently been unearthed by massive genome sequencing efforts.

**Contact** -->


