{% include header.html %}


## Planned Extensions

Four further extensions to MIxS checklists are currently under development:

*   Plant genomes, Ramona Walls (rlwalls2008@gmail.com)
*   Pathogen package, Lynn Schriml (lschriml@som.umaryland.edu)
*   Single cell genomes and genome assemblies from metagenomes, Tanja Woyke (twoyke@lbl.gov) and Susannah Tringe (SGTringe@lbl.gov)
*   [Urobiome](https://github.com/GenomicsStandardsConsortium/mixs/projects/4), Lisa Karstens.

## Developing Extensions

Proposing and developing new extensions is simple. Development of new extensions should be an open and iterative process engaging the GSC community, the GSCâ€™s MIxS developers, and finally stakeholders across your community.

We can recommend the following steps to anyone interested in developing new extensions:

1.  identify reported metadata from literature and/or sequence databases
2.  solicit community input
3.  formulate an initial checklist or package
4.  present initial list and rationale behind the extension at a GSC meeting. Alternatively, solicit feedback via our general mailing list or compliance and interoperability mailing list (gensc-developers\[at\]lists.gensc.org)
5.  coordinate efforts with the MIxS working group to finalize metadata item descriptions, requirements, and other format issues
6.  procure final approval from GSC MIxS group leader

Once the above steps are completed, the new extension is integrated into our common MIxS database and maintained there. Furthermore, with a new release of MIxS checklists, the extension will also be included.





### Minimal Information about a Biosynthetic Gene cluster (MIBiG)

[MIBiG](http://gensc.org/projects/mibig/ "MIBiG") utilizes the environmental and ecological parameters from the MIxS standards, but extends them towards covering information on biosynthetic gene clusters. Information on, e.g., enzyme function, substrate specificities, functional subclusters, regulatory and transport systems, operon structure, chemical moieties of the end compound and its intermediates, biosynthetic precursor compounds, compound bioactivity and molecular targets and compound toxicity have been added to allow cross-linking the information to biochemistry, pharmaceutical properties, genomic structure and ecology. Using the already developed computational pipeline for analysis of biosynthetic gene clusters antiSMASH (http://antismash.secondarymetabolites.org/), which has quickly become a standard in the field, information on characterized biosynthetic gene clusters will be linked to the untapped wealth of thousands of unknown gene clusters that have recently been unearthed by massive genome sequencing efforts.

**Contact**

*   Marnix Medema (marnix.medema@wur.nl)
