{% include header.html %}
<table width="100%" border="0" cellspacing="0" cellpadding="0">

<tbody>

<tr>

<td width="1%" height="145" align="left" valign="top"></td>

<td width="50%">

![egenomics_cat_logo.jpg](/gensc.github.io/images/egenomics_cat_logo.jpg)

</td>

<td width="49%" align="right" valign="middle">

## eGenomics: Cataloguing our complete genome collection  

### 11 -13th Sept 2006

</td>

</tr>

</tbody>

</table>

* * *

### General information

Our complete genome collection, the result of a significant investment of public funds, is one of our most valuable biological resources and also one of the most complex. The value of these data lies in the ability to view, compare and contrast the entire genetic complement of a wide range of organisms. The complexity of these data stems not only from the vast number of biological features within these genomes (i.e. genes, promoters, binding sites etc) but also from the complexity of the evolutionary relationships and ecological lifestyles - factors that play a major role in shaping genome features and content - of the organisms to whom these genomes belong.  
The scientific community would benefit significantly from the establishment of a data standard to capture more of this complexity. This standard could provide electronic, machine-readable reports for submission to a public database at the time of publication of each genome (analogous to the submission of genome annotation files). In this way, the experts on each organism would be directly responsible for providing data to the wider community about the detailed features of the organism.  
Currently the difficulty of obtaining such metadata in a high quality and easily accessible format is a common bottleneck in large-scale computational studies and a significant source of difficulties in the process of scientific discovery. Likewise, merging ecological and environmental data with complete genome sequences opens up a new area of research possibilities. The creation of this genome catalogue would provide an extensive amount of novel data and a powerful new research tool.

This workshop is being organised by Dawn Field of the Centre for Ecology and Hydrology, Oxford and Tatiana Tatusov of the National Centre for Biotechnology Information.

This is the third meeting of the Genomics Standards Consortium. [http://gensc.sf.net](https://web.archive.org/web/20080518174941/http://gensc.sf.net/)

### Aim of the workshop

1.  raise awareness of existing projects aimed at merging metadata with genomic sequences within this community.
2.  explore how to harmonize efforts between existing and future projects.
3.  develop the “Minimal Information about a Genome Sequence” checklist
4.  define how the culture and associated physical infrastructure required by an international working group might evolve.
5.  evaluate awareness of how this project can proceed in an e-science arena.
6.  evaluate how webservices, (workflows and LSIDs) may play a role in the implementation of such a standard.

### Programme

<table border="0" cellpadding="0">

<tbody>

<tr>

<td width="75" valign="top"></td>

<td valign="top">

**Monday 11th September**

</td>

</tr>

<tr>

<td valign="top"> </td>

<td valign="top"> </td>

</tr>

<tr>

<td width="75" valign="top">

10:00

</td>

<td valign="top">

Registration/coffee

</td>

</tr>

<tr>

<td width="75" valign="top">

10:30

</td>

<td valign="top">

Welcome, background and goals of workshop (Dawn Field and Tatiana Tatusov, Oxford Centre for Ecology and Hydrology and the NCBI)

</td>

</tr>

<tr>

<td width="75" valign="top">

10:45

</td>

<td valign="top">

Introduction to The National Institute for Environmental E-Science (Stuart Ballard, NIEeS)

</td>

</tr>

<tr>

<td valign="top"> </td>

<td valign="top"> </td>

</tr>

<tr>

<td width="75" valign="top"></td>

<td valign="top">

**Session I**: **Overview of our current and future genome collection**

**Session Chair: Dave Ussery**

</td>

</tr>

<tr>

<td width="75" valign="top">

11:00

</td>

<td valign="top">

Exploring the Tree of Life through genomics (Sandie Baldauf, Univerity of York)

</td>

</tr>

<tr>

<td width="75" valign="top">

11:30

</td>

<td valign="top">

The promise of 454 Sequencing in Characterizing Natural Diversity (Rob Edwards, University of San Diego)

</td>

</tr>

<tr>

<td width="75" valign="top">

12:00

</td>

<td valign="top">

Applying Environmental Genomics to Microbial Ecology (George  Kowalchuk)

</td>

</tr>

<tr>

<td width="75" valign="top">

12:30

</td>

<td valign="top">

CAMERA, TBC

</td>

</tr>

<tr>

<td width="75" valign="top">

13:00

</td>

<td valign="top">

Lunch

</td>

</tr>

<tr>

<td valign="top"> </td>

<td valign="top"> </td>

</tr>

<tr>

<td width="75" valign="top"></td>

<td valign="top">

**Session II:** **Databases and Metadata capture and Exchange efforts**

**Session Chair: George Garrity**

</td>

</tr>

<tr>

<td valign="top">14:00</td>

<td valign="top">

INSDC Sequencing Project Metadata submission protocol: NCBI web service API (Tatiana Tatusova, NCBI)

</td>

</tr>

<tr>

<td width="75" valign="top">

14:20

</td>

<td valign="top">

The DDJB Perspective on our complete genome collection (Yoshino Tateno, DDBJ)

</td>

</tr>

<tr>

<td width="75" valign="top">

14:40

</td>

<td valign="top">

The IMG, IMGm, and GOLD V2.0 databases (Nikos Kyripides, JGI)

</td>

</tr>

<tr>

<td width="75" valign="top">

15:00

</td>

<td valign="top">

PUMA: from genomes to GFF: metadata exchange (Natalia Maltsev, Argonne National Laboratory)

</td>

</tr>

<tr>

<td width="75" valign="top">

15:20

</td>

<td valign="top">

Coffee

</td>

</tr>

<tr>

<td width="75" valign="top">

16:00

</td>

<td valign="top">

Evaluation of the “Minimum Information about a Genome Sequence” specification / checklist (Peter Sterk, EBI)

</td>

</tr>

<tr>

<td width="75" valign="top">

16:20

</td>

<td valign="top">

Evaluation of the checklist to XML schema transformation (Dawn Field, CEH Oxford)

</td>

</tr>

<tr>

<td width="75" valign="top">

16:40

</td>

<td valign="top">

Demonstration of the Genome Catalogue (Tanya Gray, CEH Oxford)

</td>

</tr>

<tr>

<td width="75" valign="top">

17:00

</td>

<td valign="top">

Close

</td>

</tr>

<tr>

<td valign="top"> </td>

<td valign="top">

Dinner at Robinson College

</td>

</tr>

<tr>

<td valign="top"> </td>

<td valign="top"> </td>

</tr>

<tr>

<td width="75" valign="top"></td>

<td valign="top">

**Tuesday 12th September**

</td>

</tr>

<tr>

<td valign="top"> </td>

<td valign="top"> </td>

</tr>

<tr>

<td width="75" valign="top"></td>

<td valign="top">

**Session III: Allied Projects: Ontology Development**

**Session Chair: Dawn Field**

</td>

</tr>

<tr>

<td width="75" valign="top">

09:00

</td>

<td valign="top">

Names for Life and Phenobank (George Garrity, U of Michigan)

</td>

</tr>

<tr>

<td width="75" valign="top">

09:20

</td>

<td valign="top">

MICheck : bringing the MIxxx community together (Chris Taylor, EBI)

</td>

</tr>

<tr>

<td width="75" valign="top">

09:40

</td>

<td valign="top">

An Overview of OBO and the OBO Foundry (Michael Ashburner)

</td>

</tr>

<tr>

<td width="75" valign="top">

10:00

</td>

<td valign="top">

An Overview of FuGO and its relationship to the GSC (Trish Whetzel, U Penn)

</td>

</tr>

<tr>

<td width="75" valign="top">

10:20

</td>

<td valign="top">

PATO – towards a description of organismal phenotypes (Suzi Lewis, Berkeley)

</td>

</tr>

<tr>

<td width="75" valign="top">

10:40

</td>

<td valign="top">

MegX and the capture of environmental metadata (Frank-Oliver Glockner, Max Planck Institute for Marine Microbes)

</td>

</tr>

<tr>

<td width="75" valign="top">

11:00

</td>

<td valign="top">

Coffee

</td>

</tr>

<tr>

<td width="75" valign="top"></td>

<td valign="top"></td>

</tr>

<tr>

<td width="75" valign="top"></td>

<td valign="top">

**Session IV**: **Group work on the MIGS specification, genome reports and the Genome Catalogue**

</td>

</tr>

<tr>

<td width="75" valign="top">

11:30

</td>

<td valign="top">

Open discussion – the MIGS specification

</td>

</tr>

<tr>

<td width="75" valign="top">

13:00

</td>

<td valign="top">

Lunch

</td>

</tr>

<tr>

<td width="75" valign="top">

14:00

</td>

<td valign="top">

Open discussion  - the Genome Catalogue

</td>

</tr>

<tr>

<td width="75" valign="top">

15:00

</td>

<td valign="top">

Coffee

</td>

</tr>

<tr>

<td width="75" valign="top">

15:30

</td>

<td valign="top">

Open discussion – capture of terms

</td>

</tr>

<tr>

<td width="75" valign="top">

17:00

</td>

<td valign="top">

Close

</td>

</tr>

<tr>

<td valign="top"> </td>

<td>

Dinner at Robinson College

</td>

</tr>

<tr>

<td width="75" valign="top"></td>

<td></td>

</tr>

<tr>

<td width="75" valign="top"></td>

<td valign="top">

**Wednesday 13th September**

</td>

</tr>

<tr>

<td valign="top"> </td>

<td valign="top"> </td>

</tr>

<tr>

<td width="75" valign="top"></td>

<td valign="top">

**Session V: Discussion**

</td>

</tr>

<tr>

<td width="75" valign="top">

09:00

</td>

<td valign="top">

INDSC Panel Discussion

</td>

</tr>

<tr>

<td width="75" valign="top">

10:30

</td>

<td valign="top">

Coffee

</td>

</tr>

<tr>

<td width="75" valign="top">

11:00

</td>

<td valign="top">

The GSC road map (working group membership, actions, future meetings)

</td>

</tr>

<tr>

<td width="75" valign="top">

13:00

</td>

<td valign="top">

Lunch

</td>

</tr>

<tr>

<td width="75" valign="top">

14:00

</td>

<td valign="top">

Wrap up

</td>

</tr>

<tr>

<td width="75" valign="top">

15:00

</td>

<td valign="top">

Formal close of workshop and Coffee

</td>

</tr>

<tr>

<td width="75" valign="top">

15:30

</td>

<td valign="top">

Departures

</td>

</tr>

</tbody>

</table>

### Location

The workshop, accommodation, and workshop dinners will all be in [Robinson College](https://web.archive.org/web/20080518174941/http://www.robinson.cam.ac.uk/), Cambridge. Session IV, on Tuesday afternoon will held in Titan 1 computer room.  
A google map for this event is provided [here](egenomic2006_map.html)

### Follow-on

It is anticipated that the workshop may generate interest in some follow-on activities. We will discuss the sort of follow-on activities that can be arranged, such as visits to NIEeS to demonstrate some of the escience/grid technologies and to provide help with installation and usage.

### Registration

<table width="100%" border="0" cellpadding="0" cellspacing="0">

<tbody>

<tr>

<td valign="top">

Registration has now closed.

Registration is free. Registration for UK academic participants includes accommodation in College and meals. The NIEeS will also refund sensible travel and accommodation expenses within the UK for UK academic particpants.

</td>

<td valign="top" width="160"></td>

</tr>

</tbody>

</table>

### Contact and further information

For information about registration, and all other types of queries, please contact the NIEeS office in the first instance.

<table border="0" cellpadding="0" cellspacing="0" width="100%">

<tbody>

<tr height="22" valign="middle">

<td bgcolor="#8b008b" height="22"></td>

<td bgcolor="#8b008b" height="22" valign="middle">

<div align="right"><span class="lastupdate">This document last modified: Thursday, 07-Sep-2006 09:56:22 BST</span></div>

</td>

</tr>

</tbody>

</table>
