
# Mixs6 schema


MIxS 6 linkml rendering


### Classes

 * [Air](Air.md) - air
     * [AirME](AirME.md) - Combinatorial checklist Metagenome or Environmental with environmental package air
     * [AirMIGSBacteria](AirMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package air
     * [AirMIGSEukaryote](AirMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package air
     * [AirMIGSOrg](AirMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package air
     * [AirMIGSPlant](AirMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package air
     * [AirMIGSVirus](AirMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package air
     * [AirMIMAG](AirMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package air
     * [AirMIMARKSSpecimen](AirMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package air
     * [AirMIMARKSSurvey](AirMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package air
     * [AirMISAG](AirMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package air
     * [AirMIUVIG](AirMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package air
 * [BuiltEnvironment](BuiltEnvironment.md) - built environment
     * [BuiltEnvironmentME](BuiltEnvironmentME.md) - Combinatorial checklist Metagenome or Environmental with environmental package built environment
     * [BuiltEnvironmentMIGSBacteria](BuiltEnvironmentMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package built environment
     * [BuiltEnvironmentMIGSEukaryote](BuiltEnvironmentMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package built environment
     * [BuiltEnvironmentMIGSOrg](BuiltEnvironmentMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package built environment
     * [BuiltEnvironmentMIGSPlant](BuiltEnvironmentMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package built environment
     * [BuiltEnvironmentMIGSVirus](BuiltEnvironmentMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package built environment
     * [BuiltEnvironmentMIMAG](BuiltEnvironmentMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package built environment
     * [BuiltEnvironmentMIMARKSSpecimen](BuiltEnvironmentMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package built environment
     * [BuiltEnvironmentMIMARKSSurvey](BuiltEnvironmentMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package built environment
     * [BuiltEnvironmentMISAG](BuiltEnvironmentMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package built environment
     * [BuiltEnvironmentMIUVIG](BuiltEnvironmentMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package built environment
 * [Core](Core.md) - Core package. Do not use this directly, this is used to build other packages
 * [Host-associated](Host-associated.md) - host-associated
     * [Host-associatedME](Host-associatedME.md) - Combinatorial checklist Metagenome or Environmental with environmental package host-associated
     * [Host-associatedMIGSBacteria](Host-associatedMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package host-associated
     * [Host-associatedMIGSEukaryote](Host-associatedMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package host-associated
     * [Host-associatedMIGSOrg](Host-associatedMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package host-associated
     * [Host-associatedMIGSPlant](Host-associatedMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package host-associated
     * [Host-associatedMIGSVirus](Host-associatedMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package host-associated
     * [Host-associatedMIMAG](Host-associatedMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package host-associated
     * [Host-associatedMIMARKSSpecimen](Host-associatedMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package host-associated
     * [Host-associatedMIMARKSSurvey](Host-associatedMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package host-associated
     * [Host-associatedMISAG](Host-associatedMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package host-associated
     * [Host-associatedMIUVIG](Host-associatedMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package host-associated
 * [Human-associated](Human-associated.md) - human-associated
     * [Human-associatedME](Human-associatedME.md) - Combinatorial checklist Metagenome or Environmental with environmental package human-associated
     * [Human-associatedMIGSBacteria](Human-associatedMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-associated
     * [Human-associatedMIGSEukaryote](Human-associatedMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package human-associated
     * [Human-associatedMIGSOrg](Human-associatedMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package human-associated
     * [Human-associatedMIGSPlant](Human-associatedMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package human-associated
     * [Human-associatedMIGSVirus](Human-associatedMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-associated
     * [Human-associatedMIMAG](Human-associatedMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package human-associated
     * [Human-associatedMIMARKSSpecimen](Human-associatedMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package human-associated
     * [Human-associatedMIMARKSSurvey](Human-associatedMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package human-associated
     * [Human-associatedMISAG](Human-associatedMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package human-associated
     * [Human-associatedMIUVIG](Human-associatedMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package human-associated
 * [Human-gut](Human-gut.md) - human-gut
     * [Human-gutME](Human-gutME.md) - Combinatorial checklist Metagenome or Environmental with environmental package human-gut
     * [Human-gutMIGSBacteria](Human-gutMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-gut
     * [Human-gutMIGSEukaryote](Human-gutMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package human-gut
     * [Human-gutMIGSOrg](Human-gutMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package human-gut
     * [Human-gutMIGSPlant](Human-gutMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package human-gut
     * [Human-gutMIGSVirus](Human-gutMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-gut
     * [Human-gutMIMAG](Human-gutMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package human-gut
     * [Human-gutMIMARKSSpecimen](Human-gutMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package human-gut
     * [Human-gutMIMARKSSurvey](Human-gutMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package human-gut
     * [Human-gutMISAG](Human-gutMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package human-gut
     * [Human-gutMIUVIG](Human-gutMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package human-gut
 * [Human-oral](Human-oral.md) - human-oral
     * [Human-oralME](Human-oralME.md) - Combinatorial checklist Metagenome or Environmental with environmental package human-oral
     * [Human-oralMIGSBacteria](Human-oralMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-oral
     * [Human-oralMIGSEukaryote](Human-oralMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package human-oral
     * [Human-oralMIGSOrg](Human-oralMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package human-oral
     * [Human-oralMIGSPlant](Human-oralMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package human-oral
     * [Human-oralMIGSVirus](Human-oralMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-oral
     * [Human-oralMIMAG](Human-oralMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package human-oral
     * [Human-oralMIMARKSSpecimen](Human-oralMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package human-oral
     * [Human-oralMIMARKSSurvey](Human-oralMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package human-oral
     * [Human-oralMISAG](Human-oralMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package human-oral
     * [Human-oralMIUVIG](Human-oralMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package human-oral
 * [Human-skin](Human-skin.md) - human-skin
     * [Human-skinME](Human-skinME.md) - Combinatorial checklist Metagenome or Environmental with environmental package human-skin
     * [Human-skinMIGSBacteria](Human-skinMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-skin
     * [Human-skinMIGSEukaryote](Human-skinMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package human-skin
     * [Human-skinMIGSOrg](Human-skinMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package human-skin
     * [Human-skinMIGSPlant](Human-skinMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package human-skin
     * [Human-skinMIGSVirus](Human-skinMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-skin
     * [Human-skinMIMAG](Human-skinMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package human-skin
     * [Human-skinMIMARKSSpecimen](Human-skinMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package human-skin
     * [Human-skinMIMARKSSurvey](Human-skinMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package human-skin
     * [Human-skinMISAG](Human-skinMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package human-skin
     * [Human-skinMIUVIG](Human-skinMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package human-skin
 * [Human-vaginal](Human-vaginal.md) - human-vaginal
     * [Human-vaginalME](Human-vaginalME.md) - Combinatorial checklist Metagenome or Environmental with environmental package human-vaginal
     * [Human-vaginalMIGSBacteria](Human-vaginalMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-vaginal
     * [Human-vaginalMIGSEukaryote](Human-vaginalMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package human-vaginal
     * [Human-vaginalMIGSOrg](Human-vaginalMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package human-vaginal
     * [Human-vaginalMIGSPlant](Human-vaginalMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package human-vaginal
     * [Human-vaginalMIGSVirus](Human-vaginalMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-vaginal
     * [Human-vaginalMIMAG](Human-vaginalMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package human-vaginal
     * [Human-vaginalMIMARKSSpecimen](Human-vaginalMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package human-vaginal
     * [Human-vaginalMIMARKSSurvey](Human-vaginalMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package human-vaginal
     * [Human-vaginalMISAG](Human-vaginalMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package human-vaginal
     * [Human-vaginalMIUVIG](Human-vaginalMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package human-vaginal
 * [HydrocarbonResources-cores](HydrocarbonResources-cores.md) - hydrocarbon resources-cores
     * [HydrocarbonResources-coresME](HydrocarbonResources-coresME.md) - Combinatorial checklist Metagenome or Environmental with environmental package hydrocarbon resources-cores
     * [HydrocarbonResources-coresMIGSBacteria](HydrocarbonResources-coresMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package hydrocarbon resources-cores
     * [HydrocarbonResources-coresMIGSEukaryote](HydrocarbonResources-coresMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package hydrocarbon resources-cores
     * [HydrocarbonResources-coresMIGSOrg](HydrocarbonResources-coresMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package hydrocarbon resources-cores
     * [HydrocarbonResources-coresMIGSPlant](HydrocarbonResources-coresMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package hydrocarbon resources-cores
     * [HydrocarbonResources-coresMIGSVirus](HydrocarbonResources-coresMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package hydrocarbon resources-cores
     * [HydrocarbonResources-coresMIMAG](HydrocarbonResources-coresMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package hydrocarbon resources-cores
     * [HydrocarbonResources-coresMIMARKSSpecimen](HydrocarbonResources-coresMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package hydrocarbon resources-cores
     * [HydrocarbonResources-coresMIMARKSSurvey](HydrocarbonResources-coresMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package hydrocarbon resources-cores
     * [HydrocarbonResources-coresMISAG](HydrocarbonResources-coresMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package hydrocarbon resources-cores
     * [HydrocarbonResources-coresMIUVIG](HydrocarbonResources-coresMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package hydrocarbon resources-cores
 * [HydrocarbonResources-fluidsSwabs](HydrocarbonResources-fluidsSwabs.md) - hydrocarbon resources-fluids/swabs
     * [HydrocarbonResources-fluidsSwabsME](HydrocarbonResources-fluidsSwabsME.md) - Combinatorial checklist Metagenome or Environmental with environmental package hydrocarbon resources-fluids_swabs
     * [HydrocarbonResources-fluidsSwabsMIGSBacteria](HydrocarbonResources-fluidsSwabsMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package hydrocarbon resources-fluids_swabs
     * [HydrocarbonResources-fluidsSwabsMIGSEukaryote](HydrocarbonResources-fluidsSwabsMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package hydrocarbon resources-fluids_swabs
     * [HydrocarbonResources-fluidsSwabsMIGSOrg](HydrocarbonResources-fluidsSwabsMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package hydrocarbon resources-fluids_swabs
     * [HydrocarbonResources-fluidsSwabsMIGSPlant](HydrocarbonResources-fluidsSwabsMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package hydrocarbon resources-fluids_swabs
     * [HydrocarbonResources-fluidsSwabsMIGSVirus](HydrocarbonResources-fluidsSwabsMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package hydrocarbon resources-fluids_swabs
     * [HydrocarbonResources-fluidsSwabsMIMAG](HydrocarbonResources-fluidsSwabsMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package hydrocarbon resources-fluids_swabs
     * [HydrocarbonResources-fluidsSwabsMIMARKSSpecimen](HydrocarbonResources-fluidsSwabsMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package hydrocarbon resources-fluids_swabs
     * [HydrocarbonResources-fluidsSwabsMIMARKSSurvey](HydrocarbonResources-fluidsSwabsMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package hydrocarbon resources-fluids_swabs
     * [HydrocarbonResources-fluidsSwabsMISAG](HydrocarbonResources-fluidsSwabsMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package hydrocarbon resources-fluids_swabs
     * [HydrocarbonResources-fluidsSwabsMIUVIG](HydrocarbonResources-fluidsSwabsMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package hydrocarbon resources-fluids_swabs
 * [MicrobialMatBiofilm](MicrobialMatBiofilm.md) - microbial mat/biofilm
     * [MicrobialMatBiofilmME](MicrobialMatBiofilmME.md) - Combinatorial checklist Metagenome or Environmental with environmental package microbial mat_biofilm
     * [MicrobialMatBiofilmMIGSBacteria](MicrobialMatBiofilmMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package microbial mat_biofilm
     * [MicrobialMatBiofilmMIGSEukaryote](MicrobialMatBiofilmMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package microbial mat_biofilm
     * [MicrobialMatBiofilmMIGSOrg](MicrobialMatBiofilmMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package microbial mat_biofilm
     * [MicrobialMatBiofilmMIGSPlant](MicrobialMatBiofilmMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package microbial mat_biofilm
     * [MicrobialMatBiofilmMIGSVirus](MicrobialMatBiofilmMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package microbial mat_biofilm
     * [MicrobialMatBiofilmMIMAG](MicrobialMatBiofilmMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package microbial mat_biofilm
     * [MicrobialMatBiofilmMIMARKSSpecimen](MicrobialMatBiofilmMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package microbial mat_biofilm
     * [MicrobialMatBiofilmMIMARKSSurvey](MicrobialMatBiofilmMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package microbial mat_biofilm
     * [MicrobialMatBiofilmMISAG](MicrobialMatBiofilmMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package microbial mat_biofilm
     * [MicrobialMatBiofilmMIUVIG](MicrobialMatBiofilmMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package microbial mat_biofilm
 * [MiscellaneousNaturalOrArtificialEnvironment](MiscellaneousNaturalOrArtificialEnvironment.md) - miscellaneous natural or artificial environment
     * [MiscellaneousNaturalOrArtificialEnvironmentME](MiscellaneousNaturalOrArtificialEnvironmentME.md) - Combinatorial checklist Metagenome or Environmental with environmental package miscellaneous natural or artificial environment
     * [MiscellaneousNaturalOrArtificialEnvironmentMIGSBacteria](MiscellaneousNaturalOrArtificialEnvironmentMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package miscellaneous natural or artificial environment
     * [MiscellaneousNaturalOrArtificialEnvironmentMIGSEukaryote](MiscellaneousNaturalOrArtificialEnvironmentMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package miscellaneous natural or artificial environment
     * [MiscellaneousNaturalOrArtificialEnvironmentMIGSOrg](MiscellaneousNaturalOrArtificialEnvironmentMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package miscellaneous natural or artificial environment
     * [MiscellaneousNaturalOrArtificialEnvironmentMIGSPlant](MiscellaneousNaturalOrArtificialEnvironmentMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package miscellaneous natural or artificial environment
     * [MiscellaneousNaturalOrArtificialEnvironmentMIGSVirus](MiscellaneousNaturalOrArtificialEnvironmentMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package miscellaneous natural or artificial environment
     * [MiscellaneousNaturalOrArtificialEnvironmentMIMAG](MiscellaneousNaturalOrArtificialEnvironmentMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package miscellaneous natural or artificial environment
     * [MiscellaneousNaturalOrArtificialEnvironmentMIMARKSSpecimen](MiscellaneousNaturalOrArtificialEnvironmentMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package miscellaneous natural or artificial environment
     * [MiscellaneousNaturalOrArtificialEnvironmentMIMARKSSurvey](MiscellaneousNaturalOrArtificialEnvironmentMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package miscellaneous natural or artificial environment
     * [MiscellaneousNaturalOrArtificialEnvironmentMISAG](MiscellaneousNaturalOrArtificialEnvironmentMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package miscellaneous natural or artificial environment
     * [MiscellaneousNaturalOrArtificialEnvironmentMIUVIG](MiscellaneousNaturalOrArtificialEnvironmentMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package miscellaneous natural or artificial environment
 * [Plant-associated](Plant-associated.md) - plant-associated
     * [Plant-associatedME](Plant-associatedME.md) - Combinatorial checklist Metagenome or Environmental with environmental package plant-associated
     * [Plant-associatedMIGSBacteria](Plant-associatedMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package plant-associated
     * [Plant-associatedMIGSEukaryote](Plant-associatedMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package plant-associated
     * [Plant-associatedMIGSOrg](Plant-associatedMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package plant-associated
     * [Plant-associatedMIGSPlant](Plant-associatedMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package plant-associated
     * [Plant-associatedMIGSVirus](Plant-associatedMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package plant-associated
     * [Plant-associatedMIMAG](Plant-associatedMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package plant-associated
     * [Plant-associatedMIMARKSSpecimen](Plant-associatedMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package plant-associated
     * [Plant-associatedMIMARKSSurvey](Plant-associatedMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package plant-associated
     * [Plant-associatedMISAG](Plant-associatedMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package plant-associated
     * [Plant-associatedMIUVIG](Plant-associatedMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package plant-associated
 * [QuantityValue](QuantityValue.md) - used to record a measurement
 * [Sediment](Sediment.md) - sediment
     * [SedimentME](SedimentME.md) - Combinatorial checklist Metagenome or Environmental with environmental package sediment
     * [SedimentMIGSBacteria](SedimentMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package sediment
     * [SedimentMIGSEukaryote](SedimentMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package sediment
     * [SedimentMIGSOrg](SedimentMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package sediment
     * [SedimentMIGSPlant](SedimentMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package sediment
     * [SedimentMIGSVirus](SedimentMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package sediment
     * [SedimentMIMAG](SedimentMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package sediment
     * [SedimentMIMARKSSpecimen](SedimentMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package sediment
     * [SedimentMIMARKSSurvey](SedimentMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package sediment
     * [SedimentMISAG](SedimentMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package sediment
     * [SedimentMIUVIG](SedimentMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package sediment
 * [Soil](Soil.md) - soil
     * [SoilME](SoilME.md) - Combinatorial checklist Metagenome or Environmental with environmental package soil
     * [SoilMIGSBacteria](SoilMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package soil
     * [SoilMIGSEukaryote](SoilMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package soil
     * [SoilMIGSOrg](SoilMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package soil
     * [SoilMIGSPlant](SoilMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package soil
     * [SoilMIGSVirus](SoilMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package soil
     * [SoilMIMAG](SoilMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package soil
     * [SoilMIMARKSSpecimen](SoilMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package soil
     * [SoilMIMARKSSurvey](SoilMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package soil
     * [SoilMISAG](SoilMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package soil
     * [SoilMIUVIG](SoilMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package soil
 * [WastewaterSludge](WastewaterSludge.md) - wastewater/sludge
     * [WastewaterSludgeME](WastewaterSludgeME.md) - Combinatorial checklist Metagenome or Environmental with environmental package wastewater_sludge
     * [WastewaterSludgeMIGSBacteria](WastewaterSludgeMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package wastewater_sludge
     * [WastewaterSludgeMIGSEukaryote](WastewaterSludgeMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package wastewater_sludge
     * [WastewaterSludgeMIGSOrg](WastewaterSludgeMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package wastewater_sludge
     * [WastewaterSludgeMIGSPlant](WastewaterSludgeMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package wastewater_sludge
     * [WastewaterSludgeMIGSVirus](WastewaterSludgeMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package wastewater_sludge
     * [WastewaterSludgeMIMAG](WastewaterSludgeMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package wastewater_sludge
     * [WastewaterSludgeMIMARKSSpecimen](WastewaterSludgeMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package wastewater_sludge
     * [WastewaterSludgeMIMARKSSurvey](WastewaterSludgeMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package wastewater_sludge
     * [WastewaterSludgeMISAG](WastewaterSludgeMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package wastewater_sludge
     * [WastewaterSludgeMIUVIG](WastewaterSludgeMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package wastewater_sludge
 * [Water](Water.md) - water
     * [WaterME](WaterME.md) - Combinatorial checklist Metagenome or Environmental with environmental package water
     * [WaterMIGSBacteria](WaterMIGSBacteria.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package water
     * [WaterMIGSEukaryote](WaterMIGSEukaryote.md) - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package water
     * [WaterMIGSOrg](WaterMIGSOrg.md) - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package water
     * [WaterMIGSPlant](WaterMIGSPlant.md) - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package water
     * [WaterMIGSVirus](WaterMIGSVirus.md) - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package water
     * [WaterMIMAG](WaterMIMAG.md) - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package water
     * [WaterMIMARKSSpecimen](WaterMIMARKSSpecimen.md) - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package water
     * [WaterMIMARKSSurvey](WaterMIMARKSSurvey.md) - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package water
     * [WaterMISAG](WaterMISAG.md) - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package water
     * [WaterMIUVIG](WaterMIUVIG.md) - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package water

### Mixins

 * [ME](ME.md) - Metagenome or Environmental
 * [MIGSBacteria](MIGSBacteria.md) - Minimal Information about a Genome Sequence: cultured bacteria/archaea
 * [MIGSEukaryote](MIGSEukaryote.md) - Minimal Information about a Genome Sequence: eukaryote
 * [MIGSOrg](MIGSOrg.md) - Minimal Information about a Genome Sequence: org
 * [MIGSPlant](MIGSPlant.md) - Minimal Information about a Genome Sequence: plant
 * [MIGSVirus](MIGSVirus.md) - Minimal Information about a Genome Sequence: cultured bacteria/archaea
 * [MIMAG](MIMAG.md) - Minimum Information About a Metagenome-Assembled Genome
 * [MIMARKSSpecimen](MIMARKSSpecimen.md) - Minimal Information about a Marker Specimen: specimen
 * [MIMARKSSurvey](MIMARKSSurvey.md) - Minimal Information about a Marker Specimen: survey
 * [MISAG](MISAG.md) - Minimum Information About a Single Amplified Genome
 * [MIUVIG](MIUVIG.md) - Minimum Information About an Uncultivated Virus Genome

### Slots

 * [core field](core_field.md) - basic fields
     * [sample_name](sample_name.md) - Sample Name is a name that you choose for the sample. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. Every Sample Name from a single Submitter must be unique. 
         * [ME➞sample_name](ME_sample_name.md)
         * [MIGS bacteria➞sample_name](MIGS_bacteria_sample_name.md)
         * [MIGS eukaryote➞sample_name](MIGS_eukaryote_sample_name.md)
         * [MIGS org➞sample_name](MIGS_org_sample_name.md)
         * [MIGS plant➞sample_name](MIGS_plant_sample_name.md)
         * [MIGS virus➞sample_name](MIGS_virus_sample_name.md)
         * [MIMAG➞sample_name](MIMAG_sample_name.md)
         * [MIMARKS specimen➞sample_name](MIMARKS_specimen_sample_name.md)
         * [MIMARKS survey➞sample_name](MIMARKS_survey_sample_name.md)
         * [MISAG➞sample_name](MISAG_sample_name.md)
         * [MIUVIG➞sample_name](MIUVIG_sample_name.md)
 * [environment field](environment_field.md) - field describing environmental aspect of a sample
     * [abs_air_humidity](abs_air_humidity.md) - Actual mass of water vapor - mh20 - present in the air water vapor mixture
     * [add_recov_method](add_recov_method.md) - Additional (i.e. Secondary, tertiary, etc.) recovery methods deployed for increase of hydrocarbon recovery from resource and start date for each one of them. If "other" is specified, please propose entry in "additional info" field
     * [additional_info](additional_info.md) - Information that doesn't fit anywhere else. Can also be used to propose new entries for fields with controlled vocabulary
     * [address](address.md) - The street name and building number where the sampling occurred.
     * [adj_room](adj_room.md) - List of rooms (room number, room name) immediately adjacent to the sampling room
     * [aero_struc](aero_struc.md) - Aerospace structures typically consist of thin plates with stiffeners for the external surfaces, bulkheads and frames to support the shape and fasteners such as welds, rivets, screws and bolts to hold the components together
     * [agrochem_addition](agrochem_addition.md) - Addition of fertilizers, pesticides, etc. - amount and time of applications
     * [air_temp](air_temp.md) - Temperature of the air at the time of sampling
     * [air_temp_regm](air_temp_regm.md) - Information about treatment involving an exposure to varying temperatures; should include the temperature, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include different temperature regimens
     * [al_sat](al_sat.md) - Aluminum saturation (esp. For tropical soils)
     * [al_sat_meth](al_sat_meth.md) - Reference or method used in determining Al saturation
     * [alkalinity](alkalinity.md) - Alkalinity, the ability of a solution to neutralize acids to the equivalence point of carbonate or bicarbonate
     * [alkalinity_method](alkalinity_method.md) - Method used for alkalinity measurement
     * [alkyl_diethers](alkyl_diethers.md) - Concentration of alkyl diethers
     * [alt](alt.md) - Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth‚Äôs surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air
     * [aminopept_act](aminopept_act.md) - Measurement of aminopeptidase activity
     * [ammonium](ammonium.md) - Concentration of ammonium in the sample
     * [amniotic_fluid_color](amniotic_fluid_color.md) - Specification of the color of the amniotic fluid sample
     * [amount_light](amount_light.md) - The unit of illuminance and luminous emittance, measuring luminous flux per unit area
     * [ances_data](ances_data.md) - Information about either pedigree or other ancestral information description (e.g. parental variety in case of mutant or selection), e.g. A/3*B (meaning [(A x B) x B] x B)
     * [annual_precpt](annual_precpt.md) - The average of all annual precipitation values known, or an estimated equivalent value derived by such methods as regional indexes or Isohyetal maps.
     * [annual_temp](annual_temp.md) - Mean annual temperature
     * [antibiotic_regm](antibiotic_regm.md) - Information about treatment involving antibiotic administration; should include the name of antibiotic, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple antibiotic regimens
     * [api](api.md) - API gravity is a measure of how heavy or light a petroleum liquid is compared to water (source: https://en.wikipedia.org/wiki/API_gravity) (e.g. 31.1¬∞ API)
     * [arch_struc](arch_struc.md) - An architectural structure is a human-made, free-standing, immobile outdoor construction
     * [aromatics_pc](aromatics_pc.md) - Saturate, Aromatic, Resin and Asphaltene¬†(SARA) is an analysis method that divides¬†crude oil¬†components according to their polarizability and polarity. There are three main methods to obtain SARA results. The most popular one is known as the Iatroscan TLC-FID and is referred to as IP-143 (source: https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene)
     * [asphaltenes_pc](asphaltenes_pc.md) - Saturate, Aromatic, Resin and Asphaltene¬†(SARA) is an analysis method that divides¬†crude oil¬†components according to their polarizability and polarity. There are three main methods to obtain SARA results. The most popular one is known as the Iatroscan TLC-FID and is referred to as IP-143 (source: https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene)
     * [atmospheric_data](atmospheric_data.md) - Measurement of atmospheric data; can include multiple data
     * [avg_dew_point](avg_dew_point.md) - The average of dew point measures taken at the beginning of every hour over a 24 hour period on the sampling day
     * [avg_occup](avg_occup.md) - Daily average occupancy of room
     * [avg_temp](avg_temp.md) - The average of temperatures taken at the beginning of every hour over a 24 hour period on the sampling day
     * [bac_prod](bac_prod.md) - Bacterial production in the water column measured by isotope uptake
     * [bac_resp](bac_resp.md) - Measurement of bacterial respiration in the water column
     * [bacteria_carb_prod](bacteria_carb_prod.md) - Measurement of bacterial carbon production
     * [barometric_press](barometric_press.md) - Force per unit area exerted against a surface by the weight of air above that surface
     * [basin](basin.md) - Name of the basin (e.g. Campos)
     * [bathroom_count](bathroom_count.md) - The number of bathrooms in the building
     * [bedroom_count](bedroom_count.md) - The number of bedrooms in the building
     * [benzene](benzene.md) - Concentration of benzene in the sample
     * [biochem_oxygen_dem](biochem_oxygen_dem.md) - Amount of dissolved oxygen needed by aerobic biological organisms in a body of water to break down organic material present in a given water sample at certain temperature over a specific time period
     * [biocide](biocide.md) - List of biocides (commercial name of product and supplier) and date of administration
     * [biocide_admin_method](biocide_admin_method.md) - Method of biocide administration (dose, frequency, duration, time elapsed between last biociding and sampling) (e.g. 150 mg/l; weekly; 4 hr; 3 days)
     * [biol_stat](biol_stat.md) - The level of genome modification
     * [biomass](biomass.md) - Amount of biomass; should include the name for the part of biomass measured, e.g. Microbial, total. Can include multiple measurements
     * [biotic_regm](biotic_regm.md) - Information about treatment(s) involving use of biotic factors, such as bacteria, viruses or fungi.
     * [birth_control](birth_control.md) - Specification of birth control medication used
     * [bishomohopanol](bishomohopanol.md) - Concentration of bishomohopanol
     * [blood_blood_disord](blood_blood_disord.md) - History of blood disorders; can include multiple disorders.  The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, hematopoietic system disease (https://disease-ontology.org/?id=DOID:74).
     * [bromide](bromide.md) - Concentration of bromide
     * [build_docs](build_docs.md) - The building design, construction and operation documents
     * [build_occup_type](build_occup_type.md) - The primary function for which a building or discrete part of a building is intended to be used
     * [building_setting](building_setting.md) - A location (geography) where a building is set
     * [built_struc_age](built_struc_age.md) - The age of the built structure since construction
     * [built_struc_set](built_struc_set.md) - The characterization of the location of the built structure as high or low human density
     * [built_struc_type](built_struc_type.md) - A physical structure that is a body or assemblage of bodies in space to form a system capable of supporting loads
     * [calcium](calcium.md) - Concentration of calcium in the sample
     * [carb_dioxide](carb_dioxide.md) - Carbon dioxide (gas) amount or concentration at the time of sampling
     * [carb_monoxide](carb_monoxide.md) - Carbon monoxide (gas) amount or concentration at the time of sampling
     * [carb_nitro_ratio](carb_nitro_ratio.md) - Ratio of amount or concentrations of carbon to nitrogen
     * [ceil_area](ceil_area.md) - The area of the ceiling space within the room
     * [ceil_cond](ceil_cond.md) - The physical condition of the ceiling at the time of sampling; photos or video preferred; use drawings to indicate location of damaged areas
     * [ceil_finish_mat](ceil_finish_mat.md) - The type of material used to finish a ceiling
     * [ceil_struc](ceil_struc.md) - The construction format of the ceiling
     * [ceil_texture](ceil_texture.md) - The feel, appearance, or consistency of a ceiling surface
     * [ceil_thermal_mass](ceil_thermal_mass.md) - The ability of the ceiling to provide inertia against temperature fluctuations. Generally this means concrete that is exposed. A metal deck that supports a concrete slab will act thermally as long as it is exposed to room air flow
     * [ceil_type](ceil_type.md) - The type of ceiling according to the ceiling's appearance or construction
     * [ceil_water_mold](ceil_water_mold.md) - Signs of the presence of mold or mildew on the ceiling
     * [chem_administration](chem_administration.md) - List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi
     * [chem_mutagen](chem_mutagen.md) - Treatment involving use of mutagens; should include the name of mutagen, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple mutagen regimens
     * [chem_oxygen_dem](chem_oxygen_dem.md) - A measure of the capacity of water to consume oxygen during the decomposition of organic matter and the oxidation of inorganic chemicals such as ammonia and nitrite
     * [chem_treatment](chem_treatment.md) - List of chemical compounds administered upstream the sampling location where sampling occurred (e.g. Glycols, H2S scavenger, corrosion and scale inhibitors, demulsifiers, and other production chemicals etc.). The commercial name of the product and name of the supplier should be provided. The date of administration should also be included
     * [chem_treatment_method](chem_treatment_method.md) - Method of chemical administration(dose, frequency, duration, time elapsed between administration and sampling) (e.g. 50 mg/l; twice a week; 1 hr; 0 days)
     * [chloride](chloride.md) - Concentration of chloride in the sample
     * [chlorophyll](chlorophyll.md) - Concentration of chlorophyll
     * [climate_environment](climate_environment.md) - Treatment involving an exposure to a particular climate; treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple climates
     * [collection_date](collection_date.md) - The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant
         * [ME➞collection_date](ME_collection_date.md)
         * [MIGS bacteria➞collection_date](MIGS_bacteria_collection_date.md)
         * [MIGS eukaryote➞collection_date](MIGS_eukaryote_collection_date.md)
         * [MIGS org➞collection_date](MIGS_org_collection_date.md)
         * [MIGS plant➞collection_date](MIGS_plant_collection_date.md)
         * [MIGS virus➞collection_date](MIGS_virus_collection_date.md)
         * [MIMAG➞collection_date](MIMAG_collection_date.md)
         * [MIMARKS specimen➞collection_date](MIMARKS_specimen_collection_date.md)
         * [MIMARKS survey➞collection_date](MIMARKS_survey_collection_date.md)
         * [MISAG➞collection_date](MISAG_collection_date.md)
         * [MIUVIG➞collection_date](MIUVIG_collection_date.md)
     * [conduc](conduc.md) - Electrical conductivity of water
     * [cool_syst_id](cool_syst_id.md) - The cooling system identifier
     * [crop_rotation](crop_rotation.md) - Whether or not crop is rotated, and if yes, rotation schedule
     * [cult_root_med](cult_root_med.md) - Name or reference for the hydroponic or in vitro culture rooting medium; can be the name of a commonly used medium or reference to a specific medium, e.g. Murashige and Skoog medium. If the medium has not been formally published, use the rooting medium descriptors.
     * [cur_land_use](cur_land_use.md) - Present state of sample site
     * [cur_vegetation](cur_vegetation.md) - Vegetation classification from one or more standard classification systems, or agricultural crop
     * [cur_vegetation_meth](cur_vegetation_meth.md) - Reference or method used in vegetation classification
     * [date_last_rain](date_last_rain.md) - The date of the last time it rained
     * [density](density.md) - Density of the sample, which is its mass per unit volume (aka volumetric mass density)
     * [depos_env](depos_env.md) - Main depositional environment (https://en.wikipedia.org/wiki/Depositional_environment). If "other" is specified, please propose entry in "additional info" field
     * [depth](depth.md) - Please refer to the definitions of depth in the environmental packages
     * [dermatology_disord](dermatology_disord.md) - History of dermatology disorders; can include multiple disorders. The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, skin disease (https://disease-ontology.org/?id=DOID:37).
     * [dew_point](dew_point.md) - The temperature to which a given parcel of humid air must be cooled, at constant barometric pressure, for water vapor to condense into water.
     * [diet_last_six_month](diet_last_six_month.md) - Specification of major diet changes in the last six months, if yes the change should be specified
     * [diether_lipids](diether_lipids.md) - Concentration of diether lipids; can include multiple types of diether lipids
     * [diss_carb_dioxide](diss_carb_dioxide.md) - Concentration of dissolved carbon dioxide in the sample or liquid portion of the sample
     * [diss_hydrogen](diss_hydrogen.md) - Concentration of dissolved hydrogen
     * [diss_inorg_carb](diss_inorg_carb.md) - Dissolved inorganic carbon concentration in the sample, typically measured after filtering the sample using a 0.45 micrometer filter
     * [diss_inorg_nitro](diss_inorg_nitro.md) - Concentration of dissolved inorganic nitrogen
     * [diss_inorg_phosp](diss_inorg_phosp.md) - Concentration of dissolved inorganic phosphorus in the sample
     * [diss_iron](diss_iron.md) - Concentration of dissolved iron in the sample
     * [diss_org_carb](diss_org_carb.md) - Concentration of dissolved organic carbon in the sample, liquid portion of the sample, or aqueous phase of the fluid
     * [diss_org_nitro](diss_org_nitro.md) - Dissolved organic nitrogen concentration measured as; total dissolved nitrogen - NH4 - NO3 - NO2
     * [diss_oxygen](diss_oxygen.md) - Concentration of dissolved oxygen
     * [diss_oxygen_fluid](diss_oxygen_fluid.md) - Concentration of dissolved oxygen in the oil field produced fluids as it contributes to oxgen-corrosion and microbial activity (e.g. Mic).
     * [dominant_hand](dominant_hand.md) - Dominant hand of the subject
     * [door_comp_type](door_comp_type.md) - The composite type of the door
     * [door_cond](door_cond.md) - The phsical condition of the door
     * [door_direct](door_direct.md) - The direction the door opens
     * [door_loc](door_loc.md) - The relative location of the door in the room
     * [door_mat](door_mat.md) - The material the door is composed of
     * [door_move](door_move.md) - The type of movement of the door
     * [door_size](door_size.md) - The size of the door
     * [door_type](door_type.md) - The type of door material
     * [door_type_metal](door_type_metal.md) - The type of metal door
     * [door_type_wood](door_type_wood.md) - The type of wood door
     * [door_water_mold](door_water_mold.md) - Signs of the presence of mold or mildew on a door
     * [douche](douche.md) - Date of most recent douche
     * [down_par](down_par.md) - Visible waveband radiance and irradiance measurements in the water column
     * [drainage_class](drainage_class.md) - Drainage classification from a standard system such as the USDA system
     * [drawings](drawings.md) - The buildings architectural drawings; if design is chosen, indicate phase-conceptual, schematic, design development, and construction documents
     * [drug_usage](drug_usage.md) - Any drug used by subject and the frequency of usage; can include multiple drugs used
     * [efficiency_percent](efficiency_percent.md) - Percentage of volatile solids removed from the anaerobic digestor
     * [elev](elev.md) - Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit
     * [elevator](elevator.md) - The number of elevators within the built structure
     * [emulsions](emulsions.md) - Amount or concentration of substances such as paints, adhesives, mayonnaise, hair colorants, emulsified oils, etc.; can include multiple emulsion types
     * [env_broad_scale](env_broad_scale.md) - In this field, report which major environmental system your sample or specimen came from. The systems identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. were you in the desert or a rainforest?). We recommend using subclasses of ENVO’s biome class: http://purl.obolibrary.org/obo/ENVO_00000428. Format (one term): termLabel [termID], Format (multiple terms): termLabel [termID]|termLabel [termID]|termLabel [termID]. Example: Annotating a water sample from the photic zone in middle of the Atlantic Ocean, consider: oceanic epipelagic zone biome [ENVO:01000033]. Example: Annotating a sample from the Amazon rainforest consider: tropical moist broadleaf forest biome [ENVO:01000228]. If needed, request new terms on the ENVO tracker, identified here: http://www.obofoundry.org/ontology/envo.html
         * [ME➞env_broad_scale](ME_env_broad_scale.md)
         * [MIGS bacteria➞env_broad_scale](MIGS_bacteria_env_broad_scale.md)
         * [MIGS eukaryote➞env_broad_scale](MIGS_eukaryote_env_broad_scale.md)
         * [MIGS org➞env_broad_scale](MIGS_org_env_broad_scale.md)
         * [MIGS plant➞env_broad_scale](MIGS_plant_env_broad_scale.md)
         * [MIGS virus➞env_broad_scale](MIGS_virus_env_broad_scale.md)
         * [MIMAG➞env_broad_scale](MIMAG_env_broad_scale.md)
         * [MIMARKS specimen➞env_broad_scale](MIMARKS_specimen_env_broad_scale.md)
         * [MIMARKS survey➞env_broad_scale](MIMARKS_survey_env_broad_scale.md)
         * [MISAG➞env_broad_scale](MISAG_env_broad_scale.md)
         * [MIUVIG➞env_broad_scale](MIUVIG_env_broad_scale.md)
     * [env_local_scale](env_local_scale.md) - In this field, report the entity or entities which are in your sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. Please use terms that are present in ENVO and which are of smaller spatial grain than your entry for env_broad_scale. Format (one term): termLabel [termID]; Format (multiple terms): termLabel [termID]|termLabel [termID]|termLabel [termID]. Example: Annotating a pooled sample taken from various vegetation layers in a forest consider: canopy [ENVO:00000047]|herb and fern layer [ENVO:01000337]|litter layer [ENVO:01000338]|understory [01000335]|shrub layer [ENVO:01000336]. If needed, request new terms on the ENVO tracker, identified here: http://www.obofoundry.org/ontology/envo.html
         * [ME➞env_local_scale](ME_env_local_scale.md)
         * [MIGS bacteria➞env_local_scale](MIGS_bacteria_env_local_scale.md)
         * [MIGS eukaryote➞env_local_scale](MIGS_eukaryote_env_local_scale.md)
         * [MIGS org➞env_local_scale](MIGS_org_env_local_scale.md)
         * [MIGS plant➞env_local_scale](MIGS_plant_env_local_scale.md)
         * [MIGS virus➞env_local_scale](MIGS_virus_env_local_scale.md)
         * [MIMAG➞env_local_scale](MIMAG_env_local_scale.md)
         * [MIMARKS specimen➞env_local_scale](MIMARKS_specimen_env_local_scale.md)
         * [MIMARKS survey➞env_local_scale](MIMARKS_survey_env_local_scale.md)
         * [MISAG➞env_local_scale](MISAG_env_local_scale.md)
         * [MIUVIG➞env_local_scale](MIUVIG_env_local_scale.md)
     * [env_medium](env_medium.md) - In this field, report which environmental material or materials (pipe separated) immediately surrounded your sample or specimen prior to sampling, using one or more subclasses of ENVO’s environmental material class: http://purl.obolibrary.org/obo/ENVO_00010483. Format (one term): termLabel [termID]; Format (multiple terms): termLabel [termID]|termLabel [termID]|termLabel [termID]. Example: Annotating a fish swimming in the upper 100 m of the Atlantic Ocean, consider: ocean water [ENVO:00002151]. Example: Annotating a duck on a pond consider: pond water [ENVO:00002228]|air ENVO_00002005. If needed, request new terms on the ENVO tracker, identified here: http://www.obofoundry.org/ontology/envo.html
         * [ME➞env_medium](ME_env_medium.md)
         * [MIGS bacteria➞env_medium](MIGS_bacteria_env_medium.md)
         * [MIGS eukaryote➞env_medium](MIGS_eukaryote_env_medium.md)
         * [MIGS org➞env_medium](MIGS_org_env_medium.md)
         * [MIGS plant➞env_medium](MIGS_plant_env_medium.md)
         * [MIGS virus➞env_medium](MIGS_virus_env_medium.md)
         * [MIMAG➞env_medium](MIMAG_env_medium.md)
         * [MIMARKS specimen➞env_medium](MIMARKS_specimen_env_medium.md)
         * [MIMARKS survey➞env_medium](MIMARKS_survey_env_medium.md)
         * [MISAG➞env_medium](MISAG_env_medium.md)
         * [MIUVIG➞env_medium](MIUVIG_env_medium.md)
     * [escalator](escalator.md) - The number of escalators within the built structure
     * [ethylbenzene](ethylbenzene.md) - Concentration of ethylbenzene in the sample
     * [exp_duct](exp_duct.md) - The amount of exposed ductwork in the room
     * [exp_pipe](exp_pipe.md) - The number of exposed pipes in the room
     * [ext_door](ext_door.md) - The number of exterior doors in the built structure
     * [ext_wall_orient](ext_wall_orient.md) - The orientation of the exterior wall
     * [ext_window_orient](ext_window_orient.md) - The compass direction the exterior window of the room is facing
     * [extreme_event](extreme_event.md) - Unusual physical events that may have affected microbial populations
     * [extreme_salinity](extreme_salinity.md) - Measured salinity
     * [fao_class](fao_class.md) - Soil classification from the FAO World Reference Database for Soil Resources. The list can be found at http://www.fao.org/nr/land/sols/soil/wrb-soil-maps/reference-groups
     * [fertilizer_regm](fertilizer_regm.md) - Information about treatment involving the use of fertilizers; should include the name of fertilizer, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple fertilizer regimens
     * [field](field.md) - Name of the hydrocarbon field (e.g. Albacora)
     * [filter_type](filter_type.md) - A device which removes solid particulates or airborne molecular contaminants
     * [fire](fire.md) - Historical and/or physical evidence of fire
     * [fireplace_type](fireplace_type.md) - A firebox with chimney
     * [flooding](flooding.md) - Historical and/or physical evidence of flooding
     * [floor_age](floor_age.md) - The time period since installment of the carpet or flooring
     * [floor_area](floor_area.md) - The area of the floor space within the room
     * [floor_cond](floor_cond.md) - The physical condition of the floor at the time of sampling; photos or video preferred; use drawings to indicate location of damaged areas
     * [floor_count](floor_count.md) - The number of floors in the building, including basements and mechanical penthouse
     * [floor_finish_mat](floor_finish_mat.md) - The floor covering type; the finished surface that is walked on
     * [floor_struc](floor_struc.md) - Refers to the structural elements and subfloor upon which the finish flooring is installed
     * [floor_thermal_mass](floor_thermal_mass.md) - The ability of the floor to provide inertia against temperature fluctuations
     * [floor_water_mold](floor_water_mold.md) - Signs of the presence of mold or mildew in a room
     * [fluor](fluor.md) - Raw or converted fluorescence of water
     * [foetal_health_stat](foetal_health_stat.md) - Specification of foetal health status, should also include abortion
     * [freq_clean](freq_clean.md) - The number of times the building is cleaned per week
     * [freq_cook](freq_cook.md) - The number of times a meal is cooked per week
     * [fungicide_regm](fungicide_regm.md) - Information about treatment involving use of fungicides; should include the name of fungicide, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple fungicide regimens
     * [furniture](furniture.md) - The types of furniture present in the sampled room
     * [gaseous_environment](gaseous_environment.md) - Use of conditions with differing gaseous environments; should include the name of gaseous compound, amount administered, treatment duration, interval and total experimental duration; can include multiple gaseous environment regimens
     * [gaseous_substances](gaseous_substances.md) - Amount or concentration of substances such as hydrogen sulfide, carbon dioxide, methane, etc.; can include multiple substances
     * [gastrointest_disord](gastrointest_disord.md) - History of gastrointestinal tract disorders; can include multiple disorders. History of blood disorders; can include multiple disorders.  The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, gastrointestinal system disease (https://disease-ontology.org/?id=DOID:77).
     * [gender_restroom](gender_restroom.md) - The gender type of the restroom
     * [genetic_mod](genetic_mod.md) - Genetic modifications of the genome of an organism, which may occur naturally by spontaneous mutation, or be introduced by some experimental means, e.g. specification of a transgene or the gene knocked-out or details of transient transfection
     * [geo_loc_name](geo_loc_name.md) - The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (v 1.512) (http://purl.bioontology.org/ontology/GAZ)
         * [ME➞geo_loc_name](ME_geo_loc_name.md)
         * [MIGS bacteria➞geo_loc_name](MIGS_bacteria_geo_loc_name.md)
         * [MIGS eukaryote➞geo_loc_name](MIGS_eukaryote_geo_loc_name.md)
         * [MIGS org➞geo_loc_name](MIGS_org_geo_loc_name.md)
         * [MIGS plant➞geo_loc_name](MIGS_plant_geo_loc_name.md)
         * [MIGS virus➞geo_loc_name](MIGS_virus_geo_loc_name.md)
         * [MIMAG➞geo_loc_name](MIMAG_geo_loc_name.md)
         * [MIMARKS specimen➞geo_loc_name](MIMARKS_specimen_geo_loc_name.md)
         * [MIMARKS survey➞geo_loc_name](MIMARKS_survey_geo_loc_name.md)
         * [MISAG➞geo_loc_name](MISAG_geo_loc_name.md)
         * [MIUVIG➞geo_loc_name](MIUVIG_geo_loc_name.md)
     * [gestation_state](gestation_state.md) - Specification of the gestation state
     * [glucosidase_act](glucosidase_act.md) - Measurement of glucosidase activity
     * [gravidity](gravidity.md) - Whether or not subject is gravid, and if yes date due or date post-conception, specifying which is used
     * [gravity](gravity.md) - Information about treatment involving use of gravity factor to study various types of responses in presence, absence or modified levels of gravity; treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple treatments
     * [growth_facil](growth_facil.md) - Type of facility where the sampled plant was grown; controlled vocabulary: growth chamber, open top chamber, glasshouse, experimental garden, field. Alternatively use Crop Ontology (CO) terms, see http://www.cropontology.org/ontology/CO_715/Crop%20Research
     * [growth_habit](growth_habit.md) - Characteristic shape, appearance or growth form of a plant species
     * [growth_hormone_regm](growth_hormone_regm.md) - Information about treatment involving use of growth hormones; should include the name of growth hormone, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple growth hormone regimens
     * [gynecologic_disord](gynecologic_disord.md) - History of gynecological disorders; can include multiple disorders. The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, female reproductive system disease (https://disease-ontology.org/?id=DOID:229).
     * [hall_count](hall_count.md) - The total count of hallways and cooridors in the built structure
     * [handidness](handidness.md) - The handidness of the individual sampled
     * [hc_produced](hc_produced.md) - Main hydrocarbon type produced from resource (i.e. Oil, gas, condensate, etc). If "other" is specified, please propose entry in "additional info" field
     * [hcr](hcr.md) - Main Hydrocarbon Resource type. The term "Hydrocarbon Resource" HCR defined as a natural environmental feature containing large amounts of hydrocarbons at high concentrations potentially suitable for commercial exploitation. This term should not be confused with the Hydrocarbon Occurrence term which also includes hydrocarbon-rich environments with currently limited commercial interest such as seeps, outcrops, gas hydrates etc. If "other" is specified, please propose entry in "additional info" field
     * [hcr_fw_salinity](hcr_fw_salinity.md) - Original formation water salinity (prior to secondary recovery e.g. Waterflooding) expressed as TDS
     * [hcr_geol_age](hcr_geol_age.md) - Geological age of hydrocarbon resource (Additional info: https://en.wikipedia.org/wiki/Period_(geology)). If "other" is specified, please propose entry in "additional info" field
     * [hcr_pressure](hcr_pressure.md) - Original pressure of the hydrocarbon resource
     * [hcr_temp](hcr_temp.md) - Original temperature of the hydrocarbon resource
     * [heat_cool_type](heat_cool_type.md) - Methods of conditioning or heating a room or building
     * [heat_deliv_loc](heat_deliv_loc.md) - The location of heat delivery within the room
     * [heat_system_deliv_meth](heat_system_deliv_meth.md) - The method by which the heat is delivered through the system
     * [heat_system_id](heat_system_id.md) - The heating system identifier
     * [heavy_metals](heavy_metals.md) - Heavy metals present and concentrationsany drug used by subject and the frequency of usage; can include multiple heavy metals and concentrations
     * [heavy_metals_meth](heavy_metals_meth.md) - Reference or method used in determining heavy metals
     * [height_carper_fiber](height_carper_fiber.md) - The average carpet fiber height in the indoor environment
     * [herbicide_regm](herbicide_regm.md) - Information about treatment involving use of herbicides; information about treatment involving use of growth hormones; should include the name of herbicide, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens
     * [horizon](horizon.md) - Specific layer in the land area which measures parallel to the soil surface and possesses physical characteristics which differ from the layers above and beneath
     * [horizon_meth](horizon_meth.md) - Reference or method used in determining the horizon
     * [host_age](host_age.md) - Age of host at the time of sampling; relevant scale depends on species and study, e.g. Could be seconds for amoebae or centuries for trees
     * [host_blood_press_diast](host_blood_press_diast.md) - Resting diastolic blood pressure, measured as mm mercury
     * [host_blood_press_syst](host_blood_press_syst.md) - Resting systolic blood pressure, measured as mm mercury
     * [host_body_habitat](host_body_habitat.md) - Original body habitat where the sample was obtained from
     * [host_body_mass_index](host_body_mass_index.md) - Body mass index, calculated as weight/(height)squared
     * [host_body_product](host_body_product.md) - Substance produced by the body, e.g. Stool, mucus, where the sample was obtained from. For foundational model of anatomy ontology (fma) or Uber-anatomy ontology (UBERON) terms, please see https://www.ebi.ac.uk/ols/ontologies/fma or https://www.ebi.ac.uk/ols/ontologies/uberon
     * [host_body_site](host_body_site.md) - Name of body site where the sample was obtained from, such as a specific organ or tissue (tongue, lung etc...). For foundational model of anatomy ontology (fma) (v 4.11.0) or Uber-anatomy ontology (UBERON) (v releases/2014-06-15) terms, please see http://purl.bioontology.org/ontology/FMA or http://purl.bioontology.org/ontology/UBERON
     * [host_body_temp](host_body_temp.md) - Core body temperature of the host when sample was collected
     * [host_color](host_color.md) - The color of host
     * [host_common_name](host_common_name.md) - Common name of the host, e.g. Human
     * [host_diet](host_diet.md) - Type of diet depending on the host, for animals omnivore, herbivore etc., for humans high-fat, meditteranean etc.; can include multiple diet types
     * [host_disease_stat](host_disease_stat.md) - List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text
     * [host_dry_mass](host_dry_mass.md) - Measurement of dry mass
     * [host_family_relationship](host_family_relationship.md) - Relationships to other hosts in the same study; can include multiple relationships
     * [host_genotype](host_genotype.md) - Observed genotype
     * [host_growth_cond](host_growth_cond.md) - Literature reference giving growth conditions of the host
     * [host_height](host_height.md) - The height of subject
     * [host_hiv_stat](host_hiv_stat.md) - HIV status of subject, if yes HAART initiation status should also be indicated as [YES or NO]
     * [host_infra_specific_name](host_infra_specific_name.md) - Taxonomic information about the host below subspecies level
     * [host_infra_specific_rank](host_infra_specific_rank.md) - Taxonomic rank information about the host below subspecies level, such as variety, form, rank etc.
     * [host_last_meal](host_last_meal.md) - Content of last meal and time since feeding; can include multiple values
     * [host_length](host_length.md) - The length of subject
     * [host_life_stage](host_life_stage.md) - Description of life stage of host
     * [host_occupation](host_occupation.md) - Most frequent job performed by subject
     * [host_phenotype](host_phenotype.md) - Phenotype of human or other host. For phenotypic quality ontology (pato) (v 2018-03-27) terms, please see http://purl.bioontology.org/ontology/pato. For Human Phenotype Ontology (HP) (v 2018-06-13) please see http://purl.bioontology.org/ontology/HP
     * [host_pulse](host_pulse.md) - Resting pulse, measured as beats per minute
     * [host_sex](host_sex.md) - Physical sex of the host
     * [host_shape](host_shape.md) - Morphological shape of host
     * [host_subject_id](host_subject_id.md) - A unique identifier by which each subject can be referred to, de-identified, e.g. #131
     * [host_substrate](host_substrate.md) - The growth substrate of the host
     * [host_taxid](host_taxid.md) - NCBI taxon id of the host, e.g. 9606
     * [host_tot_mass](host_tot_mass.md) - Total mass of the host at collection, the unit depends on host
     * [host_wet_mass](host_wet_mass.md) - Measurement of wet mass
     * [hrt](hrt.md) - Whether subject had hormone replacement theraphy, and if yes start date
     * [humidity](humidity.md) - Amount of water vapour in the air, at the time of sampling
     * [humidity_regm](humidity_regm.md) - Information about treatment involving an exposure to varying degree of humidity; information about treatment involving use of growth hormones; should include amount of humidity administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens
     * [hysterectomy](hysterectomy.md) - Specification of whether hysterectomy was performed
     * [ihmc_ethnicity](ihmc_ethnicity.md) - Ethnicity of the subject
     * [ihmc_medication_code](ihmc_medication_code.md) - Can include multiple medication codes
     * [indoor_space](indoor_space.md) - A distinguishable space within a structure, the purpose for which discrete areas of a building is used
     * [indoor_surf](indoor_surf.md) - Type of indoor surface
     * [indust_eff_percent](indust_eff_percent.md) - Percentage of industrial effluents received by wastewater treatment plant
     * [inorg_particles](inorg_particles.md) - Concentration of particles such as sand, grit, metal particles, ceramics, etc.; can include multiple particles
     * [inside_lux](inside_lux.md) - The recorded value at sampling time (power density)
     * [int_wall_cond](int_wall_cond.md) - The physical condition of the wall at the time of sampling; photos or video preferred; use drawings to indicate location of damaged areas
     * [iw_bt_date_well](iw_bt_date_well.md) - Injection water breakthrough date per well following a secondary and/or tertiary recovery
     * [iwf](iwf.md) - Proportion of the produced fluids derived from injected water at the time of sampling. (e.g. 87%)
     * [last_clean](last_clean.md) - The last time the floor was cleaned (swept, mopped, vacuumed)
     * [lat_lon](lat_lon.md) - The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system
         * [ME➞lat_lon](ME_lat_lon.md)
         * [MIGS bacteria➞lat_lon](MIGS_bacteria_lat_lon.md)
         * [MIGS eukaryote➞lat_lon](MIGS_eukaryote_lat_lon.md)
         * [MIGS org➞lat_lon](MIGS_org_lat_lon.md)
         * [MIGS plant➞lat_lon](MIGS_plant_lat_lon.md)
         * [MIGS virus➞lat_lon](MIGS_virus_lat_lon.md)
         * [MIMAG➞lat_lon](MIMAG_lat_lon.md)
         * [MIMARKS specimen➞lat_lon](MIMARKS_specimen_lat_lon.md)
         * [MIMARKS survey➞lat_lon](MIMARKS_survey_lat_lon.md)
         * [MISAG➞lat_lon](MISAG_lat_lon.md)
         * [MIUVIG➞lat_lon](MIUVIG_lat_lon.md)
     * [light_intensity](light_intensity.md) - Measurement of light intensity
     * [light_regm](light_regm.md) - Information about treatment(s) involving exposure to light, including both light intensity and quality.
     * [light_type](light_type.md) - Application of light to achieve some practical or aesthetic effect. Lighting includes the use of both artificial light sources such as lamps and light fixtures, as well as natural illumination by capturing daylight. Can also include absence of light
     * [link_addit_analys](link_addit_analys.md) - Link to additional analysis results performed on the sample
     * [link_class_info](link_class_info.md) - Link to digitized soil maps or other soil classification information
     * [link_climate_info](link_climate_info.md) - Link to climate resource
     * [lithology](lithology.md) - Hydrocarbon resource main lithology (Additional information: http://petrowiki.org/Lithology_and_rock_type_determination). If "other" is specified, please propose entry in "additional info" field
     * [liver_disord](liver_disord.md) - History of liver disorders; can include multiple disorders. The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, liver disease (https://disease-ontology.org/?id=DOID:409).
     * [local_class](local_class.md) - Soil classification based on local soil classification system
     * [local_class_meth](local_class_meth.md) - Reference or method used in determining the local soil classification
     * [magnesium](magnesium.md) - Concentration of magnesium in the sample
     * [maternal_health_stat](maternal_health_stat.md) - Specification of the maternal health status
     * [max_occup](max_occup.md) - The maximum amount of people allowed in the indoor environment
     * [mean_frict_vel](mean_frict_vel.md) - Measurement of mean friction velocity
     * [mean_peak_frict_vel](mean_peak_frict_vel.md) - Measurement of mean peak friction velocity
     * [mech_struc](mech_struc.md) - mechanical structure: a moving structure
     * [mechanical_damage](mechanical_damage.md) - Information about any mechanical damage exerted on the plant; can include multiple damages and sites
     * [medic_hist_perform](medic_hist_perform.md) - Whether full medical history was collected
     * [menarche](menarche.md) - Date of most recent menstruation
     * [menopause](menopause.md) - Date of onset of menopause
     * [methane](methane.md) - Methane (gas) amount or concentration at the time of sampling
     * [microbial_biomass](microbial_biomass.md) - The part of the organic matter in the soil that constitutes living microorganisms smaller than 5-10 micrometer. If you keep this, you would need to have correction factors used for conversion to the final units
     * [microbial_biomass_meth](microbial_biomass_meth.md) - Reference or method used in determining microbial biomass
     * [mineral_nutr_regm](mineral_nutr_regm.md) - Information about treatment involving the use of mineral supplements; should include the name of mineral nutrient, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple mineral nutrient regimens
     * [misc_param](misc_param.md) - Any other measurement performed or parameter collected, that is not listed here
     * [n_alkanes](n_alkanes.md) - Concentration of n-alkanes; can include multiple n-alkanes
     * [nitrate](nitrate.md) - Concentration of nitrate in the sample
     * [nitrite](nitrite.md) - Concentration of nitrite in the sample
     * [nitro](nitro.md) - Concentration of nitrogen (total)
     * [non_mineral_nutr_regm](non_mineral_nutr_regm.md) - Information about treatment involving the exposure of plant to non-mineral nutrient such as oxygen, hydrogen or carbon; should include the name of non-mineral nutrient, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple non-mineral nutrient regimens
     * [nose_mouth_teeth_throat_disord](nose_mouth_teeth_throat_disord.md) - History of nose/mouth/teeth/throat disorders; can include multiple disorders. The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, nose disease (https://disease-ontology.org/?id=DOID:2825), mouth disease (https://disease-ontology.org/?id=DOID:403), tooth disease (https://disease-ontology.org/?id=DOID:1091), or upper respiratory tract disease (https://disease-ontology.org/?id=DOID:974).
     * [nose_throat_disord](nose_throat_disord.md) - History of nose-throat disorders; can include multiple disorders,  The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, lung disease (https://disease-ontology.org/?id=DOID:850), upper respiratory tract disease (https://disease-ontology.org/?id=DOID:974).
     * [number_pets](number_pets.md) - The number of pets residing in the sampled space
     * [number_plants](number_plants.md) - The number of plant(s) in the sampling space
     * [number_resident](number_resident.md) - The number of individuals currently occupying in the sampling location
     * [occup_density_samp](occup_density_samp.md) - Average number of occupants at time of sampling per square footage
     * [occup_document](occup_document.md) - The type of documentation of occupancy
     * [occup_samp](occup_samp.md) - Number of occupants present at time of sample within the given space
     * [org_carb](org_carb.md) - Concentration of organic carbon
     * [org_matter](org_matter.md) - Concentration of organic matter
     * [org_nitro](org_nitro.md) - Concentration of organic nitrogen
     * [org_particles](org_particles.md) - Concentration of particles such as faeces, hairs, food, vomit, paper fibers, plant material, humus, etc.
     * [organism_count](organism_count.md) - Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)
     * [organism_count_qpcr_info](organism_count_qpcr_info.md) - If qpcr was used for the cell count, the target gene name, the primer sequence and the cycling conditions should also be provided. (Example: 16S rrna; FWD:ACGTAGCTATGACGT REV:GTGCTAGTCGAGTAC; initial denaturation:90C_5min; denaturation:90C_2min; annealing:52C_30 sec; elongation:72C_30 sec; 90 C for 1 min; final elongation:72C_5min; 30 cycles)
     * [owc_tvdss](owc_tvdss.md) - Depth of the original oil water contact (OWC) zone (average) (m TVDSS)
     * [oxy_stat_samp](oxy_stat_samp.md) - Oxygenation status of sample
     * [oxygen](oxygen.md) - Oxygen (gas) amount or concentration at the time of sampling
     * [part_org_carb](part_org_carb.md) - Concentration of particulate organic carbon
     * [part_org_nitro](part_org_nitro.md) - Concentration of particulate organic nitrogen
     * [particle_class](particle_class.md) - Particles are classified, based on their size, into six general categories:clay, silt, sand, gravel, cobbles, and boulders; should include amount of particle preceded by the name of the particle type; can include multiple values
     * [permeability](permeability.md) - Measure of the ability of a hydrocarbon resource to allow fluids to pass through it. (Additional information: https://en.wikipedia.org/wiki/Permeability_(earth_sciences))
     * [perturbation](perturbation.md) - Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types
     * [pesticide_regm](pesticide_regm.md) - Information about treatment involving use of insecticides; should include the name of pesticide, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple pesticide regimens
     * [pet_farm_animal](pet_farm_animal.md) - Specification of presence of pets or farm animals in the environment of subject, if yes the animals should be specified; can include multiple animals present
     * [petroleum_hydrocarb](petroleum_hydrocarb.md) - Concentration of petroleum hydrocarbon
     * [ph](ph.md) - Ph measurement of the sample, or liquid portion of sample, or aqueous phase of the fluid
     * [ph_meth](ph_meth.md) - Reference or method used in determining ph
     * [ph_regm](ph_regm.md) - Information about treatment involving exposure of plants to varying levels of ph of the growth media, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimen
     * [phaeopigments](phaeopigments.md) - Concentration of phaeopigments; can include multiple phaeopigments
     * [phosphate](phosphate.md) - Concentration of phosphate
     * [phosplipid_fatt_acid](phosplipid_fatt_acid.md) - Concentration of phospholipid fatty acids; can include multiple values
     * [photon_flux](photon_flux.md) - Measurement of photon flux
     * [plant_growth_med](plant_growth_med.md) - Specification of the media for growing the plants or tissue cultured samples, e.g. soil, aeroponic, hydroponic, in vitro solid culture medium, in vitro liquid culture medium. Recommended value is a specific value from EO:plant growth medium (follow this link for terms http://purl.obolibrary.org/obo/EO_0007147) or other controlled vocabulary
     * [plant_product](plant_product.md) - Substance produced by the plant, where the sample was obtained from
     * [plant_sex](plant_sex.md) - Sex of the reproductive parts on the whole plant, e.g. pistillate, staminate, monoecieous, hermaphrodite.
     * [plant_struc](plant_struc.md) - Name of plant structure the sample was obtained from; for Plant Ontology (PO) (v releases/2017-12-14) terms, see http://purl.bioontology.org/ontology/PO, e.g. petiole epidermis (PO_0000051). If an individual flower is sampled, the sex of it can be recorded here.
     * [pollutants](pollutants.md) - Pollutant types and, amount or concentrations measured at the time of sampling; can report multiple pollutants by entering numeric values preceded by name of pollutant
     * [pool_dna_extracts](pool_dna_extracts.md) - Indicate whether multiple DNA extractions were mixed. If the answer yes, the number of extracts that were pooled should be given
     * [porosity](porosity.md) - Porosity of deposited sediment is volume of voids divided by the total volume of sample
     * [potassium](potassium.md) - Concentration of potassium in the sample
     * [pour_point](pour_point.md) - Temperature at which a liquid becomes semi solid and loses its flow characteristics. In crude oil a high¬†pour point¬†is generally associated with a high paraffin content, typically found in crude deriving from a larger proportion of plant material. (soure: https://en.wikipedia.org/wiki/pour_point)
     * [pre_treatment](pre_treatment.md) - The process of pre-treatment removes materials that can be easily collected from the raw wastewater
     * [pregnancy](pregnancy.md) - Date due of pregnancy
     * [pres_animal](pres_animal.md) - The number and type of animals present in the sampling space
     * [pressure](pressure.md) - Pressure to which the sample is subject to, in atmospheres
     * [previous_land_use](previous_land_use.md) - Previous land use and dates
     * [previous_land_use_meth](previous_land_use_meth.md) - Reference or method used in determining previous land use and dates
     * [primary_prod](primary_prod.md) - Measurement of primary production, generally measured as isotope uptake
     * [primary_treatment](primary_treatment.md) - The process to produce both a generally homogeneous liquid capable of being treated biologically and a sludge that can be separately treated or processed
     * [prod_rate](prod_rate.md) - Oil and/or gas production rates per well (e.g. 524 m3 / day)
     * [prod_start_date](prod_start_date.md) - Date of field's first production
     * [profile_position](profile_position.md) - Cross-sectional position in the hillslope where sample was collected.sample area position in relation to surrounding areas
     * [pulmonary_disord](pulmonary_disord.md) - History of pulmonary disorders; can include multiple disorders. The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, lung disease (https://disease-ontology.org/?id=DOID:850).
     * [quad_pos](quad_pos.md) - The quadrant position of the sampling room within the building
     * [radiation_regm](radiation_regm.md) - Information about treatment involving exposure of plant or a plant part to a particular radiation regimen; should include the radiation type, amount or intensity administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple radiation regimens
     * [rainfall_regm](rainfall_regm.md) - Information about treatment involving an exposure to a given amount of rainfall, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens
     * [reactor_type](reactor_type.md) - Anaerobic digesters can be designed and engineered to operate using a number of different process configurations, as batch or continuous, mesophilic, high solid or low solid, and single stage or multistage
     * [redox_potential](redox_potential.md) - Redox potential, measured relative to a hydrogen cell, indicating oxidation or reduction potential
     * [rel_air_humidity](rel_air_humidity.md) - Partial vapor and air pressure, density of the vapor and air, or by the actual mass of the vapor and air
     * [rel_humidity_out](rel_humidity_out.md) - The recorded outside relative humidity value at the time of sampling
     * [rel_samp_loc](rel_samp_loc.md) - The sampling location within the train car
     * [reservoir](reservoir.md) - Name of the reservoir (e.g. Carapebus)
     * [resins_pc](resins_pc.md) - Saturate, Aromatic, Resin and Asphaltene¬†(SARA) is an analysis method that divides¬†crude oil¬†components according to their polarizability and polarity. There are three main methods to obtain SARA results. The most popular one is known as the Iatroscan TLC-FID and is referred to as IP-143 (source: https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene)
     * [resp_part_matter](resp_part_matter.md) - Concentration of substances that remain suspended in the air, and comprise mixtures of organic and inorganic substances (PM10 and PM2.5); can report multiple PM's by entering numeric values preceded by name of PM
     * [room_air_exch_rate](room_air_exch_rate.md) - The rate at which outside air replaces indoor air in a given space
     * [room_architec_element](room_architec_element.md) - The unique details and component parts that, together, form the architecture of a distinguisahable space within a built structure
     * [room_condt](room_condt.md) - The condition of the room at the time of sampling
     * [room_connected](room_connected.md) - List of rooms connected to the sampling room by a doorway
     * [room_count](room_count.md) - The total count of rooms in the built structure including all room types
     * [room_dim](room_dim.md) - The length, width and height of sampling room
     * [room_door_dist](room_door_dist.md) - Distance between doors (meters) in the hallway between the sampling room and adjacent rooms
     * [room_door_share](room_door_share.md) - List of room(s) (room number, room name) sharing a door with the sampling room
     * [room_hallway](room_hallway.md) - List of room(s) (room number, room name) located in the same hallway as sampling room
     * [room_loc](room_loc.md) - The position of the room within the building
     * [room_moist_damage_hist](room_moist_damage_hist.md) - The history of moisture damage or mold in the past 12 months. Number of events of moisture damage or mold observed
     * [room_net_area](room_net_area.md) - The net floor area of sampling room. Net area excludes wall thicknesses
     * [room_occup](room_occup.md) - Count of room occupancy at time of sampling
     * [room_samp_pos](room_samp_pos.md) - The horizontal sampling position in the room relative to architectural elements
     * [room_type](room_type.md) - The main purpose or activity of the sampling room. A room is any distinguishable space within a structure
     * [room_vol](room_vol.md) - Volume of sampling room
     * [room_wall_share](room_wall_share.md) - List of room(s) (room number, room name) sharing a wall with the sampling room
     * [room_window_count](room_window_count.md) - Number of windows in the room
     * [root_cond](root_cond.md) - Relevant rooting conditions such as field plot size, sowing density, container dimensions, number of plants per container.
     * [root_med_carbon](root_med_carbon.md) - Source of organic carbon in the culture rooting medium; e.g. sucrose.
     * [root_med_macronutr](root_med_macronutr.md) - Measurement of the culture rooting medium macronutrients (N,P, K, Ca, Mg, S); e.g. KH2PO4 (170¬†mg/L).
     * [root_med_micronutr](root_med_micronutr.md) - Measurement of the culture rooting medium micronutrients (Fe, Mn, Zn, B, Cu, Mo); e.g. H3BO3 (6.2¬†mg/L).
     * [root_med_ph](root_med_ph.md) - pH measurement of the culture rooting medium; e.g. 5.5.
     * [root_med_regl](root_med_regl.md) - Growth regulators in the culture rooting medium such as cytokinins, auxins, gybberellins, abscisic acid; e.g. 0.5¬†mg/L NAA.
     * [root_med_solid](root_med_solid.md) - Specification of the solidifying agent in the culture rooting medium; e.g. agar.
     * [root_med_suppl](root_med_suppl.md) - Organic supplements of the culture rooting medium, such as vitamins, amino acids, organic acids, antibiotics activated charcoal; e.g. nicotinic acid (0.5¬†mg/L).
     * [salinity](salinity.md) - Salinity is the total concentration of all dissolved salts in a water sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater
     * [salinity_meth](salinity_meth.md) - Reference or method used in determining salinity
     * [salt_regm](salt_regm.md) - Information about treatment involving use of salts as supplement to liquid and soil growth media; should include the name of salt, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple salt regimens
     * [samp_capt_status](samp_capt_status.md) - Reason for the sample
     * [samp_collection_point](samp_collection_point.md) - Sampling point on the asset were sample was collected (e.g. Wellhead, storage tank, separator, etc). If "other" is specified, please propose entry in "additional info" field
     * [samp_dis_stage](samp_dis_stage.md) - Stage of the disease at the time of sample collection, e.g. inoculation, penetration, infection, growth and reproduction, dissemination of pathogen.
     * [samp_floor](samp_floor.md) - The floor of the building, where the sampling room is located
     * [samp_loc_corr_rate](samp_loc_corr_rate.md) - Metal corrosion rate is the speed of metal deterioration due to environmental conditions. As environmental conditions change corrosion rates change accordingly. Therefore, long term corrosion rates are generally more informative than short term rates and for that reason they are preferred during reporting. In the case of suspected MIC, corrosion rate measurements at the time of sampling might provide insights into the involvement of certain microbial community members in MIC as well as potential microbial interplays
     * [samp_md](samp_md.md) - In non deviated well, measured depth is equal to the true vertical depth, TVD (TVD=TVDSS plus the reference or datum it refers to). In deviated wells, the MD is the length of trajectory of the borehole measured from the same reference or datum. Common datums used are ground level (GL), drilling rig floor (DF), rotary table (RT), kelly bushing (KB) and mean sea level (MSL). If "other" is specified, please propose entry in "additional info" field
     * [samp_preserv](samp_preserv.md) - Preservative added to the sample (e.g. Rnalater, alcohol, formaldehyde, etc.). Where appropriate include volume added (e.g. Rnalater; 2 ml)
     * [samp_room_id](samp_room_id.md) - Sampling room number. This ID should be consistent with the designations on the building floor plans
     * [samp_salinity](samp_salinity.md) - Salinity is the total concentration of all dissolved salts in a liquid or solid (in the form of an extract obtained by centrifugation) sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater
     * [samp_sort_meth](samp_sort_meth.md) - Method by which samples are sorted; open face filter collecting total suspended particles, prefilter to remove particles larger than X micrometers in diameter, where common values of X would be 10 and 2.5 full size sorting in a cascade impactor.
     * [samp_store_dur](samp_store_dur.md) - Duration for which the sample was stored
     * [samp_store_loc](samp_store_loc.md) - Location at which sample was stored, usually name of a specific freezer/room
     * [samp_store_temp](samp_store_temp.md) - Temperature at which sample was stored, e.g. -80 degree Celsius
     * [samp_subtype](samp_subtype.md) - Name of sample sub-type. For example if "sample type" is "Produced Water" then subtype could be "Oil Phase" or "Water Phase". If "other" is specified, please propose entry in "additional info" field
     * [samp_time_out](samp_time_out.md) - The recent and long term history of outside sampling
     * [samp_transport_cond](samp_transport_cond.md) - Sample transport duration (in days or hrs) and temperature the sample was exposed to (e.g. 5.5 days; 20 ¬∞C)
     * [samp_tvdss](samp_tvdss.md) - Depth of the sample i.e. The vertical distance between the sea level and the sampled position in the subsurface. Depth can be reported as an interval for subsurface samples e.g. 1325.75-1362.25 m
     * [samp_type](samp_type.md) - Type of material (i.e. sample) collected. Includes types like core, rock trimmings, drill cuttings, piping section, coupon, pigging debris, solid deposit, produced fluid, produced water, injected water, swabs, etc. If "other" is specified, please propose entry in "additional info" field
     * [samp_vol_we_dna_ext](samp_vol_we_dna_ext.md) - Volume (ml), weight (g) of processed sample, or surface area swabbed from sample for DNA extraction
     * [samp_weather](samp_weather.md) - The weather on the sampling day
     * [samp_well_name](samp_well_name.md) - Name of the well (e.g. BXA1123) where sample was taken
     * [saturates_pc](saturates_pc.md) - Saturate, Aromatic, Resin and Asphaltene¬†(SARA) is an analysis method that divides¬†crude oil¬†components according to their polarizability and polarity. There are three main methods to obtain SARA results. The most popular one is known as the Iatroscan TLC-FID and is referred to as IP-143 (source: https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene)
     * [season](season.md) - The season when sampling occurred. Any of the four periods into which the year is divided by the equinoxes and solstices. This field accepts terms listed under season (http://purl.obolibrary.org/obo/NCIT_C94729).
     * [season_environment](season_environment.md) - Treatment involving an exposure to a particular season (e.g. Winter, summer, rabi, rainy etc.), treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment
     * [season_precpt](season_precpt.md) - The average of all seasonal precipitation values known, or an estimated equivalent value derived by such methods as regional indexes or Isohyetal maps.
     * [season_temp](season_temp.md) - Mean seasonal temperature
     * [season_use](season_use.md) - The seasons the space is occupied
     * [secondary_treatment](secondary_treatment.md) - The process for substantially degrading the biological content of the sewage
     * [sediment_type](sediment_type.md) - Information about the sediment type based on major constituents
     * [sewage_type](sewage_type.md) - Type of wastewater treatment plant as municipial or industrial
     * [sexual_act](sexual_act.md) - Current sexual partner and frequency of sex
     * [shading_device_cond](shading_device_cond.md) - The physical condition of the shading device at the time of sampling
     * [shading_device_loc](shading_device_loc.md) - The location of the shading device in relation to the built structure
     * [shading_device_mat](shading_device_mat.md) - The material the shading device is composed of
     * [shading_device_type](shading_device_type.md) - The type of shading device
     * [shading_device_water_mold](shading_device_water_mold.md) - Signs of the presence of mold or mildew on the shading device
     * [sieving](sieving.md) - Collection design of pooled samples and/or sieve size and amount of sample sieved
     * [silicate](silicate.md) - Concentration of silicate
     * [size_frac_low](size_frac_low.md) - Refers to the mesh/pore size used to pre-filter/pre-sort the sample. Materials larger than the size threshold are excluded from the sample
     * [size_frac_up](size_frac_up.md) - Refers to the mesh/pore size used to retain the sample. Materials smaller than the size threshold are excluded from the sample
     * [slope_aspect](slope_aspect.md) - The direction a slope faces. While looking down a slope use a compass to record the direction you are facing (direction or degrees); e.g., nw or 315 degrees. This measure provides an indication of sun and wind exposure that will influence soil temperature and evapotranspiration.
     * [slope_gradient](slope_gradient.md) - Commonly called 'slope'. The angle between ground surface and a horizontal line (in percent). This is the direction that overland water would flow. This measure is usually taken with a hand level meter or clinometer
     * [sludge_retent_time](sludge_retent_time.md) - The time activated sludge remains in reactor
     * [smoker](smoker.md) - Specification of smoking status
     * [sodium](sodium.md) - Sodium concentration in the sample
     * [soil_type](soil_type.md) - Soil series name or other lower-level classification
     * [soil_type_meth](soil_type_meth.md) - Reference or method used in determining soil series name or other lower-level classification
     * [solar_irradiance](solar_irradiance.md) - The amount of solar energy that arrives at a specific area of a surface during a specific time interval
     * [soluble_inorg_mat](soluble_inorg_mat.md) - Concentration of substances such as ammonia, road-salt, sea-salt, cyanide, hydrogen sulfide, thiocyanates, thiosulfates, etc.
     * [soluble_org_mat](soluble_org_mat.md) - Concentration of substances such as urea, fruit sugars, soluble proteins, drugs, pharmaceuticals, etc.
     * [soluble_react_phosp](soluble_react_phosp.md) - Concentration of soluble reactive phosphorus
     * [space_typ_state](space_typ_state.md) - Customary or normal state of the space
     * [special_diet](special_diet.md) - Specification of special diet; can include multiple special diets
     * [specific](specific.md) - The building specifications. If design is chosen, indicate phase: conceptual, schematic, design development, construction documents
     * [specific_humidity](specific_humidity.md) - The mass of water vapour in a unit mass of moist air, usually expressed as grams of vapour per kilogram of air, or, in air conditioning, as grains per pound.
     * [sr_dep_env](sr_dep_env.md) - Source rock depositional environment (https://en.wikipedia.org/wiki/Source_rock). If "other" is specified, please propose entry in "additional info" field
     * [sr_geol_age](sr_geol_age.md) - Geological age of source rock (Additional info: https://en.wikipedia.org/wiki/Period_(geology)). If "other" is specified, please propose entry in "additional info" field
     * [sr_kerog_type](sr_kerog_type.md) - Origin of kerogen. Type I: Algal (aquatic), Type II: planktonic and soft plant material (aquatic or terrestrial), Type III: terrestrial woody/ fibrous plant material (terrestrial), Type IV: oxidized recycled woody debris (terrestrial) (additional information: https://en.wikipedia.org/wiki/Kerogen). If "other" is specified, please propose entry in "additional info" field
     * [sr_lithology](sr_lithology.md) - Lithology of source rock (https://en.wikipedia.org/wiki/Source_rock). If "other" is specified, please propose entry in "additional info" field
     * [standing_water_regm](standing_water_regm.md) - Treatment involving an exposure to standing water during a plant's life span, types can be flood water or standing water, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens
     * [store_cond](store_cond.md) - Explain how and for how long the soil sample was stored before DNA extraction
     * [study_complt_stat](study_complt_stat.md) - Specification of study completion status, if no the reason should be specified
     * [substructure_type](substructure_type.md) - The substructure or under building is that largely hidden section of the building which is built off the foundations to the ground floor level
     * [sulfate](sulfate.md) - Concentration of sulfate in the sample
     * [sulfate_fw](sulfate_fw.md) - Original sulfate concentration in the hydrocarbon resource
     * [sulfide](sulfide.md) - Concentration of sulfide in the sample
     * [surf_air_cont](surf_air_cont.md) - Contaminant identified on surface
     * [surf_humidity](surf_humidity.md) - Surfaces: water activity as a function of air and material moisture
     * [surf_material](surf_material.md) - Surface materials at the point of sampling
     * [surf_moisture](surf_moisture.md) - Water held on a surface
     * [surf_moisture_ph](surf_moisture_ph.md) - ph measurement of surface
     * [surf_temp](surf_temp.md) - Temperature of the surface at the time of sampling
     * [suspend_part_matter](suspend_part_matter.md) - Concentration of suspended particulate matter
     * [suspend_solids](suspend_solids.md) - Concentration of substances including a wide variety of material, such as silt, decaying plant and animal matter; can include multiple substances
     * [tan](tan.md) - Total Acid Number¬†(TAN) is a measurement of acidity that is determined by the amount of¬†potassium hydroxide¬†in milligrams that is needed to neutralize the acids in one gram of oil.¬†It is an important quality measurement of¬†crude oil. (source: https://en.wikipedia.org/wiki/Total_acid_number)
     * [temp](temp.md) - Temperature of the sample at the time of sampling
     * [temp_out](temp_out.md) - The recorded temperature value at sampling time outside
     * [tertiary_treatment](tertiary_treatment.md) - The process providing a final treatment stage to raise the effluent quality before it is discharged to the receiving environment
     * [texture](texture.md) - The relative proportion of different grain sizes of mineral particles in a soil, as described using a standard system; express as % sand (50 um to 2 mm), silt (2 um to 50 um), and clay (<2 um) with textural name (e.g., silty clay loam) optional.
     * [texture_meth](texture_meth.md) - Reference or method used in determining soil texture
     * [tidal_stage](tidal_stage.md) - Stage of tide
     * [tillage](tillage.md) - Note method(s) used for tilling
     * [time_last_toothbrush](time_last_toothbrush.md) - Specification of the time since last toothbrushing
     * [time_since_last_wash](time_since_last_wash.md) - Specification of the time since last wash
     * [tiss_cult_growth_med](tiss_cult_growth_med.md) - Description of plant tissue culture growth media used
     * [toluene](toluene.md) - Concentration of toluene in the sample
     * [tot_carb](tot_carb.md) - Total carbon content
     * [tot_depth_water_col](tot_depth_water_col.md) - Measurement of total depth of water column
     * [tot_diss_nitro](tot_diss_nitro.md) - Total dissolved nitrogen concentration, reported as nitrogen, measured by: total dissolved nitrogen = NH4 + NO3NO2 + dissolved organic nitrogen
     * [tot_inorg_nitro](tot_inorg_nitro.md) - Total inorganic nitrogen content
     * [tot_iron](tot_iron.md) - Concentration of total iron in the sample
     * [tot_nitro](tot_nitro.md) - Total nitrogen concentration of water samples, calculated by: total nitrogen = total dissolved nitrogen + particulate nitrogen. Can also be measured without filtering, reported as nitrogen
     * [tot_nitro_content](tot_nitro_content.md) - Total nitrogen content of the sample
     * [tot_nitro_content_meth](tot_nitro_content_meth.md) - Reference or method used in determining the total nitrogen
     * [tot_org_c_meth](tot_org_c_meth.md) - Reference or method used in determining total organic carbon
     * [tot_org_carb](tot_org_carb.md) - Definition for soil: total organic carbon content of the soil, definition otherwise: total organic carbon content
     * [tot_part_carb](tot_part_carb.md) - Total particulate carbon content
     * [tot_phosp](tot_phosp.md) - Total phosphorus concentration in the sample, calculated by: total phosphorus = total dissolved phosphorus + particulate phosphorus
     * [tot_phosphate](tot_phosphate.md) - Total amount or concentration of phosphate
     * [tot_sulfur](tot_sulfur.md) - Concentration of total sulfur in the sample
     * [train_line](train_line.md) - The subway line name
     * [train_stat_loc](train_stat_loc.md) - The train station collection location
     * [train_stop_loc](train_stop_loc.md) - The train stop collection location
     * [travel_out_six_month](travel_out_six_month.md) - Specification of the countries travelled in the last six months; can include multiple travels
     * [turbidity](turbidity.md) - Measure of the amount of cloudiness or haziness in water caused by individual particles
     * [tvdss_of_hcr_pressure](tvdss_of_hcr_pressure.md) - True vertical depth subsea (TVDSS) of the hydrocarbon resource where the original pressure was measured (e.g. 1578 m )
     * [tvdss_of_hcr_temp](tvdss_of_hcr_temp.md) - True vertical depth subsea (TVDSS) of the hydrocarbon resource where the original temperature was measured (e.g. 1345 m)
     * [twin_sibling](twin_sibling.md) - Specification of twin sibling presence
     * [typ_occup_density](typ_occup_density.md) - Customary or normal density of occupants
     * [urine_collect_meth](urine_collect_meth.md) - Specification of urine collection method
     * [urogenit_disord](urogenit_disord.md) - History of urogenital disorders, can include multiple disorders. The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, reproductive system disease (https://disease-ontology.org/?id=DOID:15) or urinary system disease (https://disease-ontology.org/?id=DOID:18). 
     * [urogenit_tract_disor](urogenit_tract_disor.md) - History of urogenital tract disorders; can include multiple disorders. The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, urinary system disease (https://disease-ontology.org/?id=DOID:18). 
     * [ventilation_rate](ventilation_rate.md) - Ventilation rate of the system in the sampled premises
     * [ventilation_type](ventilation_type.md) - Ventilation system used in the sampled premises
     * [vfa](vfa.md) - Concentration of Volatile Fatty Acids in the sample
     * [vfa_fw](vfa_fw.md) - Original volatile fatty acid concentration in the hydrocarbon resource
     * [vis_media](vis_media.md) - The building visual media
     * [viscosity](viscosity.md) - A measure of oil's resistance¬†to gradual deformation by¬†shear stress¬†or¬†tensile stress (e.g. 3.5 cp; 100 ¬∞C)
     * [volatile_org_comp](volatile_org_comp.md) - Concentration of carbon-based chemicals that easily evaporate at room temperature; can report multiple volatile organic compounds by entering numeric values preceded by name of compound
     * [wall_area](wall_area.md) - The total area of the sampled room's walls
     * [wall_const_type](wall_const_type.md) - The building class of the wall defined by the composition of the building elements and fire-resistance rating.
     * [wall_finish_mat](wall_finish_mat.md) - The material utilized to finish the outer most layer of the wall
     * [wall_height](wall_height.md) - The average height of the walls in the sampled room
     * [wall_loc](wall_loc.md) - The relative location of the wall within the room
     * [wall_surf_treatment](wall_surf_treatment.md) - The surface treatment of interior wall
     * [wall_texture](wall_texture.md) - The feel, appearance, or consistency of a wall surface
     * [wall_thermal_mass](wall_thermal_mass.md) - The ability of the wall to provide inertia against temperature fluctuations. Generally this means concrete or concrete block that is either exposed or covered only with paint
     * [wall_water_mold](wall_water_mold.md) - Signs of the presence of mold or mildew on a wall
     * [wastewater_type](wastewater_type.md) - The origin of wastewater such as human waste, rainfall, storm drains, etc.
     * [water_content](water_content.md) - Water content measurement
     * [water_content_soil_meth](water_content_soil_meth.md) - Reference or method used in determining the water content of soil
     * [water_current](water_current.md) - Measurement of magnitude and direction of flow within a fluid
     * [water_cut](water_cut.md) - Current amount of water (%) in a produced fluid stream; or the average of the combined streams
     * [water_feat_size](water_feat_size.md) - The size of the water feature
     * [water_feat_type](water_feat_type.md) - The type of water feature present within the building being sampled
     * [water_production_rate](water_production_rate.md) - Water production rates per well (e.g. 987 m3 / day)
     * [water_temp_regm](water_temp_regm.md) - Information about treatment involving an exposure to water with varying degree of temperature, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens
     * [watering_regm](watering_regm.md) - Information about treatment involving an exposure to watering frequencies, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens
     * [weekday](weekday.md) - The day of the week when sampling occurred
     * [weight_loss_3_month](weight_loss_3_month.md) - Specification of weight loss in the last three months, if yes should be further specified to include amount of weight loss
     * [win](win.md) - A unique identifier of a well or wellbore. This is part of the Global Framework for Well Identification initiative which is compiled by the Professional Petroleum Data Management Association (PPDM) in an effort to improve well identification systems. (Supporting information: https://ppdm.org/ and http://dl.ppdm.org/dl/690)
     * [wind_direction](wind_direction.md) - Wind direction is the direction from which a wind originates
     * [wind_speed](wind_speed.md) - Speed of wind measured at the time of sampling
     * [window_cond](window_cond.md) - The physical condition of the window at the time of sampling
     * [window_cover](window_cover.md) - The type of window covering
     * [window_horiz_pos](window_horiz_pos.md) - The horizontal position of the window on the wall
     * [window_loc](window_loc.md) - The relative location of the window within the room
     * [window_mat](window_mat.md) - The type of material used to finish a window
     * [window_open_freq](window_open_freq.md) - The number of times windows are opened per week
     * [window_size](window_size.md) - The window's length and width
     * [window_status](window_status.md) - Defines whether the windows were open or closed during environmental testing
     * [window_type](window_type.md) - The type of windows
     * [window_vert_pos](window_vert_pos.md) - The vertical position of the window on the wall
     * [window_water_mold](window_water_mold.md) - Signs of the presence of mold or mildew on the window.
     * [xylene](xylene.md) - Concentration of xylene in the sample
 * [investigation field](investigation_field.md) - field describing aspect of the investigation/study to which the sample belongs
     * [experimental_factor](experimental_factor.md) - Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI
         * [ME➞experimental_factor](ME_experimental_factor.md)
         * [MIGS bacteria➞experimental_factor](MIGS_bacteria_experimental_factor.md)
         * [MIGS eukaryote➞experimental_factor](MIGS_eukaryote_experimental_factor.md)
         * [MIGS org➞experimental_factor](MIGS_org_experimental_factor.md)
         * [MIGS plant➞experimental_factor](MIGS_plant_experimental_factor.md)
         * [MIGS virus➞experimental_factor](MIGS_virus_experimental_factor.md)
         * [MIMAG➞experimental_factor](MIMAG_experimental_factor.md)
         * [MIMARKS specimen➞experimental_factor](MIMARKS_specimen_experimental_factor.md)
         * [MIMARKS survey➞experimental_factor](MIMARKS_survey_experimental_factor.md)
         * [MISAG➞experimental_factor](MISAG_experimental_factor.md)
         * [MIUVIG➞experimental_factor](MIUVIG_experimental_factor.md)
     * [investigation_type](investigation_type.md) - Nucleic Acid Sequence Report is the root element of all MIGS/MIMS compliant reports as standardized by Genomic Standards Consortium. This field is either eukaryote,bacteria,virus,plasmid,organelle, metagenome,mimarks-survey, mimarks-specimen, metatranscriptome, single amplified genome, metagenome-assembled genome, or uncultivated viral genome
         * [ME➞investigation_type](ME_investigation_type.md)
         * [MIGS bacteria➞investigation_type](MIGS_bacteria_investigation_type.md)
         * [MIGS eukaryote➞investigation_type](MIGS_eukaryote_investigation_type.md)
         * [MIGS org➞investigation_type](MIGS_org_investigation_type.md)
         * [MIGS plant➞investigation_type](MIGS_plant_investigation_type.md)
         * [MIGS virus➞investigation_type](MIGS_virus_investigation_type.md)
         * [MIMAG➞investigation_type](MIMAG_investigation_type.md)
         * [MIMARKS specimen➞investigation_type](MIMARKS_specimen_investigation_type.md)
         * [MIMARKS survey➞investigation_type](MIMARKS_survey_investigation_type.md)
         * [MISAG➞investigation_type](MISAG_investigation_type.md)
         * [MIUVIG➞investigation_type](MIUVIG_investigation_type.md)
     * [project_name](project_name.md) - Name of the project within which the sequencing was organized
         * [ME➞project_name](ME_project_name.md)
         * [MIGS bacteria➞project_name](MIGS_bacteria_project_name.md)
         * [MIGS eukaryote➞project_name](MIGS_eukaryote_project_name.md)
         * [MIGS org➞project_name](MIGS_org_project_name.md)
         * [MIGS plant➞project_name](MIGS_plant_project_name.md)
         * [MIGS virus➞project_name](MIGS_virus_project_name.md)
         * [MIMAG➞project_name](MIMAG_project_name.md)
         * [MIMARKS specimen➞project_name](MIMARKS_specimen_project_name.md)
         * [MIMARKS survey➞project_name](MIMARKS_survey_project_name.md)
         * [MISAG➞project_name](MISAG_project_name.md)
         * [MIUVIG➞project_name](MIUVIG_project_name.md)
     * [submitted_to_insdc](submitted_to_insdc.md) - Depending on the study (large-scale e.g. done with next generation sequencing technology, or small-scale) sequences have to be submitted to SRA (Sequence Read Archive), DRA (DDBJ Read Archive) or via the classical Webin/Sequin systems to Genbank, ENA and DDBJ. Although this field is mandatory, it is meant as a self-test field, therefore it is not necessary to include this field in contextual data submitted to databases
         * [ME➞submitted_to_insdc](ME_submitted_to_insdc.md)
         * [MIGS bacteria➞submitted_to_insdc](MIGS_bacteria_submitted_to_insdc.md)
         * [MIGS eukaryote➞submitted_to_insdc](MIGS_eukaryote_submitted_to_insdc.md)
         * [MIGS org➞submitted_to_insdc](MIGS_org_submitted_to_insdc.md)
         * [MIGS plant➞submitted_to_insdc](MIGS_plant_submitted_to_insdc.md)
         * [MIGS virus➞submitted_to_insdc](MIGS_virus_submitted_to_insdc.md)
         * [MIMAG➞submitted_to_insdc](MIMAG_submitted_to_insdc.md)
         * [MIMARKS specimen➞submitted_to_insdc](MIMARKS_specimen_submitted_to_insdc.md)
         * [MIMARKS survey➞submitted_to_insdc](MIMARKS_survey_submitted_to_insdc.md)
         * [MISAG➞submitted_to_insdc](MISAG_submitted_to_insdc.md)
         * [MIUVIG➞submitted_to_insdc](MIUVIG_submitted_to_insdc.md)
 * [mixs extension field](mixs_extension_field.md)
     * [env_package](env_package.md) - MIxS extension for reporting of measurements and observations obtained from one or more of the environments where the sample was obtained. All environmental packages listed here are further defined in separate subtables. By giving the name of the environmental package, a selection of fields can be made from the subtables and can be reported
         * [ME➞env_package](ME_env_package.md)
         * [MIGS bacteria➞env_package](MIGS_bacteria_env_package.md)
         * [MIGS eukaryote➞env_package](MIGS_eukaryote_env_package.md)
         * [MIGS org➞env_package](MIGS_org_env_package.md)
         * [MIGS plant➞env_package](MIGS_plant_env_package.md)
         * [MIGS virus➞env_package](MIGS_virus_env_package.md)
         * [MIMAG➞env_package](MIMAG_env_package.md)
         * [MIMARKS specimen➞env_package](MIMARKS_specimen_env_package.md)
         * [MIMARKS survey➞env_package](MIMARKS_survey_env_package.md)
         * [MISAG➞env_package](MISAG_env_package.md)
         * [MIUVIG➞env_package](MIUVIG_env_package.md)
 * [nucleic acid sequence source field](nucleic_acid_sequence_source_field.md)
     * [biotic_relationship](biotic_relationship.md) - Description of relationship(s) between the subject organism and other organism(s) it is associated with. E.g., parasite on species X; mutualist with species Y. The target organism is the subject of the relationship, and the other organism(s) is the object
         * [MIGS bacteria➞biotic_relationship](MIGS_bacteria_biotic_relationship.md)
         * [MIGS eukaryote➞biotic_relationship](MIGS_eukaryote_biotic_relationship.md)
         * [MIGS virus➞biotic_relationship](MIGS_virus_biotic_relationship.md)
         * [MIMARKS survey➞biotic_relationship](MIMARKS_survey_biotic_relationship.md)
         * [MIUVIG➞biotic_relationship](MIUVIG_biotic_relationship.md)
     * [encoded_traits](encoded_traits.md) - Should include key traits like antibiotic resistance or xenobiotic degradation phenotypes for plasmids, converting genes for phage
         * [MIGS bacteria➞encoded_traits](MIGS_bacteria_encoded_traits.md)
         * [MIGS plant➞encoded_traits](MIGS_plant_encoded_traits.md)
         * [MIGS virus➞encoded_traits](MIGS_virus_encoded_traits.md)
     * [estimated_size](estimated_size.md) - The estimated size of the genome prior to sequencing. Of particular importance in the sequencing of (eukaryotic) genome which could remain in draft form for a long or unspecified period.
         * [MIGS bacteria➞estimated_size](MIGS_bacteria_estimated_size.md)
         * [MIGS eukaryote➞estimated_size](MIGS_eukaryote_estimated_size.md)
         * [MIGS org➞estimated_size](MIGS_org_estimated_size.md)
         * [MIGS plant➞estimated_size](MIGS_plant_estimated_size.md)
         * [MIGS virus➞estimated_size](MIGS_virus_estimated_size.md)
         * [MIUVIG➞estimated_size](MIUVIG_estimated_size.md)
     * [extrachrom_elements](extrachrom_elements.md) - Do plasmids exist of significant phenotypic consequence (e.g. ones that determine virulence or antibiotic resistance). Megaplasmids? Other plasmids (borrelia has 15+ plasmids)
         * [MIGS bacteria➞extrachrom_elements](MIGS_bacteria_extrachrom_elements.md)
         * [MIGS eukaryote➞extrachrom_elements](MIGS_eukaryote_extrachrom_elements.md)
         * [MIGS org➞extrachrom_elements](MIGS_org_extrachrom_elements.md)
         * [MIMARKS survey➞extrachrom_elements](MIMARKS_survey_extrachrom_elements.md)
     * [health_disease_stat](health_disease_stat.md) - Health or disease status of specific host at time of collection
         * [MIGS bacteria➞health_disease_stat](MIGS_bacteria_health_disease_stat.md)
         * [MIGS eukaryote➞health_disease_stat](MIGS_eukaryote_health_disease_stat.md)
         * [MIGS virus➞health_disease_stat](MIGS_virus_health_disease_stat.md)
     * [host_spec_range](host_spec_range.md) - The NCBI taxonomy identifier of the specific host if it is known
         * [MIGS bacteria➞host_spec_range](MIGS_bacteria_host_spec_range.md)
         * [MIGS eukaryote➞host_spec_range](MIGS_eukaryote_host_spec_range.md)
         * [MIGS plant➞host_spec_range](MIGS_plant_host_spec_range.md)
         * [MIGS virus➞host_spec_range](MIGS_virus_host_spec_range.md)
         * [MIUVIG➞host_spec_range](MIUVIG_host_spec_range.md)
     * [isol_growth_condt](isol_growth_condt.md) - Publication reference in the form of pubmed ID (pmid), digital object identifier (doi) or url for isolation and growth condition specifications of the organism/material
         * [MIGS bacteria➞isol_growth_condt](MIGS_bacteria_isol_growth_condt.md)
         * [MIGS eukaryote➞isol_growth_condt](MIGS_eukaryote_isol_growth_condt.md)
         * [MIGS org➞isol_growth_condt](MIGS_org_isol_growth_condt.md)
         * [MIGS plant➞isol_growth_condt](MIGS_plant_isol_growth_condt.md)
         * [MIGS virus➞isol_growth_condt](MIGS_virus_isol_growth_condt.md)
         * [MIMARKS survey➞isol_growth_condt](MIMARKS_survey_isol_growth_condt.md)
     * [num_replicons](num_replicons.md) - Reports the number of replicons in a nuclear genome of eukaryotes, in the genome of a bacterium or archaea or the number of segments in a segmented virus. Always applied to the haploid chromosome count of a eukaryote
         * [MIGS bacteria➞num_replicons](MIGS_bacteria_num_replicons.md)
         * [MIGS eukaryote➞num_replicons](MIGS_eukaryote_num_replicons.md)
         * [MIGS virus➞num_replicons](MIGS_virus_num_replicons.md)
     * [pathogenicity](pathogenicity.md) - To what is the entity pathogenic
         * [MIGS bacteria➞pathogenicity](MIGS_bacteria_pathogenicity.md)
         * [MIGS eukaryote➞pathogenicity](MIGS_eukaryote_pathogenicity.md)
         * [MIGS virus➞pathogenicity](MIGS_virus_pathogenicity.md)
         * [MIUVIG➞pathogenicity](MIUVIG_pathogenicity.md)
     * [ploidy](ploidy.md) - The ploidy level of the genome (e.g. allopolyploid, haploid, diploid, triploid, tetraploid). It has implications for the downstream study of duplicated gene and regions of the genomes (and perhaps for difficulties in assembly). For terms, please select terms listed under class ploidy (PATO:001374) of Phenotypic Quality Ontology (PATO), and for a browser of PATO (v 2018-03-27) please refer to http://purl.bioontology.org/ontology/PATO
         * [MIGS eukaryote➞ploidy](MIGS_eukaryote_ploidy.md)
     * [propagation](propagation.md) - This field is specific to different taxa. For phages: lytic/lysogenic, for plasmids: incompatibility group, for eukaryotes: sexual/asexual (Note: there is the strong opinion to name phage propagation obligately lytic or temperate, therefore we also give this choice
         * [MIGS eukaryote➞propagation](MIGS_eukaryote_propagation.md)
         * [MIGS plant➞propagation](MIGS_plant_propagation.md)
         * [MIGS virus➞propagation](MIGS_virus_propagation.md)
     * [ref_biomaterial](ref_biomaterial.md) - Primary publication if isolated before genome publication; otherwise, primary genome report
         * [ME➞ref_biomaterial](ME_ref_biomaterial.md)
         * [MIGS bacteria➞ref_biomaterial](MIGS_bacteria_ref_biomaterial.md)
         * [MIGS eukaryote➞ref_biomaterial](MIGS_eukaryote_ref_biomaterial.md)
         * [MIGS org➞ref_biomaterial](MIGS_org_ref_biomaterial.md)
         * [MIGS plant➞ref_biomaterial](MIGS_plant_ref_biomaterial.md)
         * [MIGS virus➞ref_biomaterial](MIGS_virus_ref_biomaterial.md)
         * [MIMAG➞ref_biomaterial](MIMAG_ref_biomaterial.md)
         * [MISAG➞ref_biomaterial](MISAG_ref_biomaterial.md)
         * [MIUVIG➞ref_biomaterial](MIUVIG_ref_biomaterial.md)
     * [rel_to_oxygen](rel_to_oxygen.md) - Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments
         * [ME➞rel_to_oxygen](ME_rel_to_oxygen.md)
         * [MIGS bacteria➞rel_to_oxygen](MIGS_bacteria_rel_to_oxygen.md)
         * [MIMAG➞rel_to_oxygen](MIMAG_rel_to_oxygen.md)
         * [MIMARKS specimen➞rel_to_oxygen](MIMARKS_specimen_rel_to_oxygen.md)
         * [MIMARKS survey➞rel_to_oxygen](MIMARKS_survey_rel_to_oxygen.md)
         * [MISAG➞rel_to_oxygen](MISAG_rel_to_oxygen.md)
     * [samp_collect_device](samp_collect_device.md) - The method or device employed for collecting the sample
         * [ME➞samp_collect_device](ME_samp_collect_device.md)
         * [MIGS bacteria➞samp_collect_device](MIGS_bacteria_samp_collect_device.md)
         * [MIGS eukaryote➞samp_collect_device](MIGS_eukaryote_samp_collect_device.md)
         * [MIGS org➞samp_collect_device](MIGS_org_samp_collect_device.md)
         * [MIGS plant➞samp_collect_device](MIGS_plant_samp_collect_device.md)
         * [MIGS virus➞samp_collect_device](MIGS_virus_samp_collect_device.md)
         * [MIMAG➞samp_collect_device](MIMAG_samp_collect_device.md)
         * [MIMARKS specimen➞samp_collect_device](MIMARKS_specimen_samp_collect_device.md)
         * [MIMARKS survey➞samp_collect_device](MIMARKS_survey_samp_collect_device.md)
         * [MISAG➞samp_collect_device](MISAG_samp_collect_device.md)
         * [MIUVIG➞samp_collect_device](MIUVIG_samp_collect_device.md)
     * [samp_mat_process](samp_mat_process.md) - Any processing applied to the sample during or after retrieving the sample from environment. This field accepts OBI, for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI
         * [ME➞samp_mat_process](ME_samp_mat_process.md)
         * [MIGS bacteria➞samp_mat_process](MIGS_bacteria_samp_mat_process.md)
         * [MIGS eukaryote➞samp_mat_process](MIGS_eukaryote_samp_mat_process.md)
         * [MIGS org➞samp_mat_process](MIGS_org_samp_mat_process.md)
         * [MIGS plant➞samp_mat_process](MIGS_plant_samp_mat_process.md)
         * [MIGS virus➞samp_mat_process](MIGS_virus_samp_mat_process.md)
         * [MIMAG➞samp_mat_process](MIMAG_samp_mat_process.md)
         * [MIMARKS specimen➞samp_mat_process](MIMARKS_specimen_samp_mat_process.md)
         * [MIMARKS survey➞samp_mat_process](MIMARKS_survey_samp_mat_process.md)
         * [MISAG➞samp_mat_process](MISAG_samp_mat_process.md)
         * [MIUVIG➞samp_mat_process](MIUVIG_samp_mat_process.md)
     * [samp_size](samp_size.md) - Amount or size of sample (volume, mass or area) that was collected
         * [ME➞samp_size](ME_samp_size.md)
         * [MIGS bacteria➞samp_size](MIGS_bacteria_samp_size.md)
         * [MIGS eukaryote➞samp_size](MIGS_eukaryote_samp_size.md)
         * [MIGS org➞samp_size](MIGS_org_samp_size.md)
         * [MIGS plant➞samp_size](MIGS_plant_samp_size.md)
         * [MIGS virus➞samp_size](MIGS_virus_samp_size.md)
         * [MIMAG➞samp_size](MIMAG_samp_size.md)
         * [MIMARKS specimen➞samp_size](MIMARKS_specimen_samp_size.md)
         * [MIMARKS survey➞samp_size](MIMARKS_survey_samp_size.md)
         * [MISAG➞samp_size](MISAG_samp_size.md)
         * [MIUVIG➞samp_size](MIUVIG_samp_size.md)
     * [size_frac](size_frac.md) - Filtering pore size used in sample preparation
         * [ME➞size_frac](ME_size_frac.md)
         * [MIMAG➞size_frac](MIMAG_size_frac.md)
         * [MIMARKS specimen➞size_frac](MIMARKS_specimen_size_frac.md)
         * [MISAG➞size_frac](MISAG_size_frac.md)
         * [MIUVIG➞size_frac](MIUVIG_size_frac.md)
     * [source_mat_id](source_mat_id.md) - A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).
         * [ME➞source_mat_id](ME_source_mat_id.md)
         * [MIGS bacteria➞source_mat_id](MIGS_bacteria_source_mat_id.md)
         * [MIGS eukaryote➞source_mat_id](MIGS_eukaryote_source_mat_id.md)
         * [MIGS org➞source_mat_id](MIGS_org_source_mat_id.md)
         * [MIGS plant➞source_mat_id](MIGS_plant_source_mat_id.md)
         * [MIGS virus➞source_mat_id](MIGS_virus_source_mat_id.md)
         * [MIMAG➞source_mat_id](MIMAG_source_mat_id.md)
         * [MIMARKS specimen➞source_mat_id](MIMARKS_specimen_source_mat_id.md)
         * [MIMARKS survey➞source_mat_id](MIMARKS_survey_source_mat_id.md)
         * [MISAG➞source_mat_id](MISAG_source_mat_id.md)
         * [MIUVIG➞source_mat_id](MIUVIG_source_mat_id.md)
     * [source_uvig](source_uvig.md) - Type of dataset from which the UViG was obtained
         * [MIUVIG➞source_uvig](MIUVIG_source_uvig.md)
     * [specific_host](specific_host.md) - If there is a host involved, please provide its taxid (or environmental if not actually isolated from the dead or alive host - i.e. a pathogen could be isolated from a swipe of a bench etc) and report whether it is a laboratory or natural host)
         * [MIGS bacteria➞specific_host](MIGS_bacteria_specific_host.md)
         * [MIGS eukaryote➞specific_host](MIGS_eukaryote_specific_host.md)
         * [MIGS plant➞specific_host](MIGS_plant_specific_host.md)
         * [MIGS virus➞specific_host](MIGS_virus_specific_host.md)
         * [MIUVIG➞specific_host](MIUVIG_specific_host.md)
     * [subspecf_gen_lin](subspecf_gen_lin.md) - This should provide further information about the genetic distinctness of the sequenced organism by recording additional information e.g. serovar, serotype, biotype, ecotype, or any relevant genetic typing schemes like Group I plasmid. It can also contain alternative taxonomic information. It should contain both the lineage name, and the lineage rank, i.e. biovar:abc123
         * [MIGS bacteria➞subspecf_gen_lin](MIGS_bacteria_subspecf_gen_lin.md)
         * [MIGS eukaryote➞subspecf_gen_lin](MIGS_eukaryote_subspecf_gen_lin.md)
         * [MIGS org➞subspecf_gen_lin](MIGS_org_subspecf_gen_lin.md)
         * [MIGS plant➞subspecf_gen_lin](MIGS_plant_subspecf_gen_lin.md)
         * [MIGS virus➞subspecf_gen_lin](MIGS_virus_subspecf_gen_lin.md)
         * [MIMARKS survey➞subspecf_gen_lin](MIMARKS_survey_subspecf_gen_lin.md)
     * [trophic_level](trophic_level.md) - Trophic levels are the feeding position in a food chain. Microbes can be a range of producers (e.g. chemolithotroph)
         * [MIGS bacteria➞trophic_level](MIGS_bacteria_trophic_level.md)
         * [MIGS eukaryote➞trophic_level](MIGS_eukaryote_trophic_level.md)
         * [MIMARKS survey➞trophic_level](MIMARKS_survey_trophic_level.md)
     * [virus_enrich_appr](virus_enrich_appr.md) - List of approaches used to enrich the sample for viruses, if any
         * [MIGS virus➞virus_enrich_appr](MIGS_virus_virus_enrich_appr.md)
         * [MIUVIG➞virus_enrich_appr](MIUVIG_virus_enrich_appr.md)
 * [➞has numeric value](quantityValue__has_numeric_value.md)
 * [➞has raw value](quantityValue__has_raw_value.md)
 * [➞has unit](quantityValue__has_unit.md) - Example "m"
 * [sequencing field](sequencing_field.md)
     * [adapters](adapters.md) - Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters
         * [ME➞adapters](ME_adapters.md)
         * [MIGS bacteria➞adapters](MIGS_bacteria_adapters.md)
         * [MIGS eukaryote➞adapters](MIGS_eukaryote_adapters.md)
         * [MIGS org➞adapters](MIGS_org_adapters.md)
         * [MIGS plant➞adapters](MIGS_plant_adapters.md)
         * [MIGS virus➞adapters](MIGS_virus_adapters.md)
         * [MIMAG➞adapters](MIMAG_adapters.md)
         * [MIMARKS specimen➞adapters](MIMARKS_specimen_adapters.md)
         * [MISAG➞adapters](MISAG_adapters.md)
         * [MIUVIG➞adapters](MIUVIG_adapters.md)
     * [annot](annot.md) - Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter
         * [ME➞annot](ME_annot.md)
         * [MIGS bacteria➞annot](MIGS_bacteria_annot.md)
         * [MIGS eukaryote➞annot](MIGS_eukaryote_annot.md)
         * [MIGS org➞annot](MIGS_org_annot.md)
         * [MIGS plant➞annot](MIGS_plant_annot.md)
         * [MIGS virus➞annot](MIGS_virus_annot.md)
         * [MIMAG➞annot](MIMAG_annot.md)
         * [MISAG➞annot](MISAG_annot.md)
         * [MIUVIG➞annot](MIUVIG_annot.md)
     * [assembly_name](assembly_name.md) - Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community
         * [ME➞assembly_name](ME_assembly_name.md)
         * [MIGS bacteria➞assembly_name](MIGS_bacteria_assembly_name.md)
         * [MIGS eukaryote➞assembly_name](MIGS_eukaryote_assembly_name.md)
         * [MIGS org➞assembly_name](MIGS_org_assembly_name.md)
         * [MIGS plant➞assembly_name](MIGS_plant_assembly_name.md)
         * [MIGS virus➞assembly_name](MIGS_virus_assembly_name.md)
         * [MIMAG➞assembly_name](MIMAG_assembly_name.md)
         * [MISAG➞assembly_name](MISAG_assembly_name.md)
         * [MIUVIG➞assembly_name](MIUVIG_assembly_name.md)
     * [assembly_qual](assembly_qual.md) - The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling < 90% of the expected genome or replicon sequence, or for which no genome size could be estimated
         * [ME➞assembly_qual](ME_assembly_qual.md)
         * [MIGS bacteria➞assembly_qual](MIGS_bacteria_assembly_qual.md)
         * [MIGS eukaryote➞assembly_qual](MIGS_eukaryote_assembly_qual.md)
         * [MIGS org➞assembly_qual](MIGS_org_assembly_qual.md)
         * [MIGS plant➞assembly_qual](MIGS_plant_assembly_qual.md)
         * [MIGS virus➞assembly_qual](MIGS_virus_assembly_qual.md)
         * [MIMAG➞assembly_qual](MIMAG_assembly_qual.md)
         * [MISAG➞assembly_qual](MISAG_assembly_qual.md)
         * [MIUVIG➞assembly_qual](MIUVIG_assembly_qual.md)
     * [assembly_software](assembly_software.md) - Tool(s) used for assembly, including version number and parameters
         * [ME➞assembly_software](ME_assembly_software.md)
         * [MIGS bacteria➞assembly_software](MIGS_bacteria_assembly_software.md)
         * [MIGS eukaryote➞assembly_software](MIGS_eukaryote_assembly_software.md)
         * [MIGS org➞assembly_software](MIGS_org_assembly_software.md)
         * [MIGS plant➞assembly_software](MIGS_plant_assembly_software.md)
         * [MIGS virus➞assembly_software](MIGS_virus_assembly_software.md)
         * [MIMAG➞assembly_software](MIMAG_assembly_software.md)
         * [MIMARKS specimen➞assembly_software](MIMARKS_specimen_assembly_software.md)
         * [MISAG➞assembly_software](MISAG_assembly_software.md)
         * [MIUVIG➞assembly_software](MIUVIG_assembly_software.md)
     * [bin_param](bin_param.md) - The parameters that have been applied during the extraction of genomes from metagenomic datasets
         * [MIMAG➞bin_param](MIMAG_bin_param.md)
         * [MIUVIG➞bin_param](MIUVIG_bin_param.md)
     * [bin_software](bin_software.md) - Tool(s) used for the extraction of genomes from metagenomic datasets
         * [MIMAG➞bin_software](MIMAG_bin_software.md)
         * [MIUVIG➞bin_software](MIUVIG_bin_software.md)
     * [chimera_check](chimera_check.md) - A chimeric sequence, or chimera for short, is a sequence comprised of two or more phylogenetically distinct parent sequences. Chimeras are usually PCR artifacts thought to occur when a prematurely terminated amplicon reanneals to a foreign DNA strand and is copied to completion in the following PCR cycles. The point at which the chimeric sequence changes from one parent to the next is called the breakpoint or conversion point
         * [MIMARKS specimen➞chimera_check](MIMARKS_specimen_chimera_check.md)
         * [MIMARKS survey➞chimera_check](MIMARKS_survey_chimera_check.md)
     * [compl_appr](compl_appr.md) - The approach used to determine the completeness of a given SAG or MAG, which would typically make use of a set of conserved marker genes or a closely related reference genome. For UViG completeness, include reference genome or group used, and contig feature suggesting a complete genome
         * [MIMAG➞compl_appr](MIMAG_compl_appr.md)
         * [MISAG➞compl_appr](MISAG_compl_appr.md)
         * [MIUVIG➞compl_appr](MIUVIG_compl_appr.md)
     * [compl_score](compl_score.md) - Completeness score is typically based on either the fraction of markers found as compared to a database or the percent of a genome found as compared to a closely related reference genome. High Quality Draft: >90%, Medium Quality Draft: >50%, and Low Quality Draft: < 50% should have the indicated completeness scores
         * [MIGS bacteria➞compl_score](MIGS_bacteria_compl_score.md)
         * [MIGS eukaryote➞compl_score](MIGS_eukaryote_compl_score.md)
         * [MIGS org➞compl_score](MIGS_org_compl_score.md)
         * [MIGS plant➞compl_score](MIGS_plant_compl_score.md)
         * [MIGS virus➞compl_score](MIGS_virus_compl_score.md)
         * [MIMAG➞compl_score](MIMAG_compl_score.md)
         * [MISAG➞compl_score](MISAG_compl_score.md)
         * [MIUVIG➞compl_score](MIUVIG_compl_score.md)
     * [compl_software](compl_software.md) - Tools used for completion estimate, i.e. checkm, anvi'o, busco
         * [MIGS bacteria➞compl_software](MIGS_bacteria_compl_software.md)
         * [MIGS eukaryote➞compl_software](MIGS_eukaryote_compl_software.md)
         * [MIGS org➞compl_software](MIGS_org_compl_software.md)
         * [MIGS plant➞compl_software](MIGS_plant_compl_software.md)
         * [MIGS virus➞compl_software](MIGS_virus_compl_software.md)
         * [MIMAG➞compl_software](MIMAG_compl_software.md)
         * [MISAG➞compl_software](MISAG_compl_software.md)
         * [MIUVIG➞compl_software](MIUVIG_compl_software.md)
     * [contam_score](contam_score.md) - The contamination score is based on the fraction of single-copy genes that are observed more than once in a query genome. The following scores are acceptable for; High Quality Draft: < 5%, Medium Quality Draft: < 10%, Low Quality Draft: < 10%. Contamination must be below 5% for a SAG or MAG to be deposited into any of the public databases
         * [MIMAG➞contam_score](MIMAG_contam_score.md)
         * [MISAG➞contam_score](MISAG_contam_score.md)
     * [contam_screen_input](contam_screen_input.md) - The type of sequence data used as input
         * [MIMAG➞contam_screen_input](MIMAG_contam_screen_input.md)
         * [MISAG➞contam_screen_input](MISAG_contam_screen_input.md)
     * [contam_screen_param](contam_screen_param.md) - Specific parameters used in the decontamination sofware, such as reference database, coverage, and kmers. Combinations of these parameters may also be used, i.e. kmer and coverage, or reference database and kmer
         * [MIMAG➞contam_screen_param](MIMAG_contam_screen_param.md)
         * [MISAG➞contam_screen_param](MISAG_contam_screen_param.md)
     * [decontam_software](decontam_software.md) - Tool(s) used in contamination screening
         * [MIMAG➞decontam_software](MIMAG_decontam_software.md)
         * [MISAG➞decontam_software](MISAG_decontam_software.md)
     * [detec_type](detec_type.md) - Type of UViG detection
         * [MIUVIG➞detec_type](MIUVIG_detec_type.md)
     * [feat_pred](feat_pred.md) - Method used to predict UViGs features such as ORFs, integration site, etc.
         * [ME➞feat_pred](ME_feat_pred.md)
         * [MIGS bacteria➞feat_pred](MIGS_bacteria_feat_pred.md)
         * [MIGS eukaryote➞feat_pred](MIGS_eukaryote_feat_pred.md)
         * [MIGS org➞feat_pred](MIGS_org_feat_pred.md)
         * [MIGS plant➞feat_pred](MIGS_plant_feat_pred.md)
         * [MIGS virus➞feat_pred](MIGS_virus_feat_pred.md)
         * [MIMAG➞feat_pred](MIMAG_feat_pred.md)
         * [MISAG➞feat_pred](MISAG_feat_pred.md)
         * [MIUVIG➞feat_pred](MIUVIG_feat_pred.md)
     * [host_pred_appr](host_pred_appr.md) - Tool or approach used for host prediction
         * [MIUVIG➞host_pred_appr](MIUVIG_host_pred_appr.md)
     * [host_pred_est_acc](host_pred_est_acc.md) - For each tool or approach used for host prediction, estimated false discovery rates should be included, either computed de novo or from the literature
         * [MIUVIG➞host_pred_est_acc](MIUVIG_host_pred_est_acc.md)
     * [lib_layout](lib_layout.md) - Specify whether to expect single, paired, or other configuration of reads
         * [ME➞lib_layout](ME_lib_layout.md)
         * [MIGS bacteria➞lib_layout](MIGS_bacteria_lib_layout.md)
         * [MIGS eukaryote➞lib_layout](MIGS_eukaryote_lib_layout.md)
         * [MIGS org➞lib_layout](MIGS_org_lib_layout.md)
         * [MIGS plant➞lib_layout](MIGS_plant_lib_layout.md)
         * [MIGS virus➞lib_layout](MIGS_virus_lib_layout.md)
         * [MIMAG➞lib_layout](MIMAG_lib_layout.md)
         * [MIMARKS specimen➞lib_layout](MIMARKS_specimen_lib_layout.md)
         * [MISAG➞lib_layout](MISAG_lib_layout.md)
         * [MIUVIG➞lib_layout](MIUVIG_lib_layout.md)
     * [lib_reads_seqd](lib_reads_seqd.md) - Total number of clones sequenced from the library
         * [ME➞lib_reads_seqd](ME_lib_reads_seqd.md)
         * [MIGS bacteria➞lib_reads_seqd](MIGS_bacteria_lib_reads_seqd.md)
         * [MIGS eukaryote➞lib_reads_seqd](MIGS_eukaryote_lib_reads_seqd.md)
         * [MIGS org➞lib_reads_seqd](MIGS_org_lib_reads_seqd.md)
         * [MIGS plant➞lib_reads_seqd](MIGS_plant_lib_reads_seqd.md)
         * [MIGS virus➞lib_reads_seqd](MIGS_virus_lib_reads_seqd.md)
         * [MIMAG➞lib_reads_seqd](MIMAG_lib_reads_seqd.md)
         * [MIMARKS specimen➞lib_reads_seqd](MIMARKS_specimen_lib_reads_seqd.md)
         * [MISAG➞lib_reads_seqd](MISAG_lib_reads_seqd.md)
         * [MIUVIG➞lib_reads_seqd](MIUVIG_lib_reads_seqd.md)
     * [lib_screen](lib_screen.md) - Specific enrichment or screening methods applied before and/or after creating libraries
         * [ME➞lib_screen](ME_lib_screen.md)
         * [MIGS bacteria➞lib_screen](MIGS_bacteria_lib_screen.md)
         * [MIGS eukaryote➞lib_screen](MIGS_eukaryote_lib_screen.md)
         * [MIGS org➞lib_screen](MIGS_org_lib_screen.md)
         * [MIGS plant➞lib_screen](MIGS_plant_lib_screen.md)
         * [MIGS virus➞lib_screen](MIGS_virus_lib_screen.md)
         * [MIMAG➞lib_screen](MIMAG_lib_screen.md)
         * [MIMARKS specimen➞lib_screen](MIMARKS_specimen_lib_screen.md)
         * [MISAG➞lib_screen](MISAG_lib_screen.md)
         * [MIUVIG➞lib_screen](MIUVIG_lib_screen.md)
     * [lib_size](lib_size.md) - Total number of clones in the library prepared for the project
         * [ME➞lib_size](ME_lib_size.md)
         * [MIGS bacteria➞lib_size](MIGS_bacteria_lib_size.md)
         * [MIGS eukaryote➞lib_size](MIGS_eukaryote_lib_size.md)
         * [MIGS org➞lib_size](MIGS_org_lib_size.md)
         * [MIGS plant➞lib_size](MIGS_plant_lib_size.md)
         * [MIGS virus➞lib_size](MIGS_virus_lib_size.md)
         * [MIMAG➞lib_size](MIMAG_lib_size.md)
         * [MIMARKS specimen➞lib_size](MIMARKS_specimen_lib_size.md)
         * [MISAG➞lib_size](MISAG_lib_size.md)
         * [MIUVIG➞lib_size](MIUVIG_lib_size.md)
     * [lib_vector](lib_vector.md) - Cloning vector type(s) used in construction of libraries
         * [ME➞lib_vector](ME_lib_vector.md)
         * [MIGS bacteria➞lib_vector](MIGS_bacteria_lib_vector.md)
         * [MIGS eukaryote➞lib_vector](MIGS_eukaryote_lib_vector.md)
         * [MIGS org➞lib_vector](MIGS_org_lib_vector.md)
         * [MIGS plant➞lib_vector](MIGS_plant_lib_vector.md)
         * [MIGS virus➞lib_vector](MIGS_virus_lib_vector.md)
         * [MIMAG➞lib_vector](MIMAG_lib_vector.md)
         * [MIMARKS specimen➞lib_vector](MIMARKS_specimen_lib_vector.md)
         * [MISAG➞lib_vector](MISAG_lib_vector.md)
         * [MIUVIG➞lib_vector](MIUVIG_lib_vector.md)
     * [mag_cov_software](mag_cov_software.md) - Tool(s) used to determine the genome coverage if coverage is used as a binning parameter in the extraction of genomes from metagenomic datasets
         * [MIMAG➞mag_cov_software](MIMAG_mag_cov_software.md)
         * [MIUVIG➞mag_cov_software](MIUVIG_mag_cov_software.md)
     * [mid](mid.md) - Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to specifically tag unique samples in a sequencing run. Sequence should be reported in uppercase letters
         * [ME➞mid](ME_mid.md)
         * [MIMAG➞mid](MIMAG_mid.md)
         * [MIMARKS specimen➞mid](MIMARKS_specimen_mid.md)
         * [MISAG➞mid](MISAG_mid.md)
         * [MIUVIG➞mid](MIUVIG_mid.md)
     * [nucl_acid_amp](nucl_acid_amp.md) - A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids
         * [ME➞nucl_acid_amp](ME_nucl_acid_amp.md)
         * [MIGS bacteria➞nucl_acid_amp](MIGS_bacteria_nucl_acid_amp.md)
         * [MIGS eukaryote➞nucl_acid_amp](MIGS_eukaryote_nucl_acid_amp.md)
         * [MIGS org➞nucl_acid_amp](MIGS_org_nucl_acid_amp.md)
         * [MIGS plant➞nucl_acid_amp](MIGS_plant_nucl_acid_amp.md)
         * [MIGS virus➞nucl_acid_amp](MIGS_virus_nucl_acid_amp.md)
         * [MIMAG➞nucl_acid_amp](MIMAG_nucl_acid_amp.md)
         * [MIMARKS specimen➞nucl_acid_amp](MIMARKS_specimen_nucl_acid_amp.md)
         * [MIMARKS survey➞nucl_acid_amp](MIMARKS_survey_nucl_acid_amp.md)
         * [MISAG➞nucl_acid_amp](MISAG_nucl_acid_amp.md)
         * [MIUVIG➞nucl_acid_amp](MIUVIG_nucl_acid_amp.md)
     * [nucl_acid_ext](nucl_acid_ext.md) - A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample
         * [ME➞nucl_acid_ext](ME_nucl_acid_ext.md)
         * [MIGS bacteria➞nucl_acid_ext](MIGS_bacteria_nucl_acid_ext.md)
         * [MIGS eukaryote➞nucl_acid_ext](MIGS_eukaryote_nucl_acid_ext.md)
         * [MIGS org➞nucl_acid_ext](MIGS_org_nucl_acid_ext.md)
         * [MIGS plant➞nucl_acid_ext](MIGS_plant_nucl_acid_ext.md)
         * [MIGS virus➞nucl_acid_ext](MIGS_virus_nucl_acid_ext.md)
         * [MIMAG➞nucl_acid_ext](MIMAG_nucl_acid_ext.md)
         * [MIMARKS specimen➞nucl_acid_ext](MIMARKS_specimen_nucl_acid_ext.md)
         * [MIMARKS survey➞nucl_acid_ext](MIMARKS_survey_nucl_acid_ext.md)
         * [MISAG➞nucl_acid_ext](MISAG_nucl_acid_ext.md)
         * [MIUVIG➞nucl_acid_ext](MIUVIG_nucl_acid_ext.md)
     * [number_contig](number_contig.md) - Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG
         * [ME➞number_contig](ME_number_contig.md)
         * [MIGS bacteria➞number_contig](MIGS_bacteria_number_contig.md)
         * [MIGS eukaryote➞number_contig](MIGS_eukaryote_number_contig.md)
         * [MIGS org➞number_contig](MIGS_org_number_contig.md)
         * [MIGS plant➞number_contig](MIGS_plant_number_contig.md)
         * [MIGS virus➞number_contig](MIGS_virus_number_contig.md)
         * [MIMAG➞number_contig](MIMAG_number_contig.md)
         * [MISAG➞number_contig](MISAG_number_contig.md)
         * [MIUVIG➞number_contig](MIUVIG_number_contig.md)
     * [pcr_cond](pcr_cond.md) - Description of reaction conditions and components of PCR in the form of 'initial denaturation:94degC_1.5min; annealing=...'
         * [MIMARKS specimen➞pcr_cond](MIMARKS_specimen_pcr_cond.md)
         * [MIMARKS survey➞pcr_cond](MIMARKS_survey_pcr_cond.md)
     * [pcr_primers](pcr_primers.md) - PCR primers that were used to amplify the sequence of the targeted gene, locus or subfragment. This field should contain all the primers used for a single PCR reaction if multiple forward or reverse primers are present in a single PCR reaction. The primer sequence should be reported in uppercase letters
         * [MIMARKS specimen➞pcr_primers](MIMARKS_specimen_pcr_primers.md)
         * [MIMARKS survey➞pcr_primers](MIMARKS_survey_pcr_primers.md)
     * [pred_genome_struc](pred_genome_struc.md) - Expected structure of the viral genome
         * [MIUVIG➞pred_genome_struc](MIUVIG_pred_genome_struc.md)
     * [pred_genome_type](pred_genome_type.md) - Type of genome predicted for the UViG
         * [MIUVIG➞pred_genome_type](MIUVIG_pred_genome_type.md)
     * [reassembly_bin](reassembly_bin.md) - Has an assembly been performed on a genome bin extracted from a metagenomic assembly?
         * [MIMAG➞reassembly_bin](MIMAG_reassembly_bin.md)
         * [MIUVIG➞reassembly_bin](MIUVIG_reassembly_bin.md)
     * [ref_db](ref_db.md) - List of database(s) used for ORF annotation, along with version number and reference to website or publication
         * [ME➞ref_db](ME_ref_db.md)
         * [MIGS bacteria➞ref_db](MIGS_bacteria_ref_db.md)
         * [MIGS eukaryote➞ref_db](MIGS_eukaryote_ref_db.md)
         * [MIGS org➞ref_db](MIGS_org_ref_db.md)
         * [MIGS plant➞ref_db](MIGS_plant_ref_db.md)
         * [MIGS virus➞ref_db](MIGS_virus_ref_db.md)
         * [MIMAG➞ref_db](MIMAG_ref_db.md)
         * [MISAG➞ref_db](MISAG_ref_db.md)
         * [MIUVIG➞ref_db](MIUVIG_ref_db.md)
     * [seq_meth](seq_meth.md) - Sequencing method used; e.g. Sanger, ABI-solid
         * [ME➞seq_meth](ME_seq_meth.md)
         * [MIGS bacteria➞seq_meth](MIGS_bacteria_seq_meth.md)
         * [MIGS eukaryote➞seq_meth](MIGS_eukaryote_seq_meth.md)
         * [MIGS org➞seq_meth](MIGS_org_seq_meth.md)
         * [MIGS plant➞seq_meth](MIGS_plant_seq_meth.md)
         * [MIGS virus➞seq_meth](MIGS_virus_seq_meth.md)
         * [MIMAG➞seq_meth](MIMAG_seq_meth.md)
         * [MIMARKS specimen➞seq_meth](MIMARKS_specimen_seq_meth.md)
         * [MIMARKS survey➞seq_meth](MIMARKS_survey_seq_meth.md)
         * [MISAG➞seq_meth](MISAG_seq_meth.md)
         * [MIUVIG➞seq_meth](MIUVIG_seq_meth.md)
     * [seq_quality_check](seq_quality_check.md) - Indicate if the sequence has been called by automatic systems (none) or undergone a manual editing procedure (e.g. by inspecting the raw data or chromatograms). Applied only for sequences that are not submitted to SRA,ENA or DRA
         * [MIMARKS specimen➞seq_quality_check](MIMARKS_specimen_seq_quality_check.md)
         * [MIMARKS survey➞seq_quality_check](MIMARKS_survey_seq_quality_check.md)
     * [sim_search_meth](sim_search_meth.md) - Tool used to compare ORFs with database, along with version and cutoffs used
         * [ME➞sim_search_meth](ME_sim_search_meth.md)
         * [MIGS bacteria➞sim_search_meth](MIGS_bacteria_sim_search_meth.md)
         * [MIGS eukaryote➞sim_search_meth](MIGS_eukaryote_sim_search_meth.md)
         * [MIGS org➞sim_search_meth](MIGS_org_sim_search_meth.md)
         * [MIGS plant➞sim_search_meth](MIGS_plant_sim_search_meth.md)
         * [MIGS virus➞sim_search_meth](MIGS_virus_sim_search_meth.md)
         * [MIMAG➞sim_search_meth](MIMAG_sim_search_meth.md)
         * [MISAG➞sim_search_meth](MISAG_sim_search_meth.md)
         * [MIUVIG➞sim_search_meth](MIUVIG_sim_search_meth.md)
     * [single_cell_lysis_appr](single_cell_lysis_appr.md) - Method used to free DNA from interior of the cell(s) or particle(s)
         * [MISAG➞single_cell_lysis_appr](MISAG_single_cell_lysis_appr.md)
         * [MIUVIG➞single_cell_lysis_appr](MIUVIG_single_cell_lysis_appr.md)
     * [single_cell_lysis_prot](single_cell_lysis_prot.md) - Name of the kit or standard protocol used for cell(s) or particle(s) lysis
         * [MISAG➞single_cell_lysis_prot](MISAG_single_cell_lysis_prot.md)
         * [MIUVIG➞single_cell_lysis_prot](MIUVIG_single_cell_lysis_prot.md)
     * [sop](sop.md) - Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences
         * [ME➞sop](ME_sop.md)
         * [MIGS bacteria➞sop](MIGS_bacteria_sop.md)
         * [MIGS eukaryote➞sop](MIGS_eukaryote_sop.md)
         * [MIGS org➞sop](MIGS_org_sop.md)
         * [MIGS plant➞sop](MIGS_plant_sop.md)
         * [MIGS virus➞sop](MIGS_virus_sop.md)
         * [MIMAG➞sop](MIMAG_sop.md)
         * [MIMARKS specimen➞sop](MIMARKS_specimen_sop.md)
         * [MIMARKS survey➞sop](MIMARKS_survey_sop.md)
         * [MISAG➞sop](MISAG_sop.md)
         * [MIUVIG➞sop](MIUVIG_sop.md)
     * [sort_tech](sort_tech.md) - Method used to sort/isolate cells or particles of interest
         * [MISAG➞sort_tech](MISAG_sort_tech.md)
         * [MIUVIG➞sort_tech](MIUVIG_sort_tech.md)
     * [target_gene](target_gene.md) - Targeted gene or locus name for marker gene studies
         * [MIMARKS specimen➞target_gene](MIMARKS_specimen_target_gene.md)
         * [MIMARKS survey➞target_gene](MIMARKS_survey_target_gene.md)
     * [target_subfragment](target_subfragment.md) - Name of subfragment of a gene or locus. Important to e.g. identify special regions on marker genes like V6 on 16S rRNA
         * [MIMARKS specimen➞target_subfragment](MIMARKS_specimen_target_subfragment.md)
         * [MIMARKS survey➞target_subfragment](MIMARKS_survey_target_subfragment.md)
     * [tax_class](tax_class.md) - Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes
         * [ME➞tax_class](ME_tax_class.md)
         * [MIGS bacteria➞tax_class](MIGS_bacteria_tax_class.md)
         * [MIGS eukaryote➞tax_class](MIGS_eukaryote_tax_class.md)
         * [MIGS org➞tax_class](MIGS_org_tax_class.md)
         * [MIGS plant➞tax_class](MIGS_plant_tax_class.md)
         * [MIGS virus➞tax_class](MIGS_virus_tax_class.md)
         * [MIMAG➞tax_class](MIMAG_tax_class.md)
         * [MISAG➞tax_class](MISAG_tax_class.md)
         * [MIUVIG➞tax_class](MIUVIG_tax_class.md)
     * [tax_ident](tax_ident.md) - The phylogenetic marker(s) used to assign an organism name to the SAG or MAG
         * [MIGS bacteria➞tax_ident](MIGS_bacteria_tax_ident.md)
         * [MIGS eukaryote➞tax_ident](MIGS_eukaryote_tax_ident.md)
         * [MIGS org➞tax_ident](MIGS_org_tax_ident.md)
         * [MIGS plant➞tax_ident](MIGS_plant_tax_ident.md)
         * [MIGS virus➞tax_ident](MIGS_virus_tax_ident.md)
         * [MIMAG➞tax_ident](MIMAG_tax_ident.md)
         * [MISAG➞tax_ident](MISAG_tax_ident.md)
         * [MIUVIG➞tax_ident](MIUVIG_tax_ident.md)
     * [trna_ext_software](trna_ext_software.md) - Tools used for tRNA identification
         * [MIMAG➞trna_ext_software](MIMAG_trna_ext_software.md)
         * [MISAG➞trna_ext_software](MISAG_trna_ext_software.md)
         * [MIUVIG➞trna_ext_software](MIUVIG_trna_ext_software.md)
     * [trnas](trnas.md) - The total number of tRNAs identified from the SAG or MAG
         * [MIMAG➞trnas](MIMAG_trnas.md)
         * [MISAG➞trnas](MISAG_trnas.md)
         * [MIUVIG➞trnas](MIUVIG_trnas.md)
     * [url](url.md)
         * [ME➞url](ME_url.md)
         * [MIGS bacteria➞url](MIGS_bacteria_url.md)
         * [MIGS eukaryote➞url](MIGS_eukaryote_url.md)
         * [MIGS org➞url](MIGS_org_url.md)
         * [MIGS plant➞url](MIGS_plant_url.md)
         * [MIGS virus➞url](MIGS_virus_url.md)
         * [MIMAG➞url](MIMAG_url.md)
         * [MIMARKS specimen➞url](MIMARKS_specimen_url.md)
         * [MIMARKS survey➞url](MIMARKS_survey_url.md)
         * [MISAG➞url](MISAG_url.md)
         * [MIUVIG➞url](MIUVIG_url.md)
     * [vir_ident_software](vir_ident_software.md) - Tool(s) used for the identification of UViG as a viral genome, software or protocol name including version number, parameters, and cutoffs used
         * [MIUVIG➞vir_ident_software](MIUVIG_vir_ident_software.md)
     * [votu_class_appr](votu_class_appr.md) - Cutoffs and approach used when clustering new UViGs in “species-level” vOTUs. Note that results from standard 95% ANI / 85% AF clustering should be provided alongside vOTUS defined from another set of thresholds, even if the latter are the ones primarily used during the analysis
         * [MIUVIG➞votu_class_appr](MIUVIG_votu_class_appr.md)
     * [votu_db](votu_db.md) - Reference database (i.e. sequences not generated as part of the current study) used to cluster new genomes in "species-level" vOTUs, if any
         * [MIUVIG➞votu_db](MIUVIG_votu_db.md)
     * [votu_seq_comp_appr](votu_seq_comp_appr.md) - Tool and thresholds used to compare sequences when computing "species-level" vOTUs
         * [MIUVIG➞votu_seq_comp_appr](MIUVIG_votu_seq_comp_appr.md)
     * [wga_amp_appr](wga_amp_appr.md) - Method used to amplify genomic DNA in preparation for sequencing
         * [MISAG➞wga_amp_appr](MISAG_wga_amp_appr.md)
         * [MIUVIG➞wga_amp_appr](MIUVIG_wga_amp_appr.md)
     * [wga_amp_kit](wga_amp_kit.md) - Kit used to amplify genomic DNA in preparation for sequencing
         * [MISAG➞wga_amp_kit](MISAG_wga_amp_kit.md)
         * [MIUVIG➞wga_amp_kit](MIUVIG_wga_amp_kit.md)
     * [x_16s_recover](x_16s_recover.md) - Can a 16S gene be recovered from the submitted SAG or MAG?
         * [MIMAG➞x_16s_recover](MIMAG_x_16s_recover.md)
         * [MISAG➞x_16s_recover](MISAG_x_16s_recover.md)
     * [x_16s_recover_software](x_16s_recover_software.md) - Tools used for 16S rRNA gene extraction
         * [MIMAG➞x_16s_recover_software](MIMAG_x_16s_recover_software.md)
         * [MISAG➞x_16s_recover_software](MISAG_x_16s_recover_software.md)

### Types


#### Built in

 * **Bool**
 * **Decimal**
 * **ElementIdentifier**
 * **NCName**
 * **NodeIdentifier**
 * **URI**
 * **URIorCURIE**
 * **XSDDate**
 * **XSDDateTime**
 * **XSDTime**
 * **float**
 * **int**
 * **str**

#### Defined

 * [Boolean](types/Boolean.md)  (**Bool**)  - A binary (true or false) value
 * [Date](types/Date.md)  (**XSDDate**)  - a date (year, month and day) in an idealized calendar
 * [Datetime](types/Datetime.md)  (**XSDDateTime**)  - The combination of a date and time
 * [Decimal](types/Decimal.md)  (**Decimal**)  - A real number with arbitrary precision that conforms to the xsd:decimal specification
 * [Double](types/Double.md)  (**float**)  - A real number that conforms to the xsd:double specification
 * [Float](types/Float.md)  (**float**)  - A real number that conforms to the xsd:float specification
 * [Integer](types/Integer.md)  (**int**)  - An integer
 * [Ncname](types/Ncname.md)  (**NCName**)  - Prefix part of CURIE
 * [Nodeidentifier](types/Nodeidentifier.md)  (**NodeIdentifier**)  - A URI, CURIE or BNODE that represents a node in a model.
 * [Objectidentifier](types/Objectidentifier.md)  (**ElementIdentifier**)  - A URI or CURIE that represents an object in the model.
 * [String](types/String.md)  (**str**)  - A character string
 * [Time](types/Time.md)  (**XSDTime**)  - A time object represents a (local) time of day, independent of any particular day
 * [Uri](types/Uri.md)  (**URI**)  - a complete URI
 * [Uriorcurie](types/Uriorcurie.md)  (**URIorCURIE**)  - a URI or a CURIE
