---
layout: default
title: Natalia Maltsev talk on Grid-based Integrated Bioinformatics Systems for High Throughput
category: news
tag: [pyilmaz]
date: 2006-09-30
---
{% include header.html %}

Natalia Maltsev talk on Grid-based Integrated Bioinformatics Systems for High Throughput
=======================================================
Natalia Maltsev,  a member of the GSC working groups, gives a presentation at Google Tech on Grid-based Integrated Bioinformatics Systems for High Throughput:

<a href=""http://video.google.co.uk/videoplay?docid=86918197086997488"">http://video.google.co.uk/videoplay?docid=86918197086997488</a>

<a href=""http://www-unix.mcs.anl.gov/%7Emaltsev/"">Home page for Natalia Maltsev</a>
<p style=""text-align:center""><img src=""http://www-unix.mcs.anl.gov/~maltsev/mainpic1.jpg"" alt=""Natalia Maltsev"" height=""120"" width=""160"" /></p>
Further information on the Genomic Standards Consortium at <a href=""http://gensc.sf.net/"" title=""http://gensc.sf.net "">http://gensc.sf.net </a>"
---
layout: default
title: The first four GCat identifiers to appear in print
category: news
tag: [pyilmaz]
date: 2006-11-08
---
{% include header.html %}

The first four GCat identifiers to appear in print
=======================================================
The first four GCat identifiers to appear in print are now online in PloS Biology in a description of four viral metagenomes:

The Marine Viromes of Four Oceanic Regions
Florent E. Angly, Ben Felts, Mya Breitbart, Peter Salamon, Robert A. Edwards, Craig Carlson, Amy M. Chan, Matthew Haynes, Scott Kelley, Hong Liu, Joseph M. Mahaffy, Jennifer E. Mueller, Jim Nulton, Robert Olson, Rachel Parsons, Steve Rayhawk, Curtis A. Suttle, Forest Rohwer
<a href=""http://dx.doi.org/10.1371/journal.pbio.0040368"" class=""weblink"" target=""browserView"">http://dx.doi.org/10.1371/journal.pbio.0040368</a>

Synopsis: Metagenomics Offers a Big-Picture View of the Diversity and Distribution of Marine Viruses
<a href=""http://dx.doi.org/10.1371/journal.pbio.0040406"" class=""weblink"" target=""browserView"">http://dx.doi.org/10.1371/journal.pbio.0040406</a>

Here is the snippet from the paper:

The Genome Projects Database (<a href=""http://www.ncbi.nlm.nih.gov/Genomes"" class=""weblink"" target=""browserView"">http://www.ncbi.nlm.nih.gov/Genomes</a>) accession numbers for the sequences are 17765 (GOM), 17767 (BBC), 17769 (Arctic), and 17771 (SAR); the Genome Catalogue (<a href=""http://gensc.sf.net/"" class=""weblink"" target=""browserView"">http://gensc.sf.net</a>) accession numbers are 000002_GCAT (GOM), 000003_GCAT (BBC), 000004_GCAT (Arctic), and 000005_GCAT (SAR); and the GOLD database (<a href=""http://www.genomesonline.org/"" class=""weblink"" target=""browserView"">http://www.genomesonline.org</a>) GOLDstamps are GM00060 (GOM), GM00061 (BBC), GM00062 (Arctic), and GM00063 (SAR)."
---
layout: default
title: The Macaque Genome
category: news
tag: [pyilmaz]
date: 2007-04-14
---
{% include header.html %}

The Macaque Genome
=======================================================
The rhesus macaque monkey (<em>Macaca mulatta</em>) genome was described in the 13th April issue of <a href=""http://www.sciencemag.org/sciext/macaque/"">Science Magazine</a>.

<strong>Genome report for <em>Macaca mulatta</em>:</strong>

<a href=""http://darwin.nox.ac.uk/gsc/gcat/report/002485_GCAT/html"" title=""Macaque Genome report"">http://darwin.nox.ac.uk/gsc/ gcat/report/002485_GCAT/html</a>

<strong>Genome wiki page:</strong>

<a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php//gcat/report/002485_GCAT/html"" title=""GSC Wiki page for the Macaque genome"">http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php/ /gcat/report/002485_GCAT/html</a>

<strong>Genome mashup for macaque genome:</strong>

<a href=""http://darwin.nox.ac.uk/gsc/gcat/mashup?query=macaque%20genome"" title=""GSC mashup for Macaque genome"">http://darwin.nox.ac.uk/gsc/gcat /mashup?query=macaque%20genome</a>,  returned the following three videos from YouTube:

<a href=""http://www.youtube.com/?v=ZRGYX_YWnpA""><img src=""http://sjl-static3.sjl.youtube.com/vi/ZRGYX_YWnpA/2.jpg"" alt=""Macaque Genome video part I"" height=""97"" width=""130"" /></a>
<a href=""http://www.youtube.com/?v=6i0vhW10bXg""><img src=""http://img.youtube.com/vi/6i0vhW10bXg/2.jpg"" /></a>

<a href=""http://www.youtube.com/?v=gUMsuNvdXmI""><img src=""http://sjc-static8.sjc.youtube.com/vi/gUMsuNvdXmI/2.jpg"" /></a>"
---
layout: default
title: Query the Web - Genomic Standards Consortium mashup
category: news
tag: [pyilmaz]
date: 2007-04-26
---
{% include header.html %}

Query the Web - Genomic Standards Consortium mashup
=======================================================

<p align=""center""> <img src=""http://darwin.nox.ac.uk/gsc/gcat/xtr/images/gsc.png"" alt=""GSC logo"" height=""65"" width=""310"" /></p>
A new mashup function has been added to the Genome Catalogue that combines content from sites including NCBI Genome Project, PubMed, Flickr and even YouTube video!

You can either enter your own search term, or else link to the Query the Web page from one of the genome reports contained in the Genome Catalogue.

Here is an example query for Streptococcus:

<a href=""http://darwin.nox.ac.uk/gsc/gcat/mashup?query=Streptococcus"">http://darwin.nox.ac.uk/gsc/gcat/mashup?query=Streptococcus</a>"
---
layout: default
title: Genomic Standards Consortium and the EnvO (Environmental Ontology)
category: news
tag: [pyilmaz]
date: 2007-04-26
---
{% include header.html %}

Genomic Standards Consortium and the EnvO (Environmental Ontology)
=======================================================
EnvO (Environmental Ontology)  is a collaboration between many individuals and groups interested in how we describe the environment.

The EnvO project homepage is currently under construction. Contact Norman Morrison for further details: morrison at cs.man.ac.uk.

The Genomic Standards Consortium will be helping to sponsor the first <a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php/EnvO_Workshop"" title=""EnvO Workshop"">EnvO Workshop</a>.

EnvO mailing list: <a href=""https://lists.sourceforge.net/lists/listinfo/obo-envo"" class=""external"" rel=""nofollow"">https://lists.sourceforge.net/lists/listinfo/obo-envo</a>"
---
layout: default
title: Genomic Standards Consortium Publications
category: news
tag: [pyilmaz]
date: 2007-04-26
---
{% include header.html %}

Genomic Standards Consortium Publications
=======================================================

<h4>The current GSC publication list:</h4>
The MIGS manuscript:  The Nature Biotech community consultation concluded April 19th, 2007.  Decision by NBT pending.
<p style=""text-align:center""><img src=""http://darwin.nerc-oxford.ac.uk/gc_wiki/images/8/8e/Nature_biotech.gif"" alt=""Nature Biotech"" border=""1"" height=""273"" width=""436"" /></p>
<font color=""#cc3300"">GSC Meeting Reports</font>
<ul>
	<li> Dawn Field, George Garrity, Tanya Gray, Jeremy Selengut, Peter Sterk, Nick Thomson, Tatiana Tatusova, Guy Cochrane, Frank Oliver Glockner, Renzo Kottmann, Allyson L. Lister, Yoshio Tateno, and Robert Vaughan. (2007) Meeting Report: eGenomics: Cataloguing Our Complete Genome Collection III. Comparative and Functional Genomics: A journal of integrative biology (2006) June, Vol. 10, No. 2: 100 -104.</li>
</ul>
PDF - Uncorrected proof<span class=""urlexpansion""> </span>Journal PubMed
<ul>
	<li> Field D, Morrison N, Selengut J, Sterk P. Meeting Report: eGenomics: Cataloguing Our Complete Genome Collection II. OMICS: A journal of integrative biology (2006) June, Vol. 10, No. 2: 100 -104.</li>
</ul>
<a href=""http://envgen.nox.ac.uk/articles/gsc_report_2.pdf"" class=""external"" title=""http://envgen.nox.ac.uk/articles/gsc report 2.pdf"" rel=""nofollow"">PDF</a><span class=""urlexpansion""> </span> <a href=""http://www.liebertonline.com/doi/abs/10.1089/omi.2006.10.100"" class=""external"" title=""http://www.liebertonline.com/doi/abs/10.1089/omi.2006.10.100"" rel=""nofollow"">Journal</a><span class=""urlexpansion""> </span>  <a href=""http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list_uids=16901213&amp;itool=iconabstr&amp;query_hl=2&amp;itool=pubmed_docsum"" class=""external"" title=""http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list uids=16901213&amp;itool=iconabstr&amp;query hl=2&amp;itool=pubmed docsum"" rel=""nofollow"">PubMed</a>
<ul>
	<li> Field D, Garrity G, Morrison N, Selengut J, Sterk P, Thomson N, Tatusova T. Meeting Report: eGenomics: Cataloguing our Complete Genome Collection. Comparative and Functional Genomics (2006) 6: 363-368.</li>
</ul>
<a href=""http://envgen.nox.ac.uk/articles/cataloguing_genome_collection.pdf"" class=""external"" title=""http://envgen.nox.ac.uk/articles/cataloguing genome collection.pdf"" rel=""nofollow"">PDF</a><span class=""urlexpansion""> </span> <a href=""http://www.hindawi.com/GetArticle.aspx?doi=10.1002/cfg.494"" class=""external"" title=""http://www.hindawi.com/GetArticle.aspx?doi=10.1002/cfg.494"" rel=""nofollow"">Journal</a><span class=""urlexpansion""> </span>
<p class=""editsection"" style=""float:right;margin-left:5px"">&nbsp;</p>

<h4><font color=""#cc3300"">Special Issue of OMICS</font></h4>
Papers from the special issue of <a href=""http://www.liebertonline.com/toc/omi/10/2"" class=""external"" title=""http://www.liebertonline.com/toc/omi/10/2"" rel=""nofollow"">OMICS</a><span class=""urlexpansion""> (<em>http://www.liebertonline.com/toc/omi/10/2</em>)</span> on data standards that emerged from the 2nd and 1st GSC workshops:
<ul>
	<li>Field D, Sansone S-A. A special issue on Omic data standards. OMICS: A journal of integrative biology (2006) June, Vol. 10, No. 2: 84 -93</li>
</ul>
<a href=""http://nebc.nox.ac.uk/articles/omic_special_issue.pdf"" class=""external"" title=""http://nebc.nox.ac.uk/articles/omic special issue.pdf"" rel=""nofollow"">PDF</a><span class=""urlexpansion""> </span> <a href=""http://www.liebertonline.com/toc/omi/10/2"" class=""external"" title=""http://www.liebertonline.com/toc/omi/10/2"" rel=""nofollow"">Journal</a><span class=""urlexpansion""> </span> PubMed
<ul>
	<li>Morrison N, Cochrane G, Faruque N, Tatusova T, Tateno Y, Hancock D, and Field D. The concept of Sample in Omics Technology. OMICS: A journal of integrative biology (2006) June, Vol. 10, No. 2: 127 -137 .</li>
</ul>
<a href=""http://envgen.nox.ac.uk/articles/concept%20_sample.pdf"" class=""external"" title=""http://envgen.nox.ac.uk/articles/concept  sample.pdf"" rel=""nofollow"">PDF</a><span class=""urlexpansion""> </span><span class=""urlexpansion""></span> <a href=""http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list_uids=16901217&amp;itool=iconabstr&amp;query_hl=8&amp;itool=pubmed_docsum"" class=""external"" title=""http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list uids=16901217&amp;itool=iconabstr&amp;query hl=8&amp;itool=pubmed docsum"" rel=""nofollow"">PubMed</a><span class=""urlexpansion""> </span>
<p style=""text-align:center""><img src=""http://www.liebertpub.com/media/images/covers/1536-2310.jpg"" alt=""OMICS Journal"" border=""1"" height=""200"" width=""155"" /></p>
 <strong>Related papers: standardization of genomic databases</strong>
<ul>
	<li>Jeffrey L. Boore. Requirements and Standards for Organelle Genome Databases. OMICS: A Journal of Integrative Biology (2006) June, Vol. 10, No. 2 : 119 -126.</li>
</ul>
PDF<span class=""urlexpansion""> </span> <a href=""http://www.liebertonline.com/toc/omi/10/2"" class=""external"" title=""http://www.liebertonline.com/toc/omi/10/2"" rel=""nofollow"">Journal</a><span class=""urlexpansion""> </span> <a href=""http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list_uids=16901216&amp;query_hl=5&amp;itool=pubmed_docsum"" class=""external"" title=""http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list uids=16901216&amp;query hl=5&amp;itool=pubmed docsum"" rel=""nofollow"">PubMed</a>
<ul>
	<li>Guy Cochrane, Kirsty Bates, Rolf Apweiler, Yoshio Tateno, Jun Mashima, Takehide Kosuge, Ilene Karsch Mizrachi, Susan Schafer, Michael Fetchko. Evidence Standards in Experimental and Inferential INSDC Third Party Annotation Data. OMICS: A Journal of Integrative Biology (2006) June, 2006, Vol. 10, No. 2: 105-113.</li>
</ul>
PDF<span class=""urlexpansion""> </span> <a href=""http://www.liebertonline.com/toc/omi/10/2"" class=""external"" title=""http://www.liebertonline.com/toc/omi/10/2"" rel=""nofollow"">Journal</a><span class=""urlexpansion""> </span> <a href=""http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list_uids=16901214&amp;query_hl=7&amp;itool=pubmed_docsum"" class=""external"" title=""http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list uids=16901214&amp;query hl=7&amp;itool=pubmed docsum"" rel=""nofollow"">PubMed</a>
<ul>
	<li>Peter Sterk, Paul J. Kersey, Rolf Apweiler. Genome Reviews: Standardizing Content and Representation of Information about Complete Genomes OMICS: A Journal of Integrative Biology (2006) June, 2006, Vol. 10, No. 2: 114-118.</li>
</ul>
PDF<span class=""urlexpansion""> </span> <a href=""http://www.liebertonline.com/toc/omi/10/2"" class=""external"" title=""http://www.liebertonline.com/toc/omi/10/2"" rel=""nofollow"">Journal</a><span class=""urlexpansion""> </span><a href=""http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list_uids=16901215&amp;query_hl=9&amp;itool=pubmed_docsum"" class=""external"" title=""http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list uids=16901215&amp;query hl=9&amp;itool=pubmed docsum"" rel=""nofollow"">PubMed</a><span class=""urlexpansion""> </span>
<p class=""editsection"" style=""float:right;margin-left:5px"">&nbsp;</p>

<h4><font color=""#cc3300"">Development of Ontologies relevant to the description of genomes and metagenomes</font></h4>
<ul>
	<li> Whetzel P.L, Brinkman R.R, Causton H.C, Fan L, Field D, Fostel J, Fragaso G, Gray T, Heiskanen M, Hernandez-Boussard T, Morrison N, Parkinson H, Rocca-Serra P, Sansone S-A, Schober D, Smith B, Stevens R, Stoeckert Jnr C.J, Taylor C, White J, Wood A, and the FuGO Working Group. Development of FuGO: An Ontology for Functional Genomics Investigations. OMICS: A journal of integrative biology (2006) June, Vol. 10, No. 2: 199 -204.</li>
</ul>
<a href=""http://envgen.nox.ac.uk/articles/fugo_dev.pdf"" class=""external"" title=""http://envgen.nox.ac.uk/articles/fugo dev.pdf"" rel=""nofollow"">PDF</a><span class=""urlexpansion""> </span> <a href=""http://www.liebertonline.com/toc/omi/10/2"" class=""external"" title=""http://www.liebertonline.com/toc/omi/10/2"" rel=""nofollow"">Journal</a><span class=""urlexpansion""> </span> <a href=""http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list_uids=16901226&amp;itool=iconabstr&amp;query_hl=5&amp;itool=pubmed_docsum"" class=""external"" title=""http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list uids=16901226&amp;itool=iconabstr&amp;query hl=5&amp;itool=pubmed docsum"" rel=""nofollow"">PubMed</a><span class=""urlexpansion""> </span>
<h4></h4>
<font color=""#cc3300"">Initial Proposal for MIGS</font>
<ul>
	<li> Field D, &amp; Hughes J. Cataloguing our current genome collection. Microbiology (2005) 151, 1016-1019.</li>
</ul>
PDF<span class=""urlexpansion""> </span> <a href=""http://mic.sgmjournals.org/cgi/content/full/151/4/1016"" class=""external"" title=""http://mic.sgmjournals.org/cgi/content/full/151/4/1016"" rel=""nofollow"">Journal</a><span class=""urlexpansion""></span> <a href=""http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list_uids=15817771&amp;query_hl=3&amp;itool=pubmed_DocSum"" class=""external"" title=""http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&amp;cmd=Retrieve&amp;dopt=AbstractPlus&amp;list uids=15817771&amp;query hl=3&amp;itool=pubmed DocSum"" rel=""nofollow"">PubMed</a>
<ul>
	<li>The review paper that led to the above call for MIGS: Martiny, J.B.H. and D. Field. Ecological perspectives on the sequenced genome collection. (2005) Ecology Letters 8: 1334-1345.</li>
</ul>
PDF<span class=""urlexpansion""> </span> <a href=""http://www.ingentaconnect.com/content/bsc/ele/2005/00000008/00000012/art00011"" class=""external"" title=""http://www.ingentaconnect.com/content/bsc/ele/2005/00000008/00000012/art00011"" rel=""nofollow"">Journal</a>

See the GSC wiki for the latest publications list: <a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php/GSC_Publications"" title=""http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php/GSC_Publications"">http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php/GSC_Publications</a>"
---
layout: default
title: Genomic Standards Consortium's Genome Catalogue - latest release
category: news
tag: [pyilmaz]
date: 2007-04-27
---
{% include header.html %}

Genomic Standards Consortium's Genome Catalogue - latest release
=======================================================
The latest release of the Genomic Standards Consortium Genome Catalogue is now available:

<a href=""http://gensc.sf.net"" title=""GSC web site"">http://gensc.sf.net</a>

Features are described in the
<a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php/Using_the_Genome_Catalogue"" title=""GSC wiki"">GSC wiki</a>"
---
layout: default
title: First Genome Comparison of Plankton Species
category: news
tag: [pyilmaz]
date: 2007-05-01
---
{% include header.html %}

First Genome Comparison of Plankton Species
=======================================================
From a news release (April 30, 2007) describing research by an international team of scientists led by Scripps Institution of Oceanography at UC San Diego and the Department of Energy’s Joint Genome Institute:

The international team’s work, published in the online edition of Proceedings of the National Academy of Sciences, is the first comparison of the genetic makeup of two closely related eukaryotic phytoplankton and the mechanisms that make them biologically similar and distinct.

“Through our research we’ve been trying to understand Ostreococcus’ role in marine ecosystems,” said Palenik, who indicated Ostreococcus cells contain nearly five times the DNA of comparably sized organisms such as cyanobacteria. Genomics has taught us that you can learn much more when you can do a comparison. The first genome is exciting but the second genome is even more exciting because you can suddenly compare organisms and see what each is doing differently and what they are doing the same.”

The researchers comparison of Ostreococcus lucimarinus (recently sequenced by the Department Of Energy’s Joint Genome Institute) and Ostreococcus tauri yielded several surprising results, including the documentation of a “new” chromosome differing between the species. Another chromosome appeared somewhat different between the species and the researchers believe it may serve as a gene transfer “trash can” where foreign DNA is integrated. Yet another difference was the identification of a chromosome featuring the same—albeit rearranged—genes in the two species. The researchers hypothesize that this chromosome may be related to sexual functions because the rearrangements are enough to prevent sex between the species.

<strong>Genome report for <em>Ostreococcus tauri</em>:</strong>

<a href=""http://darwin.nox.ac.uk/gsc/gcat/report-file/000191_GCAT.xml/html"">http://darwin.nox.ac.uk/gsc/gcat/report-file/000191_GCAT.xml/html</a>

<strong> Genome report for <em>Ostreococcus lucimarinus</em>:</strong>

<a href=""http://darwin.nox.ac.uk/gsc/gcat/report-file/BA93C73D-0D9E-90E5-FC3C-27F61FF8D162.xml/html"">http://darwin.nox.ac.uk/gsc/gcat/report-file/BA93C73D-0D9E-90E5-FC3C-27F61FF8D162.xml/html</a>

<strong>Genome wiki page for </strong><strong><em>Ostreococcus tauri</em></strong><strong>:</strong>

<a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php//gcat/report-file/000191_GCAT.xml/html"">http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php//gcat/report-file/000191_GCAT.xml/html</a>

<strong>News release:</strong>

<a href=""http://ucsdnews.ucsd.edu/newsrel/science/04-07GenomeMA-.asp"" class=""external"" rel=""nofollow"">http://ucsdnews.ucsd.edu/newsrel/science/04-07GenomeMA-.asp</a>"
---
layout: default
title: MIGS/MIMS
category: news
tag: [pyilmaz]
date: 2007-05-11
---
{% include header.html %}

MIGS/MIMS
=======================================================
<p><strong>Minimum Information about a Genome Sequence/ Minimum Information about a Metagenomic Sequence/Sample</strong></p>
<p>Some pointers to resources to retrieve, visualize and explore the MIGS/MIMS specification:</p>
<p><strong>Introduction to MIGS/MIMS<br /></strong></p>
<p>The information in MIGS/MIMS is intended to be used in comparative genomic and metagenomic analysis, provide a better understanding of the source of each genome and enable us to situate genomes and metagenomes in their geospatial and temporal contexts (when relevant) through the specification of geographic location and sampling date.</p>
<p><a href=""https://pubmed.ncbi.nlm.nih.gov/18464787/"" target=""_blank"" rel=""noopener"">MIGS</a> Minimum Information about a Genome Sequence</p>
<p><a href=""https://pubmed.ncbi.nlm.nih.gov/18479204/"" target=""_blank"" rel=""noopener"">MIMS</a> Minimum Information about a Metagenome Sequence</p>
<p><a href=""https://www.ncbi.nlm.nih.gov/biosample/docs/packages/"" target=""_blank"" rel=""noopener""> MIGS/MIMS Packages at NCBI</a><img src=""http://darwin.nerc-oxford.ac.uk/gc_wiki/images/9/94/Schema-graph.jpg"" border=""1"" /></p>
<p>MIGS/MIMS specifications have been incorporated into the <a href=""https://gensc.org/mixs/"" target=""_blank"" rel=""noopener"">MIxS standard</a> (<a href=""https://github.com/GenomicsStandardsConsortium/mixs"" target=""_blank"" rel=""noopener"">MIxS GitHub</a>).</p>

<!-- wp:paragraph -->
<p></p>
<!-- /wp:paragraph -->"
---
layout: default
title: 4th GSC Workshop
category: news
tag: [pyilmaz]
date: 2007-05-30
---
{% include header.html %}

4th GSC Workshop
=======================================================
<strong><font color=""#cc3300"">The 4th Genomic Standards Consortium Workshop will be held next week, 6-8th June, 2007 at the National Institute for Environmental e-Science (NIEeS), Cambridge.</font></strong>

As at previous workshops there will be a mix of talks and discussion sessions, although this year the talks will be shorter and more focused on progress made towards GSC goals. There will be 6 sessions covering key topics of interest to the GSC.

There will be a special focus on progress to date, the development of MIMS, future implementation of MIGS/MIMS, CVs and ontologies for describing genomes and metagenomes, the need for a single, global list of genomes and metagenomes and the role of the GSC's proposed <a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php/Genomic_Rosetta_Stone"" title=""Genomic Rosetta Stone"">Genomic Rosetta Stone.</a>

The latest <a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/images/3/37/4_workshop_v11.doc"" title=""agenda"">agenda</a> is available on the GSC wiki. <a href=""http://www.niees.ac.uk/events/egenomics2007/event_egenomics2007.shtml"">Workshop homepage on NIEeSweb site
</a>

<strong>Date:</strong> 6-8 June, 2007

<strong>Venue:</strong> National Institute for Environmental e-Science, Cambridge, UK

<strong>Organizers:</strong> Dawn Field, Tatiana Tatusova, Tanya Gray, George Garrity, and Frank Oliver Glöckner"
---
layout: default
title: James Watson's genome in GenBank
category: news
tag: [pyilmaz]
date: 2007-06-01
---
{% include header.html %}

James Watson's genome in GenBank
=======================================================
James Watson, co-discoverer of the DNA double helix and developer of the Human Genome Project, was presented with the DNA sequence data from his personal genome at Baylor College of Medicine in Houston on May 31st 2007.

Photos of the event are on flickr: <a href=""http://www.flickr.com/photos/a99xel/sets/72157600292635997/"">photos</a>

From the <a href=""http://www.bcm.edu/news/packages/watson_genome.cfm"">press release</a>:

The $1 million, two-month project is a collaboration of 454 Life Sciences and the BCM Human Genome Sequencing Center, said Dr. Richard Gibbs, director of the HGSC and a scientific advisor to the Connecticut-based company. The announcement, aside from its meaning to Watson, is significant because it demonstrates that it will be possible in the future to sequence anyone's genome – a goal toward which many sequencing firms are working. The time and cost will decrease as the technology improves.

A report on the project and a commentary on its ethical implications are scheduled to appear in the near future. The raw sequencing data was released today to the publicly available resource called GenBank National Center for Biotechnology Information Trace Archive

<a href=""http://www.ncbi.nlm.nih.gov/Traces/trace.cgi"">http://www.ncbi.nlm.nih.gov/Traces/trace.cgi </a>"
---
layout: default
title: Presentation slides from the 4th GSC Workshop
category: news
tag: [pyilmaz]
date: 2007-06-21
---
{% include header.html %}

Presentation slides from the 4th GSC Workshop
=======================================================
Presentation slides from the 4th GSC Workshop held on 6-8th June 2007 at NIEeS in Cambridge, are now available from the <a href=""http://buffalo.niees.group.cam.ac.uk/archive2.php?event_details=egenomics2007"" title=""4th GSC Workshop slides"">NIEeS archive</a>
<p align=""center""> <img src=""http://darwin.nox.ac.uk/orbeon/gcat/xtr/images/niees.jpg"" alt=""NIEeS"" height=""47"" width=""136"" /></p>"
---
layout: default
title: eGenomics III - meeting report published
category: news
tag: [pyilmaz]
date: 2007-07-12
---
{% include header.html %}

eGenomics III - meeting report published
=======================================================
<span>A meeting report summarizing the proceedings of the “eGenomics:      Cataloguing our Complete Genome Collection III” workshop held September 11–13, 2006, at the National Institute for Environmental      <em>e</em>Science (<a href=""http://www.google.co.uk/url?sa=t&amp;ct=res&amp;cd=1&amp;url=http%3A%2F%2Fwww.niees.ac.uk%2F&amp;ei=IwCWRpHZNpGiwAGYzOj5Bg&amp;usg=AFQjCNFYy1tH8HbEVUIVjQQlBlyGu5j8zQ&amp;sig2=t_Ew8Sj_JGzEcM-qMcOh7A"">NIE<em>e</em>S</a>), Cambridge, United Kingdom, is now available:</span>

<a href=""http://www.hindawi.com/GetArticle.aspx?doi=10.1155/2007/47304"" title=""read meeting report"">download meeting report
</a>

<p class=""type""><strong><span>Comparative and Functional Genomics
Volume 2007 (2007), Article ID 47304, 7 pages
doi:10.1155/2007/47304</span></strong>
<p class=""type"">Meeting Report</p>
     <span></span>
<p class=""title""><strong>eGenomics: Cataloguing Our Complete Genome Collection III</strong></p>
<p class=""title""><em>Dawn Field, George Garrity, Tanya Gray, Jeremy Selengut, Peter Sterk, Nick Thomson, Tatiana Tatusova, Guy Cochrane, Frank Oliver Glöckner, Renzo Kottmann, Allyson L. Lister, Yoshio Tateno, and Robert Vaughan</em></p>
<p class=""title""><span>This 3rd workshop of the <a href=""http://gensc.sf.net"">Genomic Standards Consortium</a> was divided into two parts.      The first half of the three-day workshop was dedicated to reviewing the genomic      diversity of our current and future genome and metagenome collection, and      exploring linkages to a series of existing projects through formal presentations.      The second half was dedicated to strategic discussions. Outcomes of the workshop      include a revised “<a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php/MIGS"">Minimum Information about a Genome Sequence”</a>       (<a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php/MIGS"">MIGS</a>) specification (v1.1), consensus on a variety of features to be added to the       <a href=""http://darwin.nox.ac.uk/gsc/gcat/xtr/genome-catalogue"">Genome Catalogue</a> (<a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php/GCat:_The_Genome_Catalogue_for_MIGS"">GCat</a>), agreement by several researchers to adopt MIGS for       imminent genome publications, and an agreement by the <a href=""http://www.ebi.ac.uk"">EBI</a> and <a href=""http://www.ncbi.nlm.nih.gov/"">NCBI</a> to input their        genome collections into <a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php/GCat:_The_Genome_Catalogue_for_MIGS"">GCat</a> for the purpose of quantifying the amount of optional        data already available (e.g., for geographic location coordinates) and working towards        a single, global list of all public genomes and metagenomes.</span></p>
     <span></span>

<p class=""authors""><strong><a href=""http://www.hindawi.com/GetArticle.aspx?doi=10.1155/2007/47304"">http://www.hindawi.com/GetArticle.aspx?doi=10.1155/2007/47304
</a><a href=""http://www.hindawi.com/384747.html""></a></strong>"
---
layout: default
title: GSC Birds of a Feather meeting at ISMB/ECCB 2007
category: news
tag: [pyilmaz]
date: 2007-07-19
---
{% include header.html %}

GSC Birds of a Feather meeting at ISMB/ECCB 2007
=======================================================
The Genomic Standards Consortium will be hosting a Birds of a Feather meeting at ISMB/ECCB 2007.

Date: Monday 23rd July 2007
Time: 12.15pm – 1.30pm
Room: see noticeboards

An open invitation :-
Meet members of the Genomic Standards Consortium
Learn about the activities of the Genomic Standards Consortium
Find out how to participate in the work of the Genomic Standards Consortium
Join in an open discussion

Meeting agenda:

* 12.30 - 12.40 : Introduction by Tatiana Tatusov
* 12.40 - 12.50 : Dawn Field - Overview of Genomic Standards Consortium (GSC) Activities
* 12.50 - 13.00 : Robert Stevens - Harnessing Ontologies to Describe Genomes
* 13.00 - 13.10 : Lynette Hirschman - Mining Metadata for Metagenomics
* 13.10 - 13.30  : Open Discussion

<a href=""http://gensc.sf.net/"" class=""weblink"" target=""browserView"">http://gensc.sf.net</a>"
---
layout: default
title: GSC BoF at ISMB/ECCB 2007 & the Environmental Ontology workshop
category: news
tag: [pyilmaz]
date: 2007-08-02
---
{% include header.html %}

GSC BoF at ISMB/ECCB 2007 & the Environmental Ontology workshop
=======================================================
The Genomic Standards Consortium held a Birds of a Feather meeting at ISMB/ECCB 2007.

There were three presentations, and a discussion led by Robert Stevens. Presentation slides can be downloaded from the GSC wiki:

<strong>Presentation Slides </strong>

* Tatiana Tatusov <a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/images/9/93/Tatusova.ppt"" title=""slides"">slides</a>

* Dawn Field <a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/images/c/c9/FieldISMB.pdf"" title=""slides"">slides</a>

* Lynette Hirschman <a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/images/4/47/Hirschman.pdf"" title=""slides"">slides</a>

We mostly discussed how to achieve wider community involvement. In particular we discussed the fact that a group of us worked quite extensively at ISMB on the development of an ontology for the georeferencing of biological samples and records.

<strong>Environmental Ontology workshop </strong>

Related to this, the GSC is co-sponsor of an Environmental Ontology (EnvO) workshop to be held on August 29-31 2007 where the ontology obo file (called geo.obo) will be further reviewed. The agenda for the EnvO meeting is available on the GSC wiki and further details will be publishing shortly.

<a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php/EnvO_Workshop"">http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php/EnvO_Workshop </a>

<strong>Geo.obo file from Michael Ashburner </strong>

Michael Ashburner sent out an email on June 29th 2007 regarding the geo.obo file. A copy of the email and file attachments have been uploaded to the GSC wiki.

<a href=""http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php/Michael_ashburner_email"" title=""Michael Ashburner email"">http://darwin.nerc-oxford.ac.uk/gc_wiki/index.php/Michael_ashburner_email </a>"
---
layout: default
title: Genomics and Society meeting, London, October 25/26th 2007
category: news
tag: [pyilmaz]
date: 2007-10-03
---
{% include header.html %}

Genomics and Society meeting, London, October 25/26th 2007
=======================================================
The GSC will present a poster at the Genomics and Society meeting, to be held in London on 25-26th October 2007.

The event will feature lectures, debates, workshops and exhibitions, on the past, present and future roles of genomics in society. More information on the meeting is available on the dedicated web site: <a href=""http://www.genomicsandsociety.org/"" title=""http://www.genomicsandsociety.org/"">http://www.genomicsandsociety.org/</a>

If you plan to attend the meeting, please send a note to <a href=""mailto:curator@ceh.ac.uk"" title=""curator@ceh.ac.uk"">curator@ceh.ac.uk</a>.
<p align=""right"">&nbsp;</p>
<p style=""text-align:center""><a href=""http://www.genomicsandsociety.org/"" title=""http://www.genomicsandsociety.org/""><img src=""http://www.genomicsandsociety.org/images/networklogo.jpg"" alt=""genomics and society network logo"" height=""143"" width=""374"" /></a></p>"
---
layout: default
title: 5th GSC Workshop
category: news
tag: [pyilmaz]
date: 2007-12-12
---
{% include header.html %}

5th GSC Workshop
=======================================================
The 5th GSC Workshop is underway. The workshop will run from 12-14th December, and is hosted at the EBI in Hinxton.  The GSC wiki will be updated during the workshop. For further information visit

<a href=""http://gensc.org/gc_wiki/index.php/5th_GSC_Workshop"">http://gensc.org/gc_wiki/index.php/5th_GSC_Workshop</a>"
---
layout: default
title: Standardising the Description of 16S Sequences with MIGS/MIMS
category: news
tag: [pyilmaz]
date: 2007-12-19
---
{% include header.html %}

Standardising the Description of 16S Sequences with MIGS/MIMS
=======================================================
At the 5th Genomic Standards Consortium (GSC) workshop held at the EBI on 12-14th December 2007, discussions led to a proposal to standardise the description of 16S sequences using the GSC community-led standard called Minimum Information about a Genome Sequence/ Metagenome Sample (MIGS/MIMS).

MIGS/MIMS defines descriptors that can assist in the capture of a rich set of information about 16S sequences using a community-led standard.

Further information is available on the GSC wiki:

<a href=""http://gensc.org/gc_wiki/index.php/MIGS/MIMS_for_16S"">http://gensc.org/gc_wiki/index.php/MIGS/MIMS_for_16S</a>"
---
layout: default
title: Genomic Standards Consortium (GSC) 5th Workshop - slides
category: news
tag: [pyilmaz]
date: 2007-12-19
---
{% include header.html %}

Genomic Standards Consortium (GSC) 5th Workshop - slides
=======================================================
Further to the 5th Genomic Standards Consortium (GSC) workshop, slides are available to download from the GSC wiki:

<a href=""http://gensc.org/gc_wiki/index.php/5th_GSC_Workshop#Agenda_and_Presentations"" title=""presentations"">agenda and presentations</a>"
---
layout: default
title: GenCat - Generic XML data catalogue - latest release
category: news
tag: [pyilmaz]
date: 2007-12-19
---
{% include header.html %}

GenCat - Generic XML data catalogue - latest release
=======================================================
The latest release of GenCat is available to download from the project SourceForge SVN repository:

<a href=""http://gencat.svn.sourceforge.net/viewvc/gencat/gencat/tags/release-1.1/"" title=""gencat"">gencat release-1.1</a>

GenCat uses the latest technology including Orbeon Forms, eXist, AJAX, XForms and XML Pipeline Language (XPL/XPROC), and provides a generic XML data catalogue with input forms generated on-the-fly from XML schema files, to capture schema-compliant XML instances.

GenCat has been implemented as the Genome Catalogue, an online repository from the Genomic Standards Consortium (GSC) containing MIGS/MIMS-compliant reports.

For further information:

GSC web site: <a href=""http://gensc.org"" title=""http://gensc.org "">http://gensc.org</a>"
---
layout: default
title: A 2nd special issue of OMICS to be developed by the GSC based on the 5th GSC Workshop
category: news
tag: [pyilmaz]
date: 2007-12-20
---
{% include header.html %}

A 2nd special issue of OMICS to be developed by the GSC based on the 5th GSC Workshop
=======================================================
Dawn Field and Susanna-Assunta Sansone guest edited a special issue of the journal <a href=""http://www.liebertpub.com/publication.aspx?pub_id=43"" class=""external text"" title=""http://www.liebertpub.com/publication.aspx?pub_id=43"" rel=""nofollow"">OMICS</a> on data standards as an output of the <b>2nd GSC workshop</b>.  There were 22 invited papers including 5 in the area of standardization of genomic data (See: <a href=""http://gensc.org/gc_wiki/index.php/GSC_Publications#Special_Issue_of_OMICS"" class=""external free"" title=""http://gensc.org/gc_wiki/index.php/GSC_Publications#Special_Issue_of_OMICS"" rel=""nofollow"">Special_Issue_of_OMICS</a>).
An overview of the issue and its goals is captured in the foreword:  <a href=""http://www.liebertonline.com/doi/pdfplus/10.1089/omi.2006.10.84"" class=""external free"" title=""http://www.liebertonline.com/doi/pdfplus/10.1089/omi.2006.10.84"" rel=""nofollow"">http://www.liebertonline.com/doi/pdfplus/10.1089/omi.2006.10.84</a>
The entire issue was open source and many of the papers in the issue continue to be at the top of the <b>most downloaded</b> list:  <a href=""http://www.liebertonline.com/action/showMostReadArticles?journalCode=omi"" class=""external free"" title=""http://www.liebertonline.com/action/showMostReadArticles?journalCode=omi"" rel=""nofollow"">http://www.liebertonline.com/action/showMostReadArticles?journalCode=omi</a>

<a title=""Special_issue_of_OMICS_from_the_5th_GSC_Workshop"" name=""Special_issue_of_OMICS_from_the_5th_GSC_Workshop""></a>
<h3><span class=""editsection""></span><span class=""mw-headline""> Special issue of OMICS from the 5th GSC Workshop</span></h3>
Dawn Field and George Garrity have been asked by the OMICS Editor and Chief Eugene Kolker to produce a special issue of OMICS based on the <b>5th GSC Workshop</b>. After guaging interest in this prior to the workshop, and in response to developments at the workshop, we are going to accept this invitation.We are now considering proposals from the participants of the 5th GSC workshop (and their colleagues) for contributions on several key topics of special interest to the GSC.

Further information:

GSC web site:<a href=""http://gensc.org"" title=""http://gensc.org""> http://gensc.org</a>

Dawn Field :  <a href=""http://gensc.org/gc_wiki/index.php/Dawn_Field"" title=""Dawn Field"">contact info</a>

George Garrity : <a href=""http://gensc.org/gc_wiki/index.php/George_Garrity"">contact info</a>

Susanna-Assunta Sansone : <a href=""http://gensc.org/gc_wiki/index.php/Susanna-Assunta_Sansone"">contact info</a>

Eugene Kolker : <a href=""http://gensc.org/gc_wiki/index.php/Eugene_Kolker"">contact info</a>"
---
layout: default
title: Nature Biotech paper - revised MIGS/MIMS checklist now available
category: news
tag: [pyilmaz]
date: 2008-01-02
---
{% include header.html %}

Nature Biotech paper - revised MIGS/MIMS checklist now available
=======================================================
At the recent Genomic Standards Consortium (GSC) workshop, George Garrity led discussions to review and produce a final version of the MIGS/MIMS checklist that will be included in the MIGS paper to be published in Nature Biotechnology in February 2008.

The revised version of the MIGS/MIMS checklist, together with comments recorded during the GSC workshop, are available via the GSC wiki at:

<a href=""http://gensc.org/gc_wiki/index.php/MIGS_Checklist_Proofs"">http://gensc.org/gc_wiki/index.php/MIGS_Checklist_Proofs</a>

The MIGS/MIMS checklist document is available directly at:

<a href=""http://gensc.sourceforge.net/docs/migsmims/"">http://gensc.sourceforge.net/docs/migsmims/</a>

Nature Biotechnology Journal home page:

<a href=""http://www.nature.com/nbt/index.html"">http://www.nature.com/nbt/index.html</a>

Nature Biotechnology invited the scientific community to comment on the MIGS paper prior to publication. The community consultation page <a href=""http://www.nature.com/nbt/consult/index.html"">http://www.nature.com/nbt/consult/index.ht</a> includes a link to the version of the MIGS paper made available for the community consultation:

<a href=""http://www.nature.com/nbt/consult/pdf/Field_et_al.pdf"" class=""articletitle"">Towards richer descriptions of our collection of genomes and metagenomes: the “Minimum Information about a Genome Sequence” (MIGS) specification (PDF 45K)</a>"
---
layout: default
title: GSC Poster at Asia Pacific Bioinformatics Conference (APBC) 2008 : 14-17 January, Kyoto, Japan
category: news
tag: [pyilmaz]
date: 2008-01-07
---
{% include header.html %}

GSC Poster at Asia Pacific Bioinformatics Conference (APBC) 2008 : 14-17 January, Kyoto, Japan
=======================================================
A poster will be presented at the Asia Pacific Bioinformatics Conference 2008 describing GenCat and the Genome Catalogue, two products of the Genomic Standards Consortium (GSC). If you are attending  APBC 2008 and are interested in the work of the GSC, please contact Tanya Gray at tgra at ceh.ac.uk.

<b>About the GSC:</b>

The Genomic Standards Consortium (GSC) formed in 2005 with the aim to promote methods to standardize the description of genomes and the exchange and integration of genomics data.

The GSC is an open-membership international working body. Participants in the GSC include biologists, computer scientists, those building genomic databases and conducting large-scale comparative genomic analyses, and those with experience of building community-based standards.

For more information, please visit <a href=""http://gensc.org"">http://gensc.org </a>

<b>Products</b>

The GSC has released a number of products:

MIGS/MIMS
Minimum Information about a Genome Sequence/Metagenome Sequence specification. Provides an extensions to the minimum information already captured by primary nucleotide databases (DDBJ/EMBL/Genbank) (Field et al, 2007).

GCDML
Genomic Contextual Data Markup Language that incorporates the MIGS/MIMS specification and provides an extended data capture and exchange mechanism for integrating a wide range of information relevant to the in depth description of genomes and metagenomes.

Genome Catalogue
A repository of genome reports that are compliant with the MIGS/MIMS specification. The Genome Catalogue is based on the GenCat software.

GenCat
A generic XML data catalogue tool that supports the development of data standards by providing a data repository and input forms auto-generated from successive XML schema files (used to define a data standard). (http://gencat.sf.net)

<b>APBC 2008</b>

<a href=""http://sunflower.kuicr.kyoto-u.ac.jp/apbc2008/"">http://sunflower.kuicr.kyoto-u.ac.jp/apbc2008/</a>"
---
layout: default
title: Vineet K. Sharma joins the Genomic Standards Consortium (GSC)
category: news
tag: [pyilmaz]
date: 2008-01-15
---
{% include header.html %}

Vineet K. Sharma joins the Genomic Standards Consortium (GSC)
=======================================================
Further to discussions at the Asia Pacific Bioinformatics Conference 2008 (APBC2008), Vineet K. Sharma is the newest member of the Genomic Standards Consortium (GSC).

Vineet's work concerns computational analysis of metagenomic data and novel genome analysis. Vineet's group sequences   metagenomic samples from human and other organism's metabiomes. Vineet will be involved in the further development of the GSC data standard, GCDML,  that includes descriptors for metagenomic samples.

<i><b>Further Information</b></i>

<b>GCDML</b>

Genomic Contextual Data Markup Language incorporates the <a href=""http://gensc.org/gc_wiki/index.php/MIGS/MIMS"" title=""MIGS/MIMS"">MIGS/MIMS</a> specification and provides an extended data capture and exchange mechanism for integrating a wide range of information relevant to the in depth description of genomes and metagenomes.

<b> Vineet K. Sharma</b>

<a href=""http://hgp.gsc.riken.go.jp/index.php/Vineet_K._Sharma"">home page</a>"
---
layout: default
title: GSC represented at MGED workshop on standardizing UHTS data, 17-18 March 2008
category: news
tag: [pyilmaz]
date: 2008-01-28
---
{% include header.html %}

GSC represented at MGED workshop on standardizing UHTS data, 17-18 March 2008
=======================================================
The Microarray and Gene Expression (<a href=""http://www.mged.org/"">MGED</a>) Society, is calling for an effort to develop standards for exchanging ultra-high throughput sequencing (UHTS) data.

Dawn Field will be attending the workshop to represent the GSC.

Guy Cochrane will be there to represent the EBI Short Read Trace Archive.

Susanna-Assunta Sansone, a member of the MGED Board, will also be there.


UPDATED: Oct 8:  the resulting proposal is now found at the <a href=""http://www.mged.org/minseqe/"">MGED MINSEQE working group </a>homepage
"
---
layout: default
title: GSC helping to plan the first MIBBI workshop in April 2008 with funding from the UK's BBSRC
category: news
tag: [pyilmaz]
date: 2008-01-29
---
{% include header.html %}

GSC helping to plan the first MIBBI workshop in April 2008 with funding from the UK's BBSRC
=======================================================
The GSC is helping to plan the first workshop for the Minimum Information for Biological and Biomedical Investigations (MIBBI) project, to be held on 2-3 April 2008, with funding from the UK research council, BBSRC.

The organizers are Chris Taylor (<a href=""http://www.ebi.ac.uk/net-project"" class=""external text"" title=""http://www.ebi.ac.uk/net-project"" rel=""nofollow"">EBI</a>), Susanna-Assunta Sansone (<a href=""http://www.ebi.ac.uk/net-project"" class=""external text"" title=""http://www.ebi.ac.uk/net-project"" rel=""nofollow"">EBI</a>) and Dawn Field (<a href=""http://gensc.org/"" class=""external text"" title=""http://gensc.org"" rel=""nofollow"">GSC</a>, <a href=""http://www.ceh.ac.uk/"" class=""external text"" title=""http://www.ceh.ac.uk"" rel=""nofollow"">NERC Centre for Ecology and Hydrology</a>).

<b> About the MIBBI workshop</b>

<a href=""http://gensc.org/gc_wiki/index.php/MIBBI_workshop"">MIBBI workshop</a>

<a href=""http://gensc.org/gc_wiki/index.php/MIBBI_workshop""></a>
<b>About MIBBI</b>

From <a href=""http://mibbi.sourceforge.net/"">http://mibbi.sourceforge.net/:</a> <font face=""Arial, Helvetica, sans-serif""></font>

<font face=""Arial, Helvetica, sans-serif"">Throughout                    the biological and biomedical sciences prescriptive checklists                    specifying the key information to include when reporting experimental                    results are beginning to find favour with experimentalists,                    analysts, publishers and funders alike. However, such ‘minimum                    information’ (MI) checklists are usually developed independently,                    from within particular biologically- or technologically-delineated                    domains. </font>

<font face=""Arial, Helvetica, sans-serif"">Consequently, the full range of checklists can be difficult                    to establish without intensive searching, and tracking their                    evolution is non-trivial; they are also inevitably partially-redundant                    one against another, and where they overlap arbitrary decisions                    on wording and substructuring make integration difficult. </font>

<font face=""Arial, Helvetica, sans-serif"">This                    presents significant difficulties for the users of checklists;                    for example, in the area of systems biology, where data from                    multiple biological domains and technology platforms are routinely                    combined. </font>

<font face=""Arial, Helvetica, sans-serif"">We offer a common portal to such MI checklists; to                    act as a ‘one-stop shop’ for those exploring the                    range of extant projects, foster collaborative development and                    ultimately promote gradual integration.</font>

<b>
</b>"
---
layout: default
title: GSC Special Issue of OMICS - Community Consultation
category: news
tag: [pyilmaz]
date: 2008-02-18
---
{% include header.html %}

GSC Special Issue of OMICS - Community Consultation
=======================================================
Dawn Field, Susanna-Assunta Sansone and George Garrity have been asked by the OMICS Editor and Chief Eugene Kolker to produce a special issue of OMICS based on the <b><a href=""http://gensc.org/gc_wiki/index.php/5th_GSC_Workshop"">5th GSC Workshop</a>.</b>

A community consultation has now started with the posting of the first paper on the GSC wiki. <a href=""http://gensc.org/gc_wiki/index.php/GSC_Special_issues_of_OMICS"">special issue of OMICS Wiki page</a> Authors are strongly encouraged to circulate their papers widely, including to members of the GSC.

Please provide feedback directly to the authors or Editors (<a href=""http://gensc.org/gc_wiki/index.php/Dawn_Field"">Dawn Field</a>, <a href=""http://gensc.org/gc_wiki/index.php/George_Garrity"">George Garrity</a>, <a href=""http://gensc.org/gc_wiki/index.php/Susanna-Assunta_Sansone"">Susanna-Assunta</a> Sansone).

<b>Further information:</b>

Genomic Standards Consortium: <a href=""http://gensc.org"">http://gensc.org</a>

Earlier post: <a href=""http://gensc.wordpress.com/2007/12/20/special-issue-of-omics-journal/"">special issue of OMICS</a>

<a href=""http://gensc.org/gc_wiki/index.php/GSC_Special_issues_of_OMICS"">special issue of OMICS Wiki page</a>"
---
layout: default
title: Open call for development of a short list of habitat terms: Habitat-Lite
category: news
tag: [pyilmaz]
date: 2008-03-11
---
{% include header.html %}

Open call for development of a short list of habitat terms: Habitat-Lite
=======================================================
 Further information:
<a href=""http://gensc.org/gc_wiki/index.php/Habitat-Lite"">Habitat-Lite</a>

The GSC is interested in the description of 'sample' including the habitat. We are therefore exploring the creation of a limited list of terms for describing habitat.

Increasingly, short lists of habitat terms are being used to annotate databases and undertake a variety of analyses. These lists are continuously being developed a new because there is not yet a central place to put and compare lists.

We are:

1. collecting habitat lists in use within the GSC community
2. determining the overlaps in these lists
3. assessing whether it it feasible to 'merge' (unify) these lists into a single, short list, that might be widely adopted for its broad scope and suitable coverage of a wide range of samples (genomes, metagenomes, 16S, etc)."
---
layout: default
title: Members of GSC working to complete example MIGS/MIMS report in ISA-Tab format
category: news
tag: [pyilmaz]
date: 2008-03-11
---
{% include header.html %}

Members of GSC working to complete example MIGS/MIMS report in ISA-Tab format
=======================================================
The <a href=""http://www.mged.org/Workgroups/rsbi/index.html"" class=""external text"" title=""http://www.mged.org/Workgroups/rsbi/index.html"" rel=""nofollow"">RSBI</a> working group has proposed an ""Investigation Study Assay"" paradigm for the modularization of minimum information checklists to accomodate reporting of multi-omic studies. This ""ISA"" construct is now also recognized by <a href=""http://mibbi.sf.net/"" class=""external text"" title=""http://mibbi.sf.net"" rel=""nofollow"">MIBBI</a>.

The GSC's MIGS checklist was specifically revised to also reflect the ISA structure to help in its modularization - some fields describe the whole Investigation, some the Study and some the Assay (in this case sequencing by any method including ultra high-throughput methods).

<b>Further information:</b>

<a href=""http://gensc.org/gc_wiki/index.php/ISA-Tab"">http://gensc.org/gc_wiki/index.php/ISA-Tab</a>"
---
layout: default
title: GSC FAQ available to give a quick overview of all GSC activities
category: news
tag: [pyilmaz]
date: 2008-03-11
---
{% include header.html %}

GSC FAQ available to give a quick overview of all GSC activities
=======================================================
The <a href=""http://gensc.org/gc_wiki/index.php/GSC_FAQ"" title=""GSC FAQ"">GSC FAQ</a> is now available, and provides a quick overview of all GSC activities."
---
layout: default
title: GSC vote: Genomic Rosetta Stone and LinkOut implementation
category: news
tag: [pyilmaz]
date: 2008-04-01
---
{% include header.html %}

GSC vote: Genomic Rosetta Stone and LinkOut implementation
=======================================================
There is currently a vote open to GSC members, with a closing date of 2nd April 2008 23.00GMT.

The vote concerns the implementation of the Genomic Rosetta Stone utilising NCBI LinkOut. To state your preferred option please visit:

<a href=""http://gensc.org/gc_wiki/index.php/Vote.001:_GRS_LinkOut_Implementation"">http://gensc.org/gc_wiki/index.php/Vote.001:_GRS_LinkOut_Implementation</a>"
---
layout: default
title: Feedback requested from GSC on MINSEQE
category: news
tag: [pyilmaz]
date: 2008-04-03
---
{% include header.html %}

Feedback requested from GSC on MINSEQE
=======================================================
As an output of discussions at the recent MGED-organized UHTS standardization workshop, a new MI checklist is being developed. Participants at the workshop included representatives from the GSC, SRF, the short read trace archives, major sequencing centres, large projects like ENCODE, vendors, funders and researchers generating data.

The checklist is called the ""Minimum Information about a high throughput Nucleotide SeQuencing Experiment"" (MINSEQE) and is available for feedback here:  <a href=""http://uhts.lbl.gov/"">http://uhts.lbl.gov/</a>"
---
layout: default
title: GSC at Microbial Commons International Workshop
category: news
tag: [pyilmaz]
date: 2008-04-24
---
{% include header.html %}

GSC at Microbial Commons International Workshop
=======================================================
CONFERENCE ANNOUNCEMENT: MICROBIAL COMMONS

Designing the Microbial Research Commons:
learning from institutional policies
and collaborative network technologies

12-13 June 2008

Het Pand, Ghent, Belgium

<a href=""http://www.microbialcommons.org"">http://www.microbialcommons.org </a>

Dear colleague,

As part of an international undertaking and collaboration between Science
Commons, the StrainInfo.net bioportal, the Genomic Standards Consortium,
Bioversity International and the US national committees of CODATA, IUMS
and IUBS, an international conference on the building of an integrated
infrastructure in microbial research dealing with issues such as
bioinformatics, intellectual property rights, material transfer agreements,
text mining and integration with genomics databases will be organised on
June 12-13 2008 at ""Het Pand"" Conference Centre in Ghent, Belgium. This
conference is open to all involved in microbial research, microbial
resources, industrial microbiology, publishing and policy making.

Conference website: <a href=""http://www.microbialcommons.org"">http://www.microbialcommons.org</a>

Contact us: <a href=""mailto:MicrobialCommons@lists.UGent.be"">MicrobialCommons@lists.UGent.be</a>

The microbial commons conference is organized back to back with the
annual meeting of the European Culture Collections' Organisation (ECCO),
June 10-11 2008.

ECCO website: <a href=""http://www.eccosite.org/"">http://www.eccosite.org/</a>"
---
layout: default
title: MIGS paper published in Nature Biotechnology
category: news
tag: [pyilmaz]
date: 2008-05-12
---
{% include header.html %}

MIGS paper published in Nature Biotechnology
=======================================================
Thanks to the work of so many, the ""Minimum Information about a Genome Sequence"" (MIGS) perspective has finally appear in print in Nature Biotechnology.  It contains the definitive version of the MIGS checklist as it applies to genome and metagenomes and provides the background for the development of the checklists and plans for its future implementation.

</br>View it online for free here: <a href=""http://www.nature.com/nbt/journal/v26/n5/full/nbt1360.html"">http://www.nature.com/nbt/journal/v26/n5/full/nbt1360.html</a>"
---
layout: default
title: RDP runs Habitat survey
category: news
tag: [pyilmaz]
date: 2008-05-14
---
{% include header.html %}

RDP runs Habitat survey
=======================================================
The <a href=""http://rdp.cme.msu.edu"">Ribosomal Database Project</a> is running a user survey on habitat terms that are most important to users.  Click on the RDP website and the quick survey will automatically launch in your browser: <a href=""http://rdp.cme.msu.edu"">http://rdp.cme.msu.edu</a>

The survey is part of the Habitat-Lite initiative within the <a href=""http://gensc.org"">Genomic Standards Consortium</a> (GSC). Habitat-Lite is a small set of high-level terms for describing the habitat of biological samples, particular selected to cover genomes, metagenome, 16/18S sequences and related molecules.  All Habitat-Lite terms are taken from the much larger Environment Ontology (<a href=""http://environmentontology.org"">http://environmentontology.org</a>).

A paper on Habitat-Lite will appear as part of the special issue of OMICS to come from the 5th GSC workshop.  A online version can be seen here: <a href=""http://www.ncbi.nlm.nih.gov/pubmed/18416669?ordinalpos=4&amp;itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum"">Pubmed</a>"
---
layout: default
title: SILVA runs contextual data survey for rRNA sequences
category: news
tag: [pyilmaz]
date: 2008-05-21
---
{% include header.html %}

SILVA runs contextual data survey for rRNA sequences
=======================================================
The Silva database is running a survey on which types of contextual data are most important for describing rRNA sequences. 

The survey is quick to complete and can be found here: <a href=""http://www.arb-silva.de/faqs/survey/"">http://www.arb-silva.de/faqs/survey/</a>

The results of this survey will be presented in a session on ribosomal RNA in the next GSC 6 meeting in Oct, 2007 at the EBI.

Further background on the survey is found in the emerging proposal for a ""Minimum Information about an ENvironmental Sequence"" (MIENS) guideline that takes into account MIGS/MIMS and modifies/extends it to describe rRNA sequences and related markers.  This proposal is currently being developed in the GSC Wiki and feedback and contributions are welcomed from the wider community: <a href=""http://gensc.org/gc_wiki/index.php/MIGS/MIMS_for_16S"">http://gensc.org/gc_wiki/index.php/MIGS/MIMS_for_16S</a>"
---
layout: default
title: Putting Molecules on the Map
category: news
tag: [pyilmaz]
date: 2008-06-23
---
{% include header.html %}

Putting Molecules on the Map
=======================================================
At the heart of the MIGS specification is a call for all genomes and metagenomes to be reported with geographic location (lat/long). We were therefore very pleased to see a Nature Editorial calling for ALL biological samples to be tagged with lat/long.  The GSC responded with a Nature Correspondence to underscore our collaborative interest in seeing all molecules (in particular genomes, metagenomes and phylogenetic markers like 16S/18S ribosomal RNA genes) placed into a geospatial framework. Location of isolation is increasingly of relevance as an informative piece of contextual data for the interpretation of collections of sequences.  In addition to the GSC, the Correspondence was signed by the INSDC, CBOL, ICoMM, and EnvO

The original Editorial can be found here: <a href=""http://www.nature.com/nature/journal/v453/n7191/full/453002a.html"">http://www.nature.com/nature/journal/v453/n7191/full/453002a.html </a>

Our response is here (see bottom of article for all signatories including the GSC): <a href=""http://www.nature.com/nature/journal/v453/n7198/full/453978b.html"">http://www.nature.com/nature/journal/v453/n7198/full/453978b.html </a>

Both pieces are available as free content and can be accessed without a subscription to Nature. 

The full list of <a href=""http://gensc.org/gc_wiki/index.php/Putting_Molecules_on_the_Map_Workspace"">Signatories</a> on the Correspondence can be seen here."
---
layout: default
title: Special issue of OMICS from the Genomic Standards Consortium (GSC)
category: news
tag: [pyilmaz]
date: 2008-06-23
---
{% include header.html %}

Special issue of OMICS from the Genomic Standards Consortium (GSC)
=======================================================
The <a href=""http://gensc.org"">Genomic Standards Consortium (GSC)</a> has produced a second special issue of <a href=""http://www.liebertonline.com/toc/omi/12/2"">OMICS</a>.  The first, published in 2006, brought together 20 invited papers from a diversity of standards communities, many of which are still in the <a href=""http://www.liebertonline.com/action/showMostReadArticles?journalCode=omi"">most read</a> list of OMICS papers. This special issue focuses exclusively on the efforts of the GSC as defined at the 5th GSC workshop held at the European Bioinformatics Institute (EBI) in December 2007.  All papers in both issues are available online as free content and can be accessed without a subscription.

Community-driven standards activities are becoming increasingly important given the deluge of data being generated across the life-sciences, and in particular in the area of 'omics.  This is nowhere more true than in the field of genomics, especially given the explosion of metagenomic studies and the development of ultra high-throughput sequencing technologies.

The mission of the GSC is to improve the descriptions of genomes and metagenomes in the public domain and promote data sharing and exchange. Following the publication of its  ""Minimum Information about a Genome Sequence"" guideline in May 2008, the GSC is now focusing on a set of projects that are designed to make MIGS an operational standard.  Read the papers in this special issue to learn more about the Genomic Contextual Data Markup Language (GCDML) which implements MIGS in XML, the Genomic Rosetta Stone project, a mapping of genomic identifiers, and Habitat-Lite, a high-level, short list of terms selected from the Environment Ontology (EnvO).  These, and other activities, are summarized in the proceedings of the 5th GSC workshop report also found in this issue.  Combined, all of these projects are steps towards realizing the vision of creating a comprehensive and easily accessible store of information about our complete genome and metagenome collection.
"
---
layout: default
title: GSC 6 and 7
category: news
tag: [pyilmaz]
date: 2008-09-05
---
{% include header.html %}

GSC 6 and 7
=======================================================
The GSC 6 workshop will take place at the EBI Oct 15-17, 2008.  There will be a GCDML satellite meeting the 13 and 14 of that week.  Further details and an agenda have been posted into the GSC wiki. 


GSC 6: <a href=""http://gensc.org/gc_wiki/index.php/GSC_6"">http://gensc.org/gc_wiki/index.php/GSC_6</a>

This meeting will be followed by GSC 7, which will be combined with the much larger Metagenomics 2008 meeting to be held in San Diego, the week of November 3rd.

GSC 7 at MG08: <a href=""http://metagenomics.calit2.net/index.php"">http://metagenomics.calit2.net/index.php</a>

"
---
layout: default
title: GCDML and GSC 6 workshops start this week (Oct 13, 2008)
category: news
tag: [pyilmaz]
date: 2008-10-12
---
{% include header.html %}

GCDML and GSC 6 workshops start this week (Oct 13, 2008)
=======================================================
We are looking forward to seeing everyone this week starting with the GCDML technical workshop participants.  We hope to get a lot done on Monday and Tuesday on GCDML before the main workshop starts.

The agendas for both meetings are now available on the web and have been ‘wikified’ to add live links and other relevant information so we can use them as a central source of information during the workshop.

<strong><a href=""http://gensc.org/gc_wiki/index.php/GSC_6"">AGENDA and LOGISTICS</a></strong>

The three day workshop is packed with short talks in the first two days to maximize exchange of information and the formulation of ideas, but there is also time for discussion.  The third day, as always, is set aside for discussion and strategy development and is really when much of the lasting work gets done. We will split the morning into technical (GCDML) and strategic sessions and then rejoin to map out the next stage of the GSC roadmap.

As background to the workshop the GSC homepage has a new page to serve as a portal into all GSC projects.  

<a href=""http://gensc.org/gc_wiki/index.php/GSC_projects"">GSC Projects</a>

Links to the MIGS paper in Nat Biotech and the <a href=""http://gensc.wordpress.com/2008/06/23/special-issue-of-omics-from-the-genomic-standards-consortium-gsc/"">OMICS special issue</a> including papers on the core GSC project are found on the <a href=""http://gensc.org"">home page</a>.

"
---
layout: default
title: GSC projects webpage launched in Wiki
category: news
tag: [pyilmaz]
date: 2008-10-13
---
{% include header.html %}

GSC projects webpage launched in Wiki
=======================================================
Information about all GSC projects is now being collated on a single page in the GSC Wiki. Please log in during the workshop and help update pages relevant to you and your project(s). The GSC core projects are MIGS/MIMS, the Genome Catalogue, GCDML, GRS, Habitat-Lite and the GSC eJournal and SOP central repository.  In addition, the GSC projects page includes descriptions of the larger community efforts in which the GSC participates as a community (MIBBI, MINSEQE, EnvO, OBI, ISATAB).

<strong><a href=""http://gensc.org/gc_wiki/index.php/GSC_projects"">GSC Projects</a></strong>"
---
layout: default
title: GSC 7 Webcast now available
category: news
tag: [pyilmaz]
date: 2008-12-14
---
{% include header.html %}

GSC 7 Webcast now available
=======================================================
The Metagenomics 2008 meeting was a great venue for the GSC 7 meeting. The entire meeting, including the first day dedicated to GSC activities, was presented as a Webcast. This Web Cast has now been archived and all the talks can be viewed here: <a href=""http://metagenomics.calit2.net/webcast.php"">http://metagenomics.calit2.net/webcast.php</a>"
---
layout: default
title: GSC to host a Special Interest Group (SIG) meeting at ISMB 2009 in Sweden called ""M3""
category: news
tag: [pyilmaz]
date: 2009-01-14
---
{% include header.html %}

GSC to host a Special Interest Group (SIG) meeting at ISMB 2009 in Sweden called ""M3""
=======================================================
The GSC proposal to host a SIG this year at ISMB 2009 in Sweden has been successful.  Further details will follow soon but the one day meeting will be held Saturday June 27th before the start of the main ISMB / ECCB 2009 meeting in Stockholm.


The official title of the SIG is ""<strong>Metagenomics, Metadata and Metaanalysis (M3)</strong>"".  There will a set of invited speakers along with a call for papers and posters.  


Some further background about the spirit of the meeting below.  The ""M3"" Hompage will appear <a href=""http://gensc.org/gc_wiki/index.php/M3""><strong>here</strong></a> in time


*************************************************


ISMB SIG: <strong>Metagenomics, Metadata and Metaanalysis (M3)</strong>, <em>June 27th, Stockholm</em>


There are now thousands of genomes and metagenomes available for study (see the <a href=""http://www.genomesonline.org/"">Genomes Online Database</a>. Interest in improved sampling of diverse environments (e.g. ocean, soil, sediment, and a range of hosts) combined with advances in the development and application of ultra-high throughput sequence methodologies is set to vastly accelerate the pace at which new metagenomes are  generated. For example, in 2007, the Global Ocean Survey published scientific  analyses of 41 metagenomes, and as of October 2008, the submission of user-generated metagenomes to the public <a href=""http://metagenomics.nmpdr.org/"">MG-Rast</a> Annotation server surpassed 1300. We have also entered an era of ‘mega-sequencing projects’ that now include funded projects like the Genomic Encyclopedia of Bacteria and Archaea (<a href=""http://www.jgi.doe.gov/programs/GEBA/why.html"">GEBA</a>) project and the Human Microbiome Initiative (e.g. <a href=""http://nihroadmap.nih.gov/hmp/"">HMP</a>), with many more visionary projects on the horizon.

While genomes represent the full genetic (DNA) complement of a single organism, metagenomes represent the DNA of an entire community of organisms. Metagenomes are partial samples of complex and largely unknown communities that can only be poorly assembled. Genome and metagenomes are now also being complemented with studies of metatranscriptomes (community transcript profiles) and metaproteomes (community protein profiles). The comparative study of these datasets, including multi-omic data sets from the same community, bring with them the need for new computational approaches.
These data hold the promise of unparalleled insights into fundamental questions across a range of fields including evolution, ecology, environment biology, health and medicine. Advances stem from improved understandings of the combinations, abundances and functions of the organisms in these communities and their genes and pathways. We are just starting to exploit these technologies to understand the microbial world and have only scratched the surface in terms of sampling natural microbial diversity in terms of space and time.

This SIG will explore the latest concepts, algorithms, tools, informatics pipelines, databases and standards being developed to cope with the analysis of vast quantities of metagenomic data. Through a series of invited and contributed talks, a panel discussion, and flash talks associated with a poster session, we aim to highlight scientific advances in the field and identify core computational challenges facing the wider community.

We aim to bring together researchers collecting samples for metagenomic analysis with those building the computational infrastructure required to fully exploit them with those thinking about the implementation of standards. In particular, we aim to encourage the participation of researchers already using metadata to detect patterns of biological interest. For example, those that are detecting trends in collections of genomes or metagenomes based on habitat or geographic location, or by analogy for microbiome studies by host or anatomical location, or studies that integrate substantial environmental, phenotypic, or epidemiology data.

This SIG will be hosted by the <a href=""http://gensc.org"">Genomic Standards Consortium</a> (GSC). It will place emphasis on understanding the community’s needs for the complementary standards required to drive metagenomic science forward and support comparative studies. A SIG report will be published in the GSC’s online, open access journal “Standards in Genomic Sciences”. Contributed submissions based on talks and posters that describe studies that demonstrate the power of using curated (e.g. habitat or host) and measured (e.g. geographic location, salinity, temperature, or pH) contextual data in comparative metagenomic studies of large numbers of samples are encouraged. Likewise, papers describing new approaches, tools, databases, standards, ontologies or substantial new sets of curated metadata that aid in the integration and inter-operablility of disparate datasets are welcomed.

"
---
layout: default
title: Soil Metadata Survey for Microbial Studies
category: news
tag: [pyilmaz]
date: 2009-03-13
---
{% include header.html %}

Soil Metadata Survey for Microbial Studies
=======================================================
The <a href=""http://www.terragenome.org/index.php/Main_Page"">Terragenome project</a> has launched a survey to capture the most popular descriptors for describing microbial studies of soils.

Take the survey here: <a href=""http://cme.msu.edu/SoilMetadata/survey.jsp"">Soil Metadata Survey for Microbial Studies</a>

<strong>Background taken from the <a href=""http://cme.msu.edu/SoilMetadata/survey.jsp"">survey website</a>:</strong>


The purpose of this <a href=""http://cme.msu.edu/SoilMetadata/survey.jsp"">survey</a> is to determine what data about soil is believed to be important to accompany microbial molecular data entered into major databases, e.g <a href=""http://www.ncbi.nlm.nih.gov/Genbank/"">GenBank</a>, <a href=""http://www.ebi.ac.uk/embl/"">EMBL</a>, <a href=""http://rdp.cme.msu.edu/"">RDP</a>. The value of this metadata is to enable a search of the data for particular features and for determining relationships among soil features and molecular data over more global data sets. We are surveying the soil biology community to gain this assessment and then will work with the <a href=""http://gensc.org"">Genomic Standards Consortium</a> (GSC) and the <a href=""http://environmentontology.org"">Environmental Ontology</a> Consortium to ensure that soil data will be properly considered along with the metadata from other environments.

<em>
--from Jim Tiedje, Michigan State University, as an initiative
of the Genomic Standards Consortium and the new 
International Soil Metagenome Consortium (Terragenome)</em>
"
---
layout: default
title: MIGS-Compliant report published as Supplementary Table
category: news
tag: [pyilmaz]
date: 2009-03-13
---
{% include header.html %}

MIGS-Compliant report published as Supplementary Table
=======================================================
Another MIGS-compliant report has been published as a Supplementary an article - see Reference 137 and the table S1 listed in the Materials and Methods.

""Adaptations to Submarine Hydrothermal Environments Exemplified by the Genome of Nautilia profundicola""

See the full paper <a href=""http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1000362"">here</a>.



"
---
layout: default
title: Standards in Genomic Sciences Journal from the GSC
category: news
tag: [pyilmaz]
date: 2009-04-18
---
{% include header.html %}

Standards in Genomic Sciences Journal from the GSC
=======================================================
The GSC is launching an open-access online journal to support its mission.  The ""<a href=""http://standardsingenomics.org/index.php/sigen""><em>Standards in Genomic Sciences</em>"" (SIGS)</a> journal will provide a forum for publishing genome and metagenome notes structured according to the GSC's MIGS/MIMS specification.  It will also support the community by providing a venue for the publication of a wide range of articles that are standards-compliant and standards supportive. For example, this will include the publication of Standard Operating Procedures (SOPs) such as those describing the annotation pipelines of the major sequencing centres.  Submissions on a range of topics now welcomed.  For more information, please see the links below.  

SIGS: <a href=""http://standardsingenomics.org/index.php/sige"">website</a>
SIGS: <a href=""http://standardsingenomics.org/index.php/sigen/about/editorialTeam"">Editorial Board</a>
<a href=""http://standardsingenomics.org/index.php/sigen/about"">About SIGS</a>
"
---
layout: default
title: GSC 8 Outcomes
category: news
tag: [pyilmaz]
date: 2009-09-26
---
{% include header.html %}

GSC 8 Outcomes
=======================================================
The GSC 8 meeting was hosted at the Joint Genome Institute Sept 9-11, 2009.  It was preceded by an ISA-Tab-GCDML workshop.  The agenda can be see here on the <a href=""http://gensc.org/gc_wiki/index.php/GSC_8"">GSC 8 meeting homepage</a>.

The first report to appear on the meeting is in <a href=""http://www.genomeweb.com/informatics/genome-standards-consortium-evolves-and-expands-keep-pace-rapid-advances-field"">Bioinform</a>: 

The JGI is also producing a article and will post all talks as videos online soon.

Some of the GSC-8 photos are online at the <a href=""http://genomebiology.jgi-psf.org/Content/GSC8_photos.htm"">JGI website</a>.

A formal meeting report is being written by the organizers and will appear in the GSC's journal <a href=""http://standardsingenomics.org/"">Standards in the Genomic Sciences</a>."
---
layout: default
title: MIxS paper published in Nature Biotechnology
category: news
tag: [pyilmaz]
date: 2011-06-25
---
{% include header.html %}

MIxS paper published in Nature Biotechnology
=======================================================
An open-access article describing the GSC's <a title=""MIxS"" href=""http://gensc.org/index.php?title=MIxS"">MIxS</a> standard has been published in <a href=""http://www.nature.com/nbt/journal/v29/n5/full/nbt.1823.html"" rel=""nofollow"">Nature Biotechnology</a> - it includes the MIMARKS checklist and the set of environmental packages applicable to all GSC checklists and beyond."
---
layout: default
title: Standards in Genomics Sciences - Accepting Papers
category: news
tag: [pyilmaz]
date: 2011-06-25
---
{% include header.html %}

Standards in Genomics Sciences - Accepting Papers
=======================================================
The SIGS journal continues to grow and is now a leading journal for the publication of 'genome notes' (using the GSC's MIxS standard).  Please consider contributing your next articles to this Open Access journal."
---
layout: default
title: GSC Community paper in PloS Biology
category: news
tag: [pyilmaz]
date: 2011-07-25
---
{% include header.html %}

GSC Community paper in PloS Biology
=======================================================
<ul>
	<li>Please have a look at the GSC's <a href=""http://www.plosbiology.org/article/info%3Adoi%2F10.1371%2Fjournal.pbio.1001088"" rel=""nofollow"">community paper</a> in PLoS Biology</li>
</ul>"
---
layout: default
title: GSC 14, Oxford, UK
category: news
tag: [pyilmaz]
date: 2012-09-25
---
{% include header.html %}

GSC 14, Oxford, UK
=======================================================
<a title=""GSC 14"" href=""http://gensc.org/index.php?title=GSC_14"">GSC 14</a> was held in Oxford, UK, September 17-19, 2012, followed by satellite meetings and a hackathon September 19-21.  This meeting served to launch the Genomic Observatories Network."
---
layout: default
title: GSC 15 NCBI, Bethesda Maryland, USA
category: news
tag: [pyilmaz]
date: 2013-05-25
---
{% include header.html %}

GSC 15 NCBI, Bethesda Maryland, USA
=======================================================
The <a title=""GSC 15"" href=""http://gensc.org/index.php?title=GSC_15"">GSC 15</a> meeting was held at the National Institute of Health (NIH), Bethesda, MD, USA, April 22-24, 2013 (<a href=""http://gensc.org/gc_wiki/images/a/a7/GSC15MeetingBooklet.pdf"" rel=""nofollow"">Agenda GSC15) </a>with satellite meetings on April 24, 25, 26 including the GOs2 Meeting at the Smithsonian (<a href=""http://gensc.org/gc_wiki/index.php/Genomic_Observatories_Network_workshop_%28GOs2%29"" rel=""nofollow"">Agenda GOs)</a>."
---
layout: default
title: Re-Launch of GSC website
category: news
tag: [pyilmaz]
date: 2013-06-24
---
{% include header.html %}

Re-Launch of GSC website
=======================================================
We are currently working to relaunch the GSC website - combining the best features of the current wiki with structured introductory pages that include an overview of all GSC Projects and their descriptions."
---
layout: default
title: GSC Project Descriptions Launched
category: news
tag: [pyilmaz]
date: 2013-06-25
---
{% include header.html %}

GSC Project Descriptions Launched
=======================================================
As part of building the GSC Community all GSC projects have now created uniform descriptions of their activities.  There is an open call for existing and new projects to join the GSC.  Details to be posted soon."
---
layout: default
title: GSC 16 Oxford UK
category: news
tag: [pyilmaz]
date: 2014-02-04
---
{% include header.html %}

GSC 16 Oxford UK
=======================================================
The <a title=""GSC 16"" href=""http://wiki.gensc.org/index.php?title=GSC_16"">GSC 16</a> meeting will be held in Oxford, UK, March 31- April 2, 2014. Registration is now open.  Please visit the main page of the meeting for full details: <a href=""http://wiki.gensc.org/index.php?title=GSC_16"">GSC 16</a>

&nbsp;
<h2>The GSC 16 Meeting</h2>
<p dir=""ltr"">The 16th Workshop of the Genomic Standards Consortium <a href=""http://gensc.org/"">(GSC)</a></p>
<p dir=""ltr"">Theme: Biodiversity Genomics</p>
&nbsp;
<p dir=""ltr"">Date: March 31 - April 2, 2014</p>
<p dir=""ltr"">Co-organizers: Dawn Field, Neil Davies, Chris Meyer, Jack Gilbert, Rob Knight, Linda Amaral-Zettler, John Deck, Norman Morrison, Mesude Bicak, Frank Oliver Glöckner, Renzo Kottmann, Lynn Schriml, Peter Sterk, Matthias Obst, Folker Meyer, Nikos Kyrpides,Hans Peter Klenk, George Garrity, the GOs Network Steering Committee and the OSD Team</p>
&nbsp;
<p dir=""ltr"">The GSC 16th meeting will take place at Oxford University March 31 - April 2, 2014.</p>
&nbsp;
<p dir=""ltr"">Theme: The main theme of GSC16 is Biodiversity Genomics:</p>
&nbsp;
<p dir=""ltr"">Large efforts are underway to sequence much of the biodiversity on Earth.  Some focus on particular species (notably humans and other model organisms, or species of economic or ecological significance) and others on particular places or environments.  These studies, whether at the organismal or ecosystem scale, increasingly address change over time.  There is much potential overlap among these projects and an urgent need to establish common data standards as well as an opportunity to share lessons learned (e.g., biobanking, data policies, technological approaches, informatics platforms, theory and modeling etc.) GSC 16 will serve as the fourth meeting of the Genomic Observatories Network (GOs 4) and the first international meeting of the Ocean Sampling Day Consortium.</p>
&nbsp;

&nbsp;
<h2>REGISTRATION</h2>
As this is a residential meeting, registration and accommodation will be handled by Pembroke College. Please visit this <a href=""https://adobeformscentral.com/?f=BXmJF0jfjijUkN2fH7HnPw"">website</a> to register and indicate your housing requirements."
---
layout: default
title: GSC 17, May 4-6, 2015 DOE JGI, Walnut Creek, California, USA
category: news
tag: [pyilmaz]
date: 2014-09-26
---
{% include header.html %}

GSC 17, May 4-6, 2015 DOE JGI, Walnut Creek, California, USA
=======================================================
<p dir=""ltr"">The 17th Workshop of the Genomic Standards Consortium (GSC)</p>
<p dir=""ltr"">Location: DOE JGI at Walnut Creek, CA, USA</p>
Theme: Standards for the Microbial Dark Matter (uncultured microbial life).

GSC Organizers: Nikos Kyrpides, Neil Davies, Lynn Schriml, Ilene Mizrachi, Tatiana Tatusova, Folker Meyer, Hans-Peter Klenk, George Garrity, Peter Sterk
<p dir=""ltr"">Local Organizers: Tanja Woyke, Susannah Tringe, Amrita Pati, T.B.K. Reddy, Natalia Ivanova</p>
Dates: May 4-6, 2015
<p dir=""ltr"">More details about program and registration can be found on the <a href=""http://wiki.gensc.org/index.php?title=GSC_17"">GSC 17 web site</a>.</p>"
---
layout: default
title: Ocean Sampling Day movie now available
category: news
tag: [pyilmaz]
date: 2014-10-23
---
{% include header.html %}

Ocean Sampling Day movie now available
=======================================================
The Ocean Sampling Day (OSD) is a simultaneous sampling campaign of the world’s oceans which took place for the first time on the summer solstice -June 21st - this year. From footage shot during the event a movie was produced by Mediomix. The music has been produced by our musical gifted OSD Core Team member Mesude Bicak as part of TuneHouse Studios.

The movie (9:22 minutes) is now available from YouTube:
http://youtu.be/yUm7SsSe-cw?list=PLgacjRIHqvMC39eKYdGH0HAM68YszmbuJ

There is also a teaser (1:30 minutes) including the link to the OSD movie at the end:
http://youtu.be/hBGOkB-EImc?list=PLgacjRIHqvMC39eKYdGH0HAM68YszmbuJ

OSD is part of the EU 7FP project Micro B3 (Biodiversity, Bioinformatics, Biotechnology).

Links:
<ul>
	<li>http://www.microb3.eu/osd</li>
	<li>http://gensc.org/projects/micro-b3/</li>
</ul>"
---
layout: default
title: View GSC17 talks
category: news
tag: [pyilmaz]
date: 2015-05-27
---
{% include header.html %}

View GSC17 talks
=======================================================
View GSC17 <a title=""talks"" href=""https://docs.google.com/document/d/1eomJzuMthULcj3YE0e0z19TebqdBTcwTPPmxbSz0WLc/edit"" target=""_blank"">talks </a>in the GSC17 agenda."
---
layout: default
title: GSC at the Microbes in the City meeting
category: news
tag: [pyilmaz]
date: 2015-06-01
---
{% include header.html %}

GSC at the Microbes in the City meeting
=======================================================
Lynn Schriml will be presenting ""MIxS-BE"", and Jack Gilbert will be presenting ""The Built Environment Microbiome: Health and Disease"" at <a href=""http://www.nyas.org/Events/Detail.aspx?cid=6f843a9b-1e8a-4f4e-a907-c813c13eafdb"">Microbes in the City meeting</a> on June 19th, 2015"
---
layout: default
title: Ocean Sampling Day 2015
category: news
tag: [pyilmaz]
date: 2015-06-23
---
{% include header.html %}

Ocean Sampling Day 2015
=======================================================
<p style=""text-align: justify""><a href=""http://www.microb3.eu/osd"">Ocean Sampling Day (OSD)</a> was successfully completed for the second time! OSD is a simultaneous global mega-sequencing campaign aiming to generate the largest standardized microbial data set in a single day. An OSD Consortium paper has just been published in <a href=""http://www.gigasciencejournal.com/content/4/1/27"">GigaScience</a>. The consortium aims to coordinate and  support partnerships and networks between OSD sites.</p>
<img class=""size-medium wp-image-630 aligncenter"" alt=""IMG_4570"" src=""http://press3.mcs.anl.gov/gensc/files/2015/06/IMG_4570-233x300.jpg"" width=""233"" height=""300"" />"
---
layout: default
title: M2B3 Standard Published
category: news
tag: [pyilmaz]
date: 2015-06-23
---
{% include header.html %}

M2B3 Standard Published
=======================================================
<p style=""text-align: justify"">The M2B3 Reporting Standard has been published in the <a href=""http://www.standardsingenomics.com/content/10/1/20"">SIGS Journal</a>. The M2B3 standard describes minimal mandatory and recommended contextual information for a marine microbial sample obtained in the epipelagic zone, includes meaningful information for researchers in the oceanographic, biodiversity and molecular disciplines, and can easily be adopted by any marine laboratory with minimum sampling resources.</p>"
---
layout: default
title: GSC at the Microbiology of the Built Environment Conference
category: news
tag: [pyilmaz]
date: 2015-07-06
---
{% include header.html %}

GSC at the Microbiology of the Built Environment Conference
=======================================================
Lynn Schriml will be presenting a poster ""MIxS-BE"" during the fourth annual conference on the <a href=""http://microbiologybuiltenvironment.weebly.com/"">Microbiology of the Built Environment</a> between July 15-18, 2015."
---
layout: default
title: Standards for Automated Function Prediction
category: news
tag: [pyilmaz]
date: 2015-07-07
---
{% include header.html %}

Standards for Automated Function Prediction
=======================================================
GSC member Tatiana Tatusova will be present at the <a href=""http://www.iscb.org/ismbeccb2015"">ISMB/ECCB 2015</a> in Dublin, Ireland (July 10-14, 2015). She is taking part in a special interest group (SIG) meeting - <a href=""http://www.iscb.org/ismbeccb2015-program/sigs#afp-sig"">Automated Function Prediction (AFP-SIG)</a>. The mission of the AFP-SIG is to bring together computational biologists, experimental biologists and biocurators who are dealing with the important problem of gene and gene product function prediction, to share ideas and create collaborations."
---
layout: default
title: Jack Gilbert named one of the 50 Scientists that change the world
category: news
tag: [pyilmaz]
date: 2015-07-17
---
{% include header.html %}

Jack Gilbert named one of the 50 Scientists that change the world
=======================================================
Business Insider has compiled a list of 50 scientists from across the globe who are changing the world for the better, Jack Gilbert is one of them! More details are in the <a href=""http://uk.businessinsider.com/50-scientists-changing-the-world-2015-7?r=US&amp;IR=T"">article</a>."
---
layout: default
title: Reporting missing values
category: news
tag: [pyilmaz]
date: 2015-07-21
---
{% include header.html %}

Reporting missing values
=======================================================
The GSC MIxS project is supporting the missing/null value reporting language developed by the International Nucleotide Database Collaboration (<a href=""http://www.insdc.org/"">INSDC</a>). This language has been developed to be used where a value of an expected format for sample metadata reporting can not be provided. For more detailed on the supported terms see: <a href=""http://www.ebi.ac.uk/ena/about/missing-values-reporting"">http://www.ebi.ac.uk/ena/about/missing-values-reporting</a>"
---
layout: default
title: MIBiG data standard published!
category: news
tag: [pyilmaz]
date: 2015-08-19
---
{% include header.html %}

MIBiG data standard published!
=======================================================
Minimum Information about a Biosynthetic Gene cluster (MIBiG) data standard manuscript is available (open access) on Nature Chemical Biology: http://www.nature.com/nchembio/journal/v11/n9/full/nchembio.1890.html"
---
layout: default
title: MIBiG Standard featured in Innovations Report
category: news
tag: [pyilmaz]
date: 2015-09-04
---
{% include header.html %}

MIBiG Standard featured in Innovations Report
=======================================================
""Learning from Nature: Genomic database standard alleviates search for novel antibiotics"" article features the GSCs latest publication of the MIBiG standard: http://bit.ly/1QdlNND"
---
layout: default
title: New board members
category: news
tag: [pyilmaz]
date: 2015-09-18
---
{% include header.html %}

New board members
=======================================================
GSC welcomes our two new board member; Philip Hugenholtz and Granger Sutton!

Philip's research interests include the microbial ecology and evolution of host-associated ecosystems such as the termite hindgut and human microbiome, and genomic mapping of the microbial tree of life.  He has helped pioneer the use of culture-independent molecular methods to characterise microbial communities including marker gene and shotgun (metagenomic) approaches. Granger Sutton is a Professor of Informatics at JCVI and as the Prokaryotic Informatics Coordinator is responsible for all aspects of prokaryotic bioinformatics and software engineering for JCVI’s sequencing and sequence-related informatics, including assembly and annotation."
---
layout: default
title: Search NCBI BioSample for MIxS Samples
category: news
tag: [pyilmaz]
date: 2015-10-27
---
{% include header.html %}

Search NCBI BioSample for MIxS Samples
=======================================================
The BioSample database stores submitter-supplied descriptive information, or metadata, about the biological materials from which data stored in NCBI’s primary data archives are derived. The database is diverse; typical examples of a BioSample include a cell line, a primary tissue biopsy, an individual organism or an environmental isolate.
It is possible to search for BioSamples that contain MIxS-data. Search can be either on the checklist type (MIGS, MIMS or MIMARKS), or different environmental packages.
This example query retrieves all BioSamples that use the built environment package:
<a href=""https://www.ncbi.nlm.nih.gov/biosample/?term=package+migs%2Fmims%2Fmimarks+built%5BProperties%5D"">https://www.ncbi.nlm.nih.gov/biosample/?term=package+migs%2Fmims%2Fmimarks+built%5BProperties%5D</a>"
---
layout: default
title: Marine Genomics Journal and MIxS
category: news
tag: [pyilmaz]
date: 2015-10-27
---
{% include header.html %}

Marine Genomics Journal and MIxS
=======================================================
<a href=""http://www.journals.elsevier.com/marine-genomics"">Marine Genomics Journal</a> requires MIxS-compliant metadata from data article authors:

""3. Data description, with phenotypic (if available), physiological and biochemical key features following the MIxS mandatory information of the MIxS checklist and environmental packages. This information should be provided in a Table.""

&nbsp;

&nbsp;"
---
layout: default
title: IDEA Mosquito Workshop
category: news
tag: [pyilmaz]
date: 2016-02-01
---
{% include header.html %}

IDEA Mosquito Workshop
=======================================================
Neill Davies is a co-organizer of the <a href=""https://docs.google.com/document/d/1F8J7gVppBEvO2X5Y4WKBGILqjIFba4JHr5t8fdDYSPs/edit?ts=5694c9ca#"">IDEA Mosquito Workshop</a> in Moorea (1-5 February 2016). The workshop will address critical information gaps that hinder our ability to efficiently control mosquito vectors and thus the spread of mosquito-borne infectious diseases, and will look into social-ecological modelling of innovative genetic control approaches.

A key consideration during the workshop will be genetic data in its full context in order to better model vector-borne infectious diseases such as dengue and zika.

&nbsp;"
---
layout: default
title: GSC18 Agenda and Speakers
category: news
tag: [pyilmaz]
date: 2016-03-24
---
{% include header.html %}

GSC18 Agenda and Speakers
=======================================================
GSC18 preparations are continuing full speed! A summary agenda is now made available, following this link: <a href=""http://wiki.gensc.org/images/d/db/GSC18_summary_agenda.pdf"" target=""_blank"">http://wiki.gensc.org/images/d/db/GSC18_summary_agenda.pdf</a>

We are also happy to announce that the list of speakers is now complete. During the three-day workshop, we will hear from:

<strong>Keynote Speakers</strong>
<ul>
	<li>Arianna Broggiato - The Nagoya Protocol: Responsible and Ethical Global Sampling</li>
	<li>Nicky Mulder Standards Initiatives in H3ABioNet</li>
	<li>Phil Hugenholtz: Genomic Taxonomy, from Genus to Domain.</li>
	<li>Christos Ouzounis: Microbial Ecology in Greece and Bioinformatics</li>
	<li>Daniele Daffonchio: Extreme Environments and Microorganisms</li>
</ul>
<strong>Session Speakers</strong>
<ul>
	<li>Anastasis Oulas, HCMR, GR</li>
	<li>Antonio Fernandez-Guerra, MM-MPI, DE</li>
	<li>Aspasia Nisiotou, Nat Inst Agronomy, GR</li>
	<li>Claire Nedeléc, INRA, FR</li>
	<li>Danilo Ercolini, Uni Naples, IT</li>
	<li>Dimitra Manou, AUTH, GR</li>
	<li>Donovan Parks, ACE, AU</li>
	<li>Evangelos Pafilis, HCMR, GR</li>
	<li>Folker Meyer, ANL, US</li>
	<li>Frank Oliver Glöckner, MM-MPI, DE</li>
	<li>George Garrity, SIGS, US</li>
	<li>Georgios Kotoulas, HCMR, GR</li>
	<li>Georgios Tsiamis, Uni Patras, GR</li>
	<li>Guy Cochrane, EBI, UK</li>
	<li>Ilene Mizrachi, NCBI, US</li>
	<li>Jim Cole, MSU, US</li>
	<li>Klaas Deneudt, VLIZ, BE</li>
	<li>Konstantinos Kormas, Uni Thessaly, GR</li>
	<li>Lynn Schriml, UMB, US</li>
	<li>Manolis Ladoukakis, Uni Crete, GR</li>
	<li>Manolis Mandalakis, HCMR, GR</li>
	<li>Neil Davies, UCB, US</li>
	<li>Nicolas Tsesmetzis, Shell, UK</li>
	<li>Nikolaos Nikolaidis, TUC, GR</li>
	<li>Nikos Kyrpides, JGI, US</li>
	<li>Pascal Hingamp, Uni Méditerranée, FR</li>
	<li>Pantelis Katharios, HCMR, GR</li>
	<li>Pavlos Pavlidis, ICS FORTH, GR</li>
	<li>Pelin Yilmaz, MM-MPI, DE</li>
	<li>Petra ten Hoopen, EBI, UK</li>
	<li>Phillip Goldstein, Uni Colorado, US</li>
	<li>Pier Buttigieg, AWI, DE</li>
	<li>Renzo Kottmann, MM-MPI, DE</li>
	<li>Rob Finn, EBI, UK</li>
	<li>Scott Tighe, Uni Vermont, US</li>
	<li>Simon Claus, VLIZ, BE</li>
	<li>Wiebe Koiistra, SZN, IT</li>
</ul>
Remember to register for the meeting at: <a href=""https://www.regonline.com/builder/site/?eventid=1804035"" target=""_blank"">https://www.regonline.com/builder/site/?eventid=1804035</a>

For further details like accommodation, flights, and information about Crete , please visit our meeting website: <a href=""http://wiki.gensc.org/index.php?title=GSC_18"" target=""_blank"">http://wiki.gensc.org/index.php?title=GSC_18</a>"
---
layout: default
title: GSC18 registration is closed, see you in Crete!
category: news
tag: [pyilmaz]
date: 2016-05-30
---
{% include header.html %}

GSC18 registration is closed, see you in Crete!
=======================================================
"
---
layout: default
title: GSC18 Final Agenda
category: news
tag: [pyilmaz]
date: 2016-06-06
---
{% include header.html %}

GSC18 Final Agenda
=======================================================
Six more days to go until GSC18 in Crete! The final agenda is here: <a href=""http://wiki.gensc.org/images/d/d8/GSC18_agenda.pdf"">http://wiki.gensc.org/images/d/d8/GSC18_agenda.pdf</a>"
---
layout: default
title: GSC19 -- Save the date -- May 14-17th, 2017 Brisbane, Australia
category: news
tag: [lschriml]
date: 2016-09-01
---
{% include header.html %}

GSC19 -- Save the date -- May 14-17th, 2017 Brisbane, Australia
=======================================================
"
---
layout: default
title: GSC19 May14-17 Brisbane
category: news
tag: [pyilmaz]
date: 2016-12-07
---
{% include header.html %}

GSC19 May14-17 Brisbane
=======================================================
GSC19 workshop website is now online: <a href=""http://wiki.gensc.org/index.php?title=GSC_19"">http://wiki.gensc.org/index.php?title=GSC_19</a>"
---
layout: default
title: GSC19 registration now open
category: news
tag: [pyilmaz]
date: 2017-02-01
---
{% include header.html %}

GSC19 registration now open
=======================================================
We are happy to announce that registration for GSC19 is now open!
<a href=""https://www.regonline.com/builder/site/?eventid=1913695"">https://www.regonline.com/builder/site/?eventid=1913695</a>

The meeting will take place at the Stamford Plaza, Brisbane, Australia between 15th-17th May, 2017.
The theme of the meeting will be Extending Standards to Viruses and Microbial Eukaryotes.

The agenda is packed with exciting sessions and topics:
<li>Viral classification and genome standards</li>
<li>Microbial eukaryote classification and genome standards</li>
<li>Bioinformatic workflow standards, (large scale, reproducible, cost-effective)</li>
<li>Microbiomes of Australia</li>
<li>Modes and tempo of evolution</li>
<li>Genome taxonomy</li>

The agenda is still taking shape, you can find specific session details and confirmed speakers on the GSC19 homepage: <a href=""http://wiki.gensc.org/index.php?title=GSC_19"">http://wiki.gensc.org/index.php?title=GSC_19</a>


We are looking forward to seeing you all in Brisbane!"
---
layout: default
title: GSC19 speakers and agenda
category: news
tag: [pyilmaz]
date: 2017-02-25
---
{% include header.html %}

GSC19 speakers and agenda
=======================================================
GSC19 agenda and speakers are now available:

<img src=""http://press3.mcs.anl.gov/gensc/files/2017/05/GSC19-Final-Agenda-May-2017.jpg"" />

Registration will be open until April, 15 2017:
<a href=""https://www.regonline.com/builder/site/?eventid=1913695"">https://www.regonline.com/builder/site/?eventid=1913695</a>

The meeting will take place at the Stamford Plaza, Brisbane, Australia between 15th-17th May, 2017.
The theme of the meeting will be Extending Standards to Viruses and Microbial Eukaryotes.

The agenda is packed with exciting sessions and topics:
<ul>
 	<li>Viral classification and genome standards</li>
 	<li>Microbial eukaryote classification and genome standards</li>
 	<li>Bioinformatic workflow standards, (large scale, reproducible, cost-effective)</li>
 	<li>Microbiomes of Australia</li>
 	<li>Modes and tempo of evolution</li>
 	<li>Genome taxonomy</li>
</ul>
For further details about GSC19, please see the <a href=""http://wiki.gensc.org/index.php?title=GSC_19"">meeting page</a>.

We are looking forward to seeing you all in Brisbane!"
---
layout: default
title: GSC20 Next Year at UCSD
category: news
tag: [pyilmaz]
date: 2017-08-03
---
{% include header.html %}

GSC20 Next Year at UCSD
=======================================================
<a href=""http://press3.mcs.anl.gov/gensc/files/2017/08/Slide1-1.jpg""><img class=""alignnone size-full wp-image-1484"" src=""http://press3.mcs.anl.gov/gensc/files/2017/08/Slide1-1.jpg"" alt="""" width=""720"" height=""540"" /></a>"
---
layout: default
title: Expanding MIxS Standards to Genomes from Uncultivated Microorganisms
category: news
tag: [pyilmaz]
date: 2017-08-10
---
{% include header.html %}

Expanding MIxS Standards to Genomes from Uncultivated Microorganisms
=======================================================
<p class=""article-heading"">Two new checklists, Minimum information about a single amplified genome (MISAG) and a metagenome-assembled genome (MIMAG) of bacteria and archaea, have just been <a href=""http://www.nature.com/nbt/journal/v35/n8/full/nbt.3893.html?foxtrotcallback=true"">published in Nature Biotechnology</a>!</p>
The checklists were conceived by Tanja Woyke Robert Bowers of JGI. These extensions complement the MIxS suite of metadata standards by defining the key data elements pertinent for describing the sampling and sequencing of single-cell genomes and genomes from metagenomes, and they open up a whole new area of metadata data exploration as the vast majority of microbes, referred to as microbial dark matter, are currently not described within the MIxS standard.

&nbsp;"
---
layout: default
title: Genomic Observatories Metadatabase
category: news
tag: [pyilmaz]
date: 2017-08-14
---
{% include header.html %}

Genomic Observatories Metadatabase
=======================================================
Check out the <a href=""http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2002925"">GeOMe repositor</a>y, great work by our GBWG colleagues! The Genomic Observatories Metadatabase (GeOMe, <a href=""http://www.geome-db.org/"">http://www.geome-db.org/</a>) is an open access repository for geographic and ecological metadata associated with biosamples and genetic data. Whereas public databases have served as vital repositories for nucleotide sequences, they do not accession all the metadata required for ecological or evolutionary analyses. GeOMe fills this need, providing a user-friendly, web-based interface for both data contributors and data recipients. The interface allows data contributors to create a customized yet standard-compliant spreadsheet that captures the temporal and geospatial context of each biosample. These metadata are then validated and permanently linked to archived genetic data stored in the National Center for Biotechnology Information’s (NCBI’s) Sequence Read Archive (SRA) via unique persistent identifiers. By linking ecologically and evolutionarily relevant metadata with publicly archived sequence data in a structured manner, GeOMe sets a gold standard for data management in biodiversity science."
---
layout: default
title: GSC Speakers at Hellenic Bioinformatics 10
category: news
tag: [pyilmaz]
date: 2017-08-28
---
{% include header.html %}

GSC Speakers at Hellenic Bioinformatics 10
=======================================================
GSC colleagues will be speaking during the <a href=""https://hscbio.wordpress.com/conferences-when/2017-2/"">Hellenic Bioinformatics conference</a> (September 6-9 2017, Heraklion, Greece):
<ul>
 	<li>Folker Meyer, Argonne National Laboratory, USA</li>
</ul>
Title: A novel pipeline for high quality amplicon analysis integrated into MG-RAST
<ul>
 	<li>Lynn Schriml, University of Maryland, USA</li>
</ul>
Title: GSC metadata standards in action: MetaSUB and ISD.
<ul>
 	<li>David Ussery, University of Arkansas, USA</li>
</ul>
Title: What is Life? Conserved Functional Domains in a Hundred Thousand Bacterial Genomes

&nbsp;"
---
layout: default
title: GSC20 Agenda & Logistics
category: news
tag: [lschriml]
date: 2018-04-21
---
{% include header.html %}

GSC20 Agenda & Logistics
=======================================================
<a href=""http://gensc.org/meetings/gsc20/"" target=""_blank"" rel=""noopener"">GSC20</a>

<a href=""https://docs.google.com/document/d/1-BiqeKLefCJJIESACSjPpHyxxytT4PaW-ggnRwNzBb0/edit"" target=""_blank"" rel=""noopener"">AGENDA</a> : daily agenda, speakers

May 1: 2:00 pm GSC Board Meeting
May 2-4: GSC20 Annual Meeting
Doors open at 8:30 am.
GSC20:  9:00 am – 5:00 pm.

<a href=""https://docs.google.com/document/d/1Xt9QtzDz26A_znnz076pv7AyAGXB4UCNQZS7LqIrZXw/edit?usp=sharing"" target=""_blank"" rel=""noopener"">LOGISTICS</a>: walking directions, maps, parking

Venue: <span class=""s1"">Center for Microbiome Innovation, </span><span class=""s1""><strong>Atkinson Hall</strong> at </span>University of California San Diego, USA

We look forward to exploring the nexus of genomic standards, microbiomes
and advances in metabolomics, immunome and taxonomy at <a href=""http://gensc.org/meetings/gsc20/"" target=""_blank"" rel=""noopener"">GSC20</a>.

<strong>Registration</strong> has closed<strong>. </strong>(April 21, 2018)

&nbsp;"
---
layout: default
title: New Board Members
category: news
tag: [pyilmaz]
date: 2018-05-07
---
{% include header.html %}

New Board Members
=======================================================
GSC welcomes two new board members; Emiley Eloe-Fadrosh and Christopher Hunter!

Emiley is the Metagenome Program Lead at the DOE-JGI, and her current research focuses on leveraging thousands of metagenomic datasets from host-associated and environmental samples to identify novel microbial life and viral diversity.

Christopher is currently the lead curator of Gigascience. He project manages GigaDB, a database of large-scale biological and biomedical datasets."
---
layout: default
title: GSC21@Vienna
category: news
tag: [pyilmaz]
date: 2018-07-30
---
{% include header.html %}

GSC21@Vienna
=======================================================
GSC21 preparations are on! The meeting will take place in Vienna (Austria), on May 20-23, 2019. For further details, be sure to check the meeting page <a href=""http://www.gensc.org/meetings/gsc21"">here</a>.

&nbsp;"
---
layout: default
title: GSC21 Meeting - agenda, maps, logistics
category: news
tag: [lschriml]
date: 2019-05-03
---
{% include header.html %}

GSC21 Meeting - agenda, maps, logistics
=======================================================
<h1>GSC21 Meeting page: <a href=""https://press3.mcs.anl.gov/gensc/meetings/gsc21/"">Click Here</a></h1>
<h1>Thank you for registering for GSC21 !!</h1>
<h1>May 20-23, 2019</h1>
<h1>Vienna,Austria</h1>"
---
layout: default
title: Save The Date: GSC22 July 13-17, 2020 Bangkok, Thailand
category: news
tag: [lschriml]
date: 2019-07-26
---
{% include header.html %}

Save The Date: GSC22 July 13-17, 2020 Bangkok, Thailand
=======================================================
<a href=""https://press3.mcs.anl.gov/gensc/gsc22/"" target=""_blank"" rel=""noopener"">GSC22 Meeting Page</a>

The <strong><a href=""https://www2.si.mahidol.ac.th/en/"" target=""_blank"" rel=""noopener"">Faculty of Medicine Siriraj Hospital at Mahidol University</a></strong> and the <b><a href=""http://www.biotec.or.th/en/"" target=""_blank"" rel=""noopener"">National Center for Genetic Engineering and Biotechnology</a> (BIOTEC/NSTDA) </b>will host the 22<sup>nd </sup>Genomic Standards Consortium (GSC) meeting from July 13<sup>13h</sup>-17<sup>th</sup>, 2020 in Bangkok, Thailand.

<strong>GSC22: </strong><strong>Genomic Standards for Precision Medicine, Agriculture, </strong>
<strong>                     Comparative Genomics, and Metabolomics</strong>

This meeting is intended for professionals within the fields of medical genomics, agriculture, comparative genomics, and metabolomics tackling metadata challenges related to the sample collection, processing and genomic sequencing workflow critical for small and large scale datasets. It is our intention to discuss and plan for evolving genomic and laboratory standards to address the needs of the clinical, academic and industry researchers with a focus on precision medicine and agriculture."
---
layout: default
title: GSC22 (Thailand) is being postponed until late 2020 or early 2021.
category: news
tag: [lschriml]
date: 2020-04-13
---
{% include header.html %}

GSC22 (Thailand) is being postponed until late 2020 or early 2021.
=======================================================
"
