[![Genomic Standards Consortium](/images/cropped-full_gsc_logo_sml.png)](https://gensc.org/)

| [Home](http://gensc.org) | [Meeting](/pages/meetings.md) | [About](/pages/about.md) | [News](/pages/news.md) | [Projects](/pages/projects.md) | [Checklists](/pages/checklists.md) | [Contact](/pages/contact.md) | 
 

# GSC Minimum Information about any Sequence (MIxS)

The GSC family of minimum information standards (checklists) â€“ Minimum Information about any (x) Sequence (MIxS)

MIxS currently consists of three separate checklists; **[MIGS for genomes](https://pubmed.ncbi.nlm.nih.gov/18464787 "MIGS/MIMS")**, **[MIMS for metagenomes](https://pubmed.ncbi.nlm.nih.gov/18479204/ "MIGS/MIMS")**, and [**MIMARKS for marker genes**](https://pubmed.ncbi.nlm.nih.gov/21552244/).

To create a single entry point to all minimum information checklists from the GSC and to the environmental packages, we created an overarching framework, the MIxS standard (publication in Nature Biotechnology). MIxS includes the technology-specific checklists from the previous MIGS and MIMS standards, provides a way of introducing additional checklists such as MIMARKS, and also allows annotation of sample data using environmental packages.

## [The checklists](/pages/standards/mixs.md)
[Click here for more details on the MIxS checklists](/pages/standards/mixs.md)

Visit our GitHub repository to find previous versions of the checklists.

## [Compliance and Implementation](/pages/standards/compliance.md)
Details of how to comply with the standard, and which repositories and institutions are currently making use of them

## [Extensions](/pages/standards/mixs-ext-and-profiles.md)
Many of the core packages also have multiple extensions of recomended terms that have been compiled by experts in the relevant field of research. Explore all the ready made checklist extensions that we have available
If there is not a suitable checklist already available and you are interested in developing your own extention, [this page](/pages/standards/mixs-ext-and-profiles.md) also contains details on Developing Extensions that you may find useful.


